{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "import subprocess\n",
    "import os\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>TargetID</th>\n",
       "      <th>Chr</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>149</th>\n",
       "      <td>ENSG00000226143.1</td>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>151</th>\n",
       "      <td>ENSG00000125846.11</td>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>253</th>\n",
       "      <td>ENSG00000177410.8</td>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>397</th>\n",
       "      <td>ENSG00000235167.1</td>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>479</th>\n",
       "      <td>ENSG00000230568.1</td>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23561</th>\n",
       "      <td>ENSG00000215063.3</td>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23613</th>\n",
       "      <td>ENSG00000124228.9</td>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23646</th>\n",
       "      <td>ENSG00000125966.7</td>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23672</th>\n",
       "      <td>ENSG00000254995.4</td>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23683</th>\n",
       "      <td>ENSG00000232241.1</td>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>561 rows Ã— 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                 TargetID Chr\n",
       "149     ENSG00000226143.1  20\n",
       "151    ENSG00000125846.11  20\n",
       "253     ENSG00000177410.8  20\n",
       "397     ENSG00000235167.1  20\n",
       "479     ENSG00000230568.1  20\n",
       "...                   ...  ..\n",
       "23561   ENSG00000215063.3  20\n",
       "23613   ENSG00000124228.9  20\n",
       "23646   ENSG00000125966.7  20\n",
       "23672   ENSG00000254995.4  20\n",
       "23683   ENSG00000232241.1  20\n",
       "\n",
       "[561 rows x 2 columns]"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "phenotypes = pd.read_csv('../data/GD462.GeneQuantRPKM.50FN.samplename.resk10.txt.gz', compression='gzip', sep='\\t')\n",
    "filtered_phenotypes = phenotypes[~phenotypes['Chr'].isin(['X', 'Y', 'M'])]\n",
    "to_calc = filtered_phenotypes[['TargetID', 'Chr']]\n",
    "to_calc = to_calc[to_calc['Chr'].isin(['20'])]\n",
    "to_calc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000226143.1 --out compute_weights_out/ENSG00000226143.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000226143.1.txt\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226143.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000226143.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000226143.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000226143.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226143.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000226143.1.bed +\n",
      "temp_files/test_chr20_ENSG00000226143.1.bim +\n",
      "temp_files/test_chr20_ENSG00000226143.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226143.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000226143.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000226143.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226143.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000226143.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000226143.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125846.11 --out compute_weights_out/ENSG00000125846.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125846.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000226143.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125846.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125846.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125846.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000125846.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125846.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125846.11.bed +\n",
      "temp_files/test_chr20_ENSG00000125846.11.bim +\n",
      "temp_files/test_chr20_ENSG00000125846.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125846.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125846.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125846.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125846.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125846.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125846.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000177410.8 --out compute_weights_out/ENSG00000177410.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000177410.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125846.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000177410.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000177410.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000177410.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000177410.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000177410.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000177410.8.bed +\n",
      "temp_files/test_chr20_ENSG00000177410.8.bim +\n",
      "temp_files/test_chr20_ENSG00000177410.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000177410.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000177410.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000177410.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000177410.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000177410.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000177410.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000235167.1 --out compute_weights_out/ENSG00000235167.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000235167.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000177410.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000235167.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000235167.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000235167.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000235167.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000235167.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000235167.1.bed +\n",
      "temp_files/test_chr20_ENSG00000235167.1.bim +\n",
      "temp_files/test_chr20_ENSG00000235167.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000235167.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000235167.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000235167.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000235167.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000235167.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000235167.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000230568.1 --out compute_weights_out/ENSG00000230568.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000230568.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000235167.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000230568.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000230568.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000230568.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000230568.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000230568.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000230568.1.bed +\n",
      "temp_files/test_chr20_ENSG00000230568.1.bim +\n",
      "temp_files/test_chr20_ENSG00000230568.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000230568.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000230568.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000230568.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000230568.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000230568.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000230568.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000100982.6 --out compute_weights_out/ENSG00000100982.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000100982.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000230568.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000100982.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000100982.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000100982.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000100982.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000100982.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000100982.6.bed +\n",
      "temp_files/test_chr20_ENSG00000100982.6.bim +\n",
      "temp_files/test_chr20_ENSG00000100982.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000100982.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000100982.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000100982.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000100982.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000100982.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000100982.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101439.4 --out compute_weights_out/ENSG00000101439.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101439.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000100982.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101439.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101439.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101439.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000101439.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101439.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101439.4.bed +\n",
      "temp_files/test_chr20_ENSG00000101439.4.bim +\n",
      "temp_files/test_chr20_ENSG00000101439.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101439.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101439.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101439.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101439.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101439.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101439.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125869.4 --out compute_weights_out/ENSG00000125869.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125869.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101439.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125869.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125869.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125869.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000125869.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125869.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125869.4.bed +\n",
      "temp_files/test_chr20_ENSG00000125869.4.bim +\n",
      "temp_files/test_chr20_ENSG00000125869.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125869.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125869.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125869.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125869.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125869.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125869.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000149474.7 --out compute_weights_out/ENSG00000149474.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000149474.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125869.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149474.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000149474.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000149474.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000149474.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149474.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000149474.7.bed +\n",
      "temp_files/test_chr20_ENSG00000149474.7.bim +\n",
      "temp_files/test_chr20_ENSG00000149474.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149474.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000149474.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000149474.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149474.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000149474.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000149474.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000088340.10 --out compute_weights_out/ENSG00000088340.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000088340.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000149474.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088340.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000088340.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000088340.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000088340.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088340.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000088340.10.bed +\n",
      "temp_files/test_chr20_ENSG00000088340.10.bim +\n",
      "temp_files/test_chr20_ENSG00000088340.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088340.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000088340.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000088340.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088340.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000088340.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000088340.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124103.8 --out compute_weights_out/ENSG00000124103.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124103.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000088340.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124103.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124103.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124103.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000124103.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124103.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124103.8.bed +\n",
      "temp_files/test_chr20_ENSG00000124103.8.bim +\n",
      "temp_files/test_chr20_ENSG00000124103.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124103.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124103.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124103.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124103.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124103.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124103.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000089057.10 --out compute_weights_out/ENSG00000089057.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000089057.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124103.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000089057.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000089057.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000089057.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000089057.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000089057.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000089057.10.bed +\n",
      "temp_files/test_chr20_ENSG00000089057.10.bim +\n",
      "temp_files/test_chr20_ENSG00000089057.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000089057.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000089057.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000089057.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000089057.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000089057.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000089057.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000131043.7 --out compute_weights_out/ENSG00000131043.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000131043.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000089057.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000131043.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000131043.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000131043.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000131043.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000131043.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000131043.7.bed +\n",
      "temp_files/test_chr20_ENSG00000131043.7.bim +\n",
      "temp_files/test_chr20_ENSG00000131043.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000131043.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000131043.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000131043.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000131043.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000131043.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000131043.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000054793.9 --out compute_weights_out/ENSG00000054793.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000054793.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000131043.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000054793.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000054793.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000054793.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000054793.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000054793.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000054793.9.bed +\n",
      "temp_files/test_chr20_ENSG00000054793.9.bim +\n",
      "temp_files/test_chr20_ENSG00000054793.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000054793.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000054793.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000054793.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000054793.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000054793.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000054793.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000244274.3 --out compute_weights_out/ENSG00000244274.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000244274.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000054793.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000244274.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000244274.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000244274.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000244274.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000244274.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000244274.3.bed +\n",
      "temp_files/test_chr20_ENSG00000244274.3.bim +\n",
      "temp_files/test_chr20_ENSG00000244274.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000244274.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000244274.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000244274.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000244274.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000244274.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000244274.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101181.12 --out compute_weights_out/ENSG00000101181.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101181.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000244274.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101181.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101181.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101181.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000101181.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101181.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101181.12.bed +\n",
      "temp_files/test_chr20_ENSG00000101181.12.bim +\n",
      "temp_files/test_chr20_ENSG00000101181.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101181.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101181.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101181.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101181.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101181.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101181.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000214612.3 --out compute_weights_out/ENSG00000214612.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000214612.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101181.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000214612.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000214612.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000214612.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000214612.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000214612.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000214612.3.bed +\n",
      "temp_files/test_chr20_ENSG00000214612.3.bim +\n",
      "temp_files/test_chr20_ENSG00000214612.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000214612.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000214612.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000214612.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000214612.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000214612.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000214612.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125772.7 --out compute_weights_out/ENSG00000125772.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125772.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000214612.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125772.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125772.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125772.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000125772.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125772.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125772.7.bed +\n",
      "temp_files/test_chr20_ENSG00000125772.7.bim +\n",
      "temp_files/test_chr20_ENSG00000125772.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125772.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125772.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125772.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125772.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125772.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125772.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000237788.1 --out compute_weights_out/ENSG00000237788.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000237788.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125772.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000237788.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000237788.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000237788.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000237788.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000237788.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000237788.1.bed +\n",
      "temp_files/test_chr20_ENSG00000237788.1.bim +\n",
      "temp_files/test_chr20_ENSG00000237788.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000237788.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000237788.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000237788.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000237788.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000237788.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000237788.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000172070.5 --out compute_weights_out/ENSG00000172070.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000172070.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000237788.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000172070.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000172070.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000172070.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000172070.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000172070.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000172070.5.bed +\n",
      "temp_files/test_chr20_ENSG00000172070.5.bim +\n",
      "temp_files/test_chr20_ENSG00000172070.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000172070.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000172070.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000172070.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000172070.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000172070.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000172070.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000198646.7 --out compute_weights_out/ENSG00000198646.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000198646.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000172070.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000198646.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000198646.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000198646.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000198646.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000198646.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000198646.7.bed +\n",
      "temp_files/test_chr20_ENSG00000198646.7.bim +\n",
      "temp_files/test_chr20_ENSG00000198646.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000198646.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000198646.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000198646.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000198646.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000198646.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000198646.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000132823.5 --out compute_weights_out/ENSG00000132823.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000132823.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000198646.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132823.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000132823.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000132823.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000132823.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132823.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000132823.5.bed +\n",
      "temp_files/test_chr20_ENSG00000132823.5.bim +\n",
      "temp_files/test_chr20_ENSG00000132823.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132823.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000132823.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000132823.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132823.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000132823.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000132823.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125863.12 --out compute_weights_out/ENSG00000125863.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125863.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000132823.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125863.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125863.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125863.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000125863.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125863.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125863.12.bed +\n",
      "temp_files/test_chr20_ENSG00000125863.12.bim +\n",
      "temp_files/test_chr20_ENSG00000125863.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125863.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125863.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125863.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125863.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125863.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125863.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101161.6 --out compute_weights_out/ENSG00000101161.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101161.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125863.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101161.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101161.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101161.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000101161.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101161.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101161.6.bed +\n",
      "temp_files/test_chr20_ENSG00000101161.6.bim +\n",
      "temp_files/test_chr20_ENSG00000101161.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101161.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101161.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101161.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101161.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101161.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101161.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000215305.4 --out compute_weights_out/ENSG00000215305.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000215305.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101161.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000215305.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000215305.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000215305.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000215305.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000215305.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000215305.4.bed +\n",
      "temp_files/test_chr20_ENSG00000215305.4.bim +\n",
      "temp_files/test_chr20_ENSG00000215305.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000215305.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000215305.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000215305.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000215305.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000215305.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000215305.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000203896.5 --out compute_weights_out/ENSG00000203896.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000203896.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000215305.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000203896.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000203896.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000203896.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000203896.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000203896.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000203896.5.bed +\n",
      "temp_files/test_chr20_ENSG00000203896.5.bim +\n",
      "temp_files/test_chr20_ENSG00000203896.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000203896.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000203896.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000203896.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000203896.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000203896.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000203896.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101474.7 --out compute_weights_out/ENSG00000101474.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101474.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000203896.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101474.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101474.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101474.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000101474.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101474.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101474.7.bed +\n",
      "temp_files/test_chr20_ENSG00000101474.7.bim +\n",
      "temp_files/test_chr20_ENSG00000101474.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101474.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101474.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101474.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101474.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101474.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101474.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000229299.1 --out compute_weights_out/ENSG00000229299.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000229299.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101474.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000229299.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000229299.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000229299.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000229299.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000229299.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000229299.1.bed +\n",
      "temp_files/test_chr20_ENSG00000229299.1.bim +\n",
      "temp_files/test_chr20_ENSG00000229299.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000229299.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000229299.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000229299.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000229299.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000229299.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000229299.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000225988.1 --out compute_weights_out/ENSG00000225988.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000225988.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000229299.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225988.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000225988.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000225988.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000225988.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225988.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000225988.1.bed +\n",
      "temp_files/test_chr20_ENSG00000225988.1.bim +\n",
      "temp_files/test_chr20_ENSG00000225988.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225988.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000225988.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000225988.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225988.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000225988.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000225988.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000132635.10 --out compute_weights_out/ENSG00000132635.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000132635.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000225988.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132635.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000132635.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000132635.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000132635.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132635.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000132635.10.bed +\n",
      "temp_files/test_chr20_ENSG00000132635.10.bim +\n",
      "temp_files/test_chr20_ENSG00000132635.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132635.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000132635.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000132635.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132635.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000132635.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000132635.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000230010.1 --out compute_weights_out/ENSG00000230010.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000230010.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000132635.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000230010.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000230010.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000230010.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000230010.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000230010.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000230010.1.bed +\n",
      "temp_files/test_chr20_ENSG00000230010.1.bim +\n",
      "temp_files/test_chr20_ENSG00000230010.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000230010.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000230010.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000230010.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000230010.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000230010.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000230010.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000237595.1 --out compute_weights_out/ENSG00000237595.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000237595.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000230010.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000237595.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000237595.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000237595.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000237595.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000237595.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000237595.1.bed +\n",
      "temp_files/test_chr20_ENSG00000237595.1.bim +\n",
      "temp_files/test_chr20_ENSG00000237595.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000237595.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000237595.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000237595.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000237595.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000237595.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000237595.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101040.14 --out compute_weights_out/ENSG00000101040.14_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101040.14.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000237595.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101040.14.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101040.14.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101040.14\n",
      "  --pheno ../data/gene_expressions/ENSG00000101040.14.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101040.14.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101040.14.bed +\n",
      "temp_files/test_chr20_ENSG00000101040.14.bim +\n",
      "temp_files/test_chr20_ENSG00000101040.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101040.14.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101040.14\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101040.14\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101040.14.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101040.14.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101040.14.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000171700.9 --out compute_weights_out/ENSG00000171700.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000171700.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101040.14 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000171700.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000171700.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000171700.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000171700.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000171700.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000171700.9.bed +\n",
      "temp_files/test_chr20_ENSG00000171700.9.bim +\n",
      "temp_files/test_chr20_ENSG00000171700.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000171700.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000171700.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000171700.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000171700.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000171700.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000171700.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000212232.1 --out compute_weights_out/ENSG00000212232.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000212232.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000171700.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000212232.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000212232.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000212232.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000212232.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000212232.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000212232.1.bed +\n",
      "temp_files/test_chr20_ENSG00000212232.1.bim +\n",
      "temp_files/test_chr20_ENSG00000212232.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000212232.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000212232.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000212232.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000212232.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000212232.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000212232.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000149634.4 --out compute_weights_out/ENSG00000149634.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000149634.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000212232.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149634.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000149634.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000149634.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000149634.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149634.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000149634.4.bed +\n",
      "temp_files/test_chr20_ENSG00000149634.4.bim +\n",
      "temp_files/test_chr20_ENSG00000149634.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149634.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000149634.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000149634.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149634.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000149634.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000149634.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124098.9 --out compute_weights_out/ENSG00000124098.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124098.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000149634.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124098.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124098.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124098.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000124098.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124098.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124098.9.bed +\n",
      "temp_files/test_chr20_ENSG00000124098.9.bim +\n",
      "temp_files/test_chr20_ENSG00000124098.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124098.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124098.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124098.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124098.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124098.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124098.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125871.9 --out compute_weights_out/ENSG00000125871.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125871.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124098.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125871.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125871.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125871.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000125871.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125871.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125871.9.bed +\n",
      "temp_files/test_chr20_ENSG00000125871.9.bim +\n",
      "temp_files/test_chr20_ENSG00000125871.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125871.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125871.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125871.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125871.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125871.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125871.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101265.10 --out compute_weights_out/ENSG00000101265.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101265.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125871.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101265.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101265.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101265.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000101265.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101265.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101265.10.bed +\n",
      "temp_files/test_chr20_ENSG00000101265.10.bim +\n",
      "temp_files/test_chr20_ENSG00000101265.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101265.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101265.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101265.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101265.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101265.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101265.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000171984.9 --out compute_weights_out/ENSG00000171984.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000171984.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101265.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000171984.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000171984.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000171984.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000171984.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000171984.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000171984.9.bed +\n",
      "temp_files/test_chr20_ENSG00000171984.9.bim +\n",
      "temp_files/test_chr20_ENSG00000171984.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000171984.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000171984.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000171984.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000171984.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000171984.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000171984.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000149531.9 --out compute_weights_out/ENSG00000149531.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000149531.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000171984.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149531.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000149531.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000149531.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000149531.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149531.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000149531.9.bed +\n",
      "temp_files/test_chr20_ENSG00000149531.9.bim +\n",
      "temp_files/test_chr20_ENSG00000149531.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149531.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000149531.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000149531.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149531.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000149531.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000149531.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000223346.1 --out compute_weights_out/ENSG00000223346.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000223346.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000149531.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000223346.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000223346.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000223346.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000223346.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000223346.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000223346.1.bed +\n",
      "temp_files/test_chr20_ENSG00000223346.1.bim +\n",
      "temp_files/test_chr20_ENSG00000223346.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000223346.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000223346.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000223346.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000223346.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000223346.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000223346.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124177.10 --out compute_weights_out/ENSG00000124177.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124177.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000223346.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124177.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124177.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124177.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000124177.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124177.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124177.10.bed +\n",
      "temp_files/test_chr20_ENSG00000124177.10.bim +\n",
      "temp_files/test_chr20_ENSG00000124177.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124177.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124177.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124177.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124177.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124177.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124177.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000089195.10 --out compute_weights_out/ENSG00000089195.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000089195.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124177.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000089195.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000089195.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000089195.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000089195.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000089195.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000089195.10.bed +\n",
      "temp_files/test_chr20_ENSG00000089195.10.bim +\n",
      "temp_files/test_chr20_ENSG00000089195.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000089195.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000089195.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000089195.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000089195.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000089195.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000089195.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000175170.8 --out compute_weights_out/ENSG00000175170.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000175170.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000089195.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000175170.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000175170.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000175170.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000175170.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000175170.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000175170.8.bed +\n",
      "temp_files/test_chr20_ENSG00000175170.8.bim +\n",
      "temp_files/test_chr20_ENSG00000175170.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000175170.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000175170.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000175170.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000175170.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000175170.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000175170.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000175730.7 --out compute_weights_out/ENSG00000175730.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000175730.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000175170.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000175730.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000175730.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000175730.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000175730.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000175730.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000175730.7.bed +\n",
      "temp_files/test_chr20_ENSG00000175730.7.bim +\n",
      "temp_files/test_chr20_ENSG00000175730.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000175730.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000175730.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000175730.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000175730.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000175730.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000175730.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000158296.9 --out compute_weights_out/ENSG00000158296.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000158296.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000175730.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000158296.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000158296.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000158296.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000158296.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000158296.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000158296.9.bed +\n",
      "temp_files/test_chr20_ENSG00000158296.9.bim +\n",
      "temp_files/test_chr20_ENSG00000158296.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000158296.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000158296.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000158296.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000158296.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000158296.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000158296.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000042062.7 --out compute_weights_out/ENSG00000042062.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000042062.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000158296.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000042062.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000042062.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000042062.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000042062.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000042062.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000042062.7.bed +\n",
      "temp_files/test_chr20_ENSG00000042062.7.bim +\n",
      "temp_files/test_chr20_ENSG00000042062.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000042062.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000042062.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000042062.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000042062.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000042062.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000042062.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000100983.5 --out compute_weights_out/ENSG00000100983.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000100983.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000042062.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000100983.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000100983.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000100983.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000100983.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000100983.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000100983.5.bed +\n",
      "temp_files/test_chr20_ENSG00000100983.5.bim +\n",
      "temp_files/test_chr20_ENSG00000100983.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000100983.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000100983.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000100983.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000100983.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000100983.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000100983.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000183260.5 --out compute_weights_out/ENSG00000183260.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000183260.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000100983.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000183260.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000183260.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000183260.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000183260.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000183260.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000183260.5.bed +\n",
      "temp_files/test_chr20_ENSG00000183260.5.bim +\n",
      "temp_files/test_chr20_ENSG00000183260.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000183260.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000183260.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000183260.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000183260.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000183260.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000183260.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000089048.9 --out compute_weights_out/ENSG00000089048.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000089048.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000183260.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000089048.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000089048.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000089048.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000089048.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000089048.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000089048.9.bed +\n",
      "temp_files/test_chr20_ENSG00000089048.9.bim +\n",
      "temp_files/test_chr20_ENSG00000089048.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000089048.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000089048.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000089048.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000089048.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000089048.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000089048.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000171703.12 --out compute_weights_out/ENSG00000171703.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000171703.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000089048.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000171703.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000171703.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000171703.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000171703.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000171703.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000171703.12.bed +\n",
      "temp_files/test_chr20_ENSG00000171703.12.bim +\n",
      "temp_files/test_chr20_ENSG00000171703.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000171703.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000171703.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000171703.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000171703.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000171703.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000171703.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101447.8 --out compute_weights_out/ENSG00000101447.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101447.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000171703.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101447.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101447.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101447.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000101447.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101447.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101447.8.bed +\n",
      "temp_files/test_chr20_ENSG00000101447.8.bim +\n",
      "temp_files/test_chr20_ENSG00000101447.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101447.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101447.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101447.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101447.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101447.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101447.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101162.3 --out compute_weights_out/ENSG00000101162.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101162.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101447.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101162.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101162.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101162.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000101162.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101162.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101162.3.bed +\n",
      "temp_files/test_chr20_ENSG00000101162.3.bim +\n",
      "temp_files/test_chr20_ENSG00000101162.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101162.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101162.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101162.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101162.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101162.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101162.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101444.7 --out compute_weights_out/ENSG00000101444.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101444.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101162.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101444.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101444.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101444.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000101444.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101444.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101444.7.bed +\n",
      "temp_files/test_chr20_ENSG00000101444.7.bim +\n",
      "temp_files/test_chr20_ENSG00000101444.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101444.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101444.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101444.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101444.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101444.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101444.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000231133.1 --out compute_weights_out/ENSG00000231133.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000231133.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101444.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000231133.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000231133.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000231133.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000231133.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000231133.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000231133.1.bed +\n",
      "temp_files/test_chr20_ENSG00000231133.1.bim +\n",
      "temp_files/test_chr20_ENSG00000231133.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000231133.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000231133.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000231133.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000231133.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000231133.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000231133.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000213950.2 --out compute_weights_out/ENSG00000213950.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000213950.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000231133.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000213950.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000213950.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000213950.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000213950.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000213950.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000213950.2.bed +\n",
      "temp_files/test_chr20_ENSG00000213950.2.bim +\n",
      "temp_files/test_chr20_ENSG00000213950.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000213950.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000213950.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000213950.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000213950.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000213950.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000213950.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125817.6 --out compute_weights_out/ENSG00000125817.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125817.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000213950.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125817.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125817.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125817.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000125817.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125817.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125817.6.bed +\n",
      "temp_files/test_chr20_ENSG00000125817.6.bim +\n",
      "temp_files/test_chr20_ENSG00000125817.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125817.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125817.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125817.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125817.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125817.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125817.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000130589.11 --out compute_weights_out/ENSG00000130589.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000130589.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125817.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000130589.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000130589.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000130589.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000130589.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000130589.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000130589.11.bed +\n",
      "temp_files/test_chr20_ENSG00000130589.11.bim +\n",
      "temp_files/test_chr20_ENSG00000130589.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000130589.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000130589.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000130589.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000130589.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000130589.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000130589.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000118707.4 --out compute_weights_out/ENSG00000118707.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000118707.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000130589.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000118707.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000118707.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000118707.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000118707.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000118707.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000118707.4.bed +\n",
      "temp_files/test_chr20_ENSG00000118707.4.bim +\n",
      "temp_files/test_chr20_ENSG00000118707.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000118707.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000118707.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000118707.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000118707.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000118707.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000118707.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101194.12 --out compute_weights_out/ENSG00000101194.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101194.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000118707.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101194.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101194.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101194.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000101194.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101194.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101194.12.bed +\n",
      "temp_files/test_chr20_ENSG00000101194.12.bim +\n",
      "temp_files/test_chr20_ENSG00000101194.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101194.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101194.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101194.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101194.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101194.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101194.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000088970.10 --out compute_weights_out/ENSG00000088970.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000088970.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101194.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088970.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000088970.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000088970.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000088970.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088970.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000088970.10.bed +\n",
      "temp_files/test_chr20_ENSG00000088970.10.bim +\n",
      "temp_files/test_chr20_ENSG00000088970.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088970.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000088970.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000088970.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088970.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000088970.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000088970.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000149599.10 --out compute_weights_out/ENSG00000149599.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000149599.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000088970.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149599.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000149599.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000149599.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000149599.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149599.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000149599.10.bed +\n",
      "temp_files/test_chr20_ENSG00000149599.10.bim +\n",
      "temp_files/test_chr20_ENSG00000149599.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149599.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000149599.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000149599.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149599.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000149599.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000149599.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000224497.1 --out compute_weights_out/ENSG00000224497.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000224497.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000149599.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000224497.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000224497.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000224497.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000224497.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000224497.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000224497.1.bed +\n",
      "temp_files/test_chr20_ENSG00000224497.1.bim +\n",
      "temp_files/test_chr20_ENSG00000224497.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000224497.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000224497.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000224497.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000224497.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000224497.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000224497.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000176659.5 --out compute_weights_out/ENSG00000176659.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000176659.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000224497.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000176659.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000176659.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000176659.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000176659.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000176659.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000176659.5.bed +\n",
      "temp_files/test_chr20_ENSG00000176659.5.bim +\n",
      "temp_files/test_chr20_ENSG00000176659.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000176659.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000176659.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000176659.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000176659.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000176659.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000176659.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000226284.1 --out compute_weights_out/ENSG00000226284.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000226284.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000176659.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226284.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000226284.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000226284.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000226284.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226284.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000226284.1.bed +\n",
      "temp_files/test_chr20_ENSG00000226284.1.bim +\n",
      "temp_files/test_chr20_ENSG00000226284.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226284.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000226284.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000226284.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226284.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000226284.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000226284.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000236456.1 --out compute_weights_out/ENSG00000236456.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000236456.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000226284.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000236456.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000236456.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000236456.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000236456.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000236456.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000236456.1.bed +\n",
      "temp_files/test_chr20_ENSG00000236456.1.bim +\n",
      "temp_files/test_chr20_ENSG00000236456.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000236456.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000236456.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000236456.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000236456.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000236456.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000236456.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000100979.10 --out compute_weights_out/ENSG00000100979.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000100979.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000236456.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000100979.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000100979.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000100979.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000100979.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000100979.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000100979.10.bed +\n",
      "temp_files/test_chr20_ENSG00000100979.10.bim +\n",
      "temp_files/test_chr20_ENSG00000100979.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000100979.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000100979.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000100979.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000100979.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000100979.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000100979.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000182463.7 --out compute_weights_out/ENSG00000182463.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000182463.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000100979.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000182463.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000182463.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000182463.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000182463.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000182463.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000182463.7.bed +\n",
      "temp_files/test_chr20_ENSG00000182463.7.bim +\n",
      "temp_files/test_chr20_ENSG00000182463.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000182463.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000182463.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000182463.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000182463.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000182463.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000182463.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000087460.15 --out compute_weights_out/ENSG00000087460.15_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000087460.15.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000182463.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000087460.15.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000087460.15.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000087460.15\n",
      "  --pheno ../data/gene_expressions/ENSG00000087460.15.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000087460.15.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000087460.15.bed +\n",
      "temp_files/test_chr20_ENSG00000087460.15.bim +\n",
      "temp_files/test_chr20_ENSG00000087460.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000087460.15.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000087460.15\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000087460.15\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000087460.15.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000087460.15.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000087460.15.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000203880.6 --out compute_weights_out/ENSG00000203880.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000203880.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000087460.15 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000203880.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000203880.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000203880.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000203880.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000203880.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000203880.6.bed +\n",
      "temp_files/test_chr20_ENSG00000203880.6.bim +\n",
      "temp_files/test_chr20_ENSG00000203880.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000203880.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000203880.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000203880.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000203880.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000203880.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000203880.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124116.14 --out compute_weights_out/ENSG00000124116.14_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124116.14.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000203880.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124116.14.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124116.14.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124116.14\n",
      "  --pheno ../data/gene_expressions/ENSG00000124116.14.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124116.14.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124116.14.bed +\n",
      "temp_files/test_chr20_ENSG00000124116.14.bim +\n",
      "temp_files/test_chr20_ENSG00000124116.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124116.14.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124116.14\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124116.14\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124116.14.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124116.14.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124116.14.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000254419.1 --out compute_weights_out/ENSG00000254419.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000254419.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124116.14 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000254419.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000254419.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000254419.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000254419.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000254419.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000254419.1.bed +\n",
      "temp_files/test_chr20_ENSG00000254419.1.bim +\n",
      "temp_files/test_chr20_ENSG00000254419.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000254419.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000254419.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000254419.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000254419.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000254419.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000254419.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000088305.12 --out compute_weights_out/ENSG00000088305.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000088305.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000254419.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088305.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000088305.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000088305.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000088305.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088305.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000088305.12.bed +\n",
      "temp_files/test_chr20_ENSG00000088305.12.bim +\n",
      "temp_files/test_chr20_ENSG00000088305.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088305.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000088305.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000088305.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088305.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000088305.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000088305.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101104.6 --out compute_weights_out/ENSG00000101104.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101104.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000088305.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101104.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101104.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101104.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000101104.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101104.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101104.6.bed +\n",
      "temp_files/test_chr20_ENSG00000101104.6.bim +\n",
      "temp_files/test_chr20_ENSG00000101104.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101104.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101104.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101104.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101104.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101104.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101104.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000197122.6 --out compute_weights_out/ENSG00000197122.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000197122.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101104.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000197122.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000197122.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000197122.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000197122.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000197122.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000197122.6.bed +\n",
      "temp_files/test_chr20_ENSG00000197122.6.bim +\n",
      "temp_files/test_chr20_ENSG00000197122.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000197122.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000197122.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000197122.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000197122.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000197122.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000197122.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000132825.5 --out compute_weights_out/ENSG00000132825.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000132825.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000197122.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132825.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000132825.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000132825.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000132825.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132825.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000132825.5.bed +\n",
      "temp_files/test_chr20_ENSG00000132825.5.bim +\n",
      "temp_files/test_chr20_ENSG00000132825.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132825.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000132825.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000132825.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132825.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000132825.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000132825.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000203630.3 --out compute_weights_out/ENSG00000203630.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000203630.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000132825.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000203630.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000203630.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000203630.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000203630.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000203630.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000203630.3.bed +\n",
      "temp_files/test_chr20_ENSG00000203630.3.bim +\n",
      "temp_files/test_chr20_ENSG00000203630.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000203630.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000203630.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000203630.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000203630.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000203630.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000203630.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000078804.8 --out compute_weights_out/ENSG00000078804.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000078804.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000203630.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000078804.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000078804.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000078804.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000078804.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000078804.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000078804.8.bed +\n",
      "temp_files/test_chr20_ENSG00000078804.8.bim +\n",
      "temp_files/test_chr20_ENSG00000078804.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000078804.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000078804.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000078804.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000078804.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000078804.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000078804.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000213959.2 --out compute_weights_out/ENSG00000213959.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000213959.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000078804.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000213959.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000213959.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000213959.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000213959.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000213959.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000213959.2.bed +\n",
      "temp_files/test_chr20_ENSG00000213959.2.bim +\n",
      "temp_files/test_chr20_ENSG00000213959.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000213959.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000213959.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000213959.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000213959.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000213959.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000213959.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101413.7 --out compute_weights_out/ENSG00000101413.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101413.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000213959.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101413.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101413.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101413.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000101413.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101413.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101413.7.bed +\n",
      "temp_files/test_chr20_ENSG00000101413.7.bim +\n",
      "temp_files/test_chr20_ENSG00000101413.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101413.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101413.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101413.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101413.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101413.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101413.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000088881.13 --out compute_weights_out/ENSG00000088881.13_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000088881.13.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101413.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088881.13.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000088881.13.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000088881.13\n",
      "  --pheno ../data/gene_expressions/ENSG00000088881.13.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088881.13.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000088881.13.bed +\n",
      "temp_files/test_chr20_ENSG00000088881.13.bim +\n",
      "temp_files/test_chr20_ENSG00000088881.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088881.13.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000088881.13\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000088881.13\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088881.13.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000088881.13.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000088881.13.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124251.6 --out compute_weights_out/ENSG00000124251.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124251.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000088881.13 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124251.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124251.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124251.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000124251.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124251.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124251.6.bed +\n",
      "temp_files/test_chr20_ENSG00000124251.6.bim +\n",
      "temp_files/test_chr20_ENSG00000124251.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124251.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124251.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124251.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124251.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124251.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124251.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000158480.6 --out compute_weights_out/ENSG00000158480.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000158480.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124251.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000158480.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000158480.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000158480.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000158480.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000158480.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000158480.6.bed +\n",
      "temp_files/test_chr20_ENSG00000158480.6.bim +\n",
      "temp_files/test_chr20_ENSG00000158480.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000158480.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000158480.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000158480.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000158480.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000158480.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000158480.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125844.10 --out compute_weights_out/ENSG00000125844.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125844.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000158480.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125844.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125844.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125844.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000125844.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125844.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125844.10.bed +\n",
      "temp_files/test_chr20_ENSG00000125844.10.bim +\n",
      "temp_files/test_chr20_ENSG00000125844.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125844.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125844.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125844.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125844.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125844.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125844.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000088854.11 --out compute_weights_out/ENSG00000088854.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000088854.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125844.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088854.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000088854.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000088854.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000088854.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088854.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000088854.11.bed +\n",
      "temp_files/test_chr20_ENSG00000088854.11.bim +\n",
      "temp_files/test_chr20_ENSG00000088854.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088854.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000088854.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000088854.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088854.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000088854.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000088854.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000232803.1 --out compute_weights_out/ENSG00000232803.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000232803.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000088854.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000232803.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000232803.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000232803.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000232803.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000232803.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000232803.1.bed +\n",
      "temp_files/test_chr20_ENSG00000232803.1.bim +\n",
      "temp_files/test_chr20_ENSG00000232803.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000232803.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000232803.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000232803.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000232803.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000232803.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000232803.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000226144.1 --out compute_weights_out/ENSG00000226144.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000226144.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000232803.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226144.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000226144.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000226144.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000226144.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226144.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000226144.1.bed +\n",
      "temp_files/test_chr20_ENSG00000226144.1.bim +\n",
      "temp_files/test_chr20_ENSG00000226144.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226144.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000226144.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000226144.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226144.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000226144.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000226144.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000243467.1 --out compute_weights_out/ENSG00000243467.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000243467.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000226144.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000243467.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000243467.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000243467.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000243467.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000243467.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000243467.1.bed +\n",
      "temp_files/test_chr20_ENSG00000243467.1.bim +\n",
      "temp_files/test_chr20_ENSG00000243467.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000243467.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000243467.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000243467.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000243467.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000243467.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000243467.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000230753.1 --out compute_weights_out/ENSG00000230753.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000230753.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000243467.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000230753.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000230753.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000230753.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000230753.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000230753.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000230753.1.bed +\n",
      "temp_files/test_chr20_ENSG00000230753.1.bim +\n",
      "temp_files/test_chr20_ENSG00000230753.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000230753.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000230753.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000230753.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000230753.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000230753.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000230753.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000025293.11 --out compute_weights_out/ENSG00000025293.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000025293.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000230753.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000025293.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000025293.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000025293.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000025293.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000025293.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000025293.11.bed +\n",
      "temp_files/test_chr20_ENSG00000025293.11.bim +\n",
      "temp_files/test_chr20_ENSG00000025293.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000025293.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000025293.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000025293.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000025293.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000025293.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000025293.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000205611.3 --out compute_weights_out/ENSG00000205611.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000205611.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000025293.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000205611.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000205611.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000205611.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000205611.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000205611.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000205611.3.bed +\n",
      "temp_files/test_chr20_ENSG00000205611.3.bim +\n",
      "temp_files/test_chr20_ENSG00000205611.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000205611.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000205611.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000205611.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000205611.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000205611.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000205611.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101346.6 --out compute_weights_out/ENSG00000101346.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101346.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000205611.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101346.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101346.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101346.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000101346.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101346.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101346.6.bed +\n",
      "temp_files/test_chr20_ENSG00000101346.6.bim +\n",
      "temp_files/test_chr20_ENSG00000101346.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101346.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101346.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101346.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101346.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101346.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101346.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000259456.1 --out compute_weights_out/ENSG00000259456.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000259456.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101346.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000259456.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000259456.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000259456.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000259456.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000259456.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000259456.1.bed +\n",
      "temp_files/test_chr20_ENSG00000259456.1.bim +\n",
      "temp_files/test_chr20_ENSG00000259456.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000259456.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000259456.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000259456.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000259456.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000259456.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000259456.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000132819.12 --out compute_weights_out/ENSG00000132819.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000132819.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000259456.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132819.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000132819.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000132819.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000132819.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132819.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000132819.12.bed +\n",
      "temp_files/test_chr20_ENSG00000132819.12.bim +\n",
      "temp_files/test_chr20_ENSG00000132819.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132819.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000132819.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000132819.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132819.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000132819.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000132819.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000232043.1 --out compute_weights_out/ENSG00000232043.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000232043.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000132819.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000232043.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000232043.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000232043.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000232043.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000232043.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000232043.1.bed +\n",
      "temp_files/test_chr20_ENSG00000232043.1.bim +\n",
      "temp_files/test_chr20_ENSG00000232043.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000232043.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000232043.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000232043.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000232043.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000232043.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000232043.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000223891.1 --out compute_weights_out/ENSG00000223891.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000223891.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000232043.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000223891.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000223891.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000223891.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000223891.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000223891.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000223891.1.bed +\n",
      "temp_files/test_chr20_ENSG00000223891.1.bim +\n",
      "temp_files/test_chr20_ENSG00000223891.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000223891.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000223891.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000223891.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000223891.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000223891.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000223891.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125812.11 --out compute_weights_out/ENSG00000125812.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125812.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000223891.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125812.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125812.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125812.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000125812.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125812.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125812.11.bed +\n",
      "temp_files/test_chr20_ENSG00000125812.11.bim +\n",
      "temp_files/test_chr20_ENSG00000125812.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125812.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125812.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125812.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125812.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125812.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125812.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000131055.4 --out compute_weights_out/ENSG00000131055.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000131055.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125812.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000131055.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000131055.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000131055.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000131055.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000131055.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000131055.4.bed +\n",
      "temp_files/test_chr20_ENSG00000131055.4.bim +\n",
      "temp_files/test_chr20_ENSG00000131055.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000131055.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000131055.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000131055.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000131055.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000131055.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000131055.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000026036.16 --out compute_weights_out/ENSG00000026036.16_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000026036.16.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000131055.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000026036.16.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000026036.16.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000026036.16\n",
      "  --pheno ../data/gene_expressions/ENSG00000026036.16.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000026036.16.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000026036.16.bed +\n",
      "temp_files/test_chr20_ENSG00000026036.16.bim +\n",
      "temp_files/test_chr20_ENSG00000026036.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000026036.16.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000026036.16\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000026036.16\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000026036.16.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000026036.16.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000026036.16.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125995.11 --out compute_weights_out/ENSG00000125995.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125995.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000026036.16 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125995.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125995.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125995.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000125995.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125995.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125995.11.bed +\n",
      "temp_files/test_chr20_ENSG00000125995.11.bim +\n",
      "temp_files/test_chr20_ENSG00000125995.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125995.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125995.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125995.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125995.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125995.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125995.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101191.11 --out compute_weights_out/ENSG00000101191.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101191.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125995.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101191.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101191.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101191.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000101191.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101191.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101191.11.bed +\n",
      "temp_files/test_chr20_ENSG00000101191.11.bim +\n",
      "temp_files/test_chr20_ENSG00000101191.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101191.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101191.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101191.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101191.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101191.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101191.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000132793.6 --out compute_weights_out/ENSG00000132793.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000132793.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101191.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132793.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000132793.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000132793.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000132793.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132793.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000132793.6.bed +\n",
      "temp_files/test_chr20_ENSG00000132793.6.bim +\n",
      "temp_files/test_chr20_ENSG00000132793.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132793.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000132793.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000132793.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132793.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000132793.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000132793.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125971.12 --out compute_weights_out/ENSG00000125971.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125971.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000132793.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125971.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125971.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125971.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000125971.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125971.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125971.12.bed +\n",
      "temp_files/test_chr20_ENSG00000125971.12.bim +\n",
      "temp_files/test_chr20_ENSG00000125971.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125971.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125971.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125971.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125971.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125971.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125971.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000225865.1 --out compute_weights_out/ENSG00000225865.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000225865.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125971.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225865.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000225865.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000225865.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000225865.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225865.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000225865.1.bed +\n",
      "temp_files/test_chr20_ENSG00000225865.1.bim +\n",
      "temp_files/test_chr20_ENSG00000225865.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225865.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000225865.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000225865.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225865.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000225865.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000225865.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000260257.1 --out compute_weights_out/ENSG00000260257.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000260257.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000225865.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000260257.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000260257.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000260257.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000260257.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000260257.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000260257.1.bed +\n",
      "temp_files/test_chr20_ENSG00000260257.1.bim +\n",
      "temp_files/test_chr20_ENSG00000260257.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000260257.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000260257.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000260257.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000260257.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000260257.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000260257.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000230772.1 --out compute_weights_out/ENSG00000230772.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000230772.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000260257.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000230772.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000230772.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000230772.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000230772.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000230772.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000230772.1.bed +\n",
      "temp_files/test_chr20_ENSG00000230772.1.bim +\n",
      "temp_files/test_chr20_ENSG00000230772.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000230772.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000230772.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000230772.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000230772.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000230772.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000230772.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000247315.2 --out compute_weights_out/ENSG00000247315.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000247315.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000230772.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000247315.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000247315.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000247315.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000247315.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000247315.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000247315.2.bed +\n",
      "temp_files/test_chr20_ENSG00000247315.2.bim +\n",
      "temp_files/test_chr20_ENSG00000247315.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000247315.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000247315.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000247315.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000247315.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000247315.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000247315.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125885.8 --out compute_weights_out/ENSG00000125885.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125885.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000247315.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125885.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125885.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125885.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000125885.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125885.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125885.8.bed +\n",
      "temp_files/test_chr20_ENSG00000125885.8.bim +\n",
      "temp_files/test_chr20_ENSG00000125885.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125885.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125885.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125885.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125885.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125885.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125885.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000235958.1 --out compute_weights_out/ENSG00000235958.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000235958.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125885.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000235958.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000235958.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000235958.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000235958.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000235958.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000235958.1.bed +\n",
      "temp_files/test_chr20_ENSG00000235958.1.bim +\n",
      "temp_files/test_chr20_ENSG00000235958.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000235958.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000235958.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000235958.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000235958.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000235958.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000235958.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125967.11 --out compute_weights_out/ENSG00000125967.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125967.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000235958.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125967.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125967.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125967.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000125967.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125967.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125967.11.bed +\n",
      "temp_files/test_chr20_ENSG00000125967.11.bim +\n",
      "temp_files/test_chr20_ENSG00000125967.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125967.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125967.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125967.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125967.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125967.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125967.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000197586.7 --out compute_weights_out/ENSG00000197586.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000197586.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125967.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000197586.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000197586.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000197586.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000197586.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000197586.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000197586.7.bed +\n",
      "temp_files/test_chr20_ENSG00000197586.7.bim +\n",
      "temp_files/test_chr20_ENSG00000197586.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000197586.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000197586.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000197586.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000197586.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000197586.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000197586.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000149679.6 --out compute_weights_out/ENSG00000149679.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000149679.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000197586.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149679.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000149679.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000149679.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000149679.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149679.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000149679.6.bed +\n",
      "temp_files/test_chr20_ENSG00000149679.6.bim +\n",
      "temp_files/test_chr20_ENSG00000149679.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149679.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000149679.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000149679.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149679.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000149679.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000149679.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000174365.14 --out compute_weights_out/ENSG00000174365.14_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000174365.14.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000149679.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000174365.14.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000174365.14.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000174365.14\n",
      "  --pheno ../data/gene_expressions/ENSG00000174365.14.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000174365.14.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000174365.14.bed +\n",
      "temp_files/test_chr20_ENSG00000174365.14.bim +\n",
      "temp_files/test_chr20_ENSG00000174365.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000174365.14.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000174365.14\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000174365.14\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000174365.14.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000174365.14.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000174365.14.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125534.4 --out compute_weights_out/ENSG00000125534.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125534.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000174365.14 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125534.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125534.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125534.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000125534.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125534.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125534.4.bed +\n",
      "temp_files/test_chr20_ENSG00000125534.4.bim +\n",
      "temp_files/test_chr20_ENSG00000125534.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125534.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125534.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125534.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125534.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125534.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125534.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000241624.1 --out compute_weights_out/ENSG00000241624.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000241624.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125534.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000241624.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000241624.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000241624.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000241624.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000241624.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000241624.1.bed +\n",
      "temp_files/test_chr20_ENSG00000241624.1.bim +\n",
      "temp_files/test_chr20_ENSG00000241624.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000241624.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000241624.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000241624.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000241624.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000241624.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000241624.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000197296.5 --out compute_weights_out/ENSG00000197296.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000197296.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000241624.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000197296.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000197296.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000197296.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000197296.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000197296.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000197296.5.bed +\n",
      "temp_files/test_chr20_ENSG00000197296.5.bim +\n",
      "temp_files/test_chr20_ENSG00000197296.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000197296.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000197296.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000197296.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000197296.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000197296.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000197296.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124171.4 --out compute_weights_out/ENSG00000124171.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124171.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000197296.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124171.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124171.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124171.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000124171.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124171.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124171.4.bed +\n",
      "temp_files/test_chr20_ENSG00000124171.4.bim +\n",
      "temp_files/test_chr20_ENSG00000124171.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124171.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124171.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124171.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124171.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124171.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124171.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125877.6 --out compute_weights_out/ENSG00000125877.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125877.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124171.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125877.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125877.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125877.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000125877.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125877.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125877.6.bed +\n",
      "temp_files/test_chr20_ENSG00000125877.6.bim +\n",
      "temp_files/test_chr20_ENSG00000125877.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125877.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125877.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125877.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125877.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125877.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125877.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000259399.1 --out compute_weights_out/ENSG00000259399.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000259399.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125877.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000259399.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000259399.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000259399.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000259399.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000259399.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000259399.1.bed +\n",
      "temp_files/test_chr20_ENSG00000259399.1.bim +\n",
      "temp_files/test_chr20_ENSG00000259399.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000259399.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000259399.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000259399.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000259399.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000259399.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000259399.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000227343.1 --out compute_weights_out/ENSG00000227343.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000227343.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000259399.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000227343.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000227343.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000227343.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000227343.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000227343.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000227343.1.bed +\n",
      "temp_files/test_chr20_ENSG00000227343.1.bim +\n",
      "temp_files/test_chr20_ENSG00000227343.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000227343.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000227343.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000227343.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000227343.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000227343.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000227343.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125804.8 --out compute_weights_out/ENSG00000125804.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125804.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000227343.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125804.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125804.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125804.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000125804.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125804.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125804.8.bed +\n",
      "temp_files/test_chr20_ENSG00000125804.8.bim +\n",
      "temp_files/test_chr20_ENSG00000125804.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125804.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125804.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125804.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125804.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125804.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125804.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000226405.1 --out compute_weights_out/ENSG00000226405.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000226405.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125804.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226405.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000226405.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000226405.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000226405.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226405.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000226405.1.bed +\n",
      "temp_files/test_chr20_ENSG00000226405.1.bim +\n",
      "temp_files/test_chr20_ENSG00000226405.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226405.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000226405.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000226405.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226405.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000226405.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000226405.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000204070.5 --out compute_weights_out/ENSG00000204070.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000204070.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000226405.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000204070.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000204070.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000204070.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000204070.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000204070.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000204070.5.bed +\n",
      "temp_files/test_chr20_ENSG00000204070.5.bim +\n",
      "temp_files/test_chr20_ENSG00000204070.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000204070.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000204070.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000204070.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000204070.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000204070.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000204070.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000236559.1 --out compute_weights_out/ENSG00000236559.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000236559.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000204070.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000236559.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000236559.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000236559.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000236559.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000236559.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000236559.1.bed +\n",
      "temp_files/test_chr20_ENSG00000236559.1.bim +\n",
      "temp_files/test_chr20_ENSG00000236559.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000236559.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000236559.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000236559.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000236559.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000236559.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000236559.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000092758.10 --out compute_weights_out/ENSG00000092758.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000092758.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000236559.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000092758.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000092758.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000092758.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000092758.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000092758.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000092758.10.bed +\n",
      "temp_files/test_chr20_ENSG00000092758.10.bim +\n",
      "temp_files/test_chr20_ENSG00000092758.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000092758.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000092758.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000092758.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000092758.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000092758.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000092758.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125779.15 --out compute_weights_out/ENSG00000125779.15_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125779.15.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000092758.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125779.15.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125779.15.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125779.15\n",
      "  --pheno ../data/gene_expressions/ENSG00000125779.15.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125779.15.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125779.15.bed +\n",
      "temp_files/test_chr20_ENSG00000125779.15.bim +\n",
      "temp_files/test_chr20_ENSG00000125779.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125779.15.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125779.15\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125779.15\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125779.15.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125779.15.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125779.15.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125898.7 --out compute_weights_out/ENSG00000125898.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125898.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125779.15 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125898.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125898.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125898.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000125898.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125898.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125898.7.bed +\n",
      "temp_files/test_chr20_ENSG00000125898.7.bim +\n",
      "temp_files/test_chr20_ENSG00000125898.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125898.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125898.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125898.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125898.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125898.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125898.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101333.11 --out compute_weights_out/ENSG00000101333.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101333.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125898.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101333.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101333.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101333.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000101333.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101333.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101333.11.bed +\n",
      "temp_files/test_chr20_ENSG00000101333.11.bim +\n",
      "temp_files/test_chr20_ENSG00000101333.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101333.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101333.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101333.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101333.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101333.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101333.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000172216.4 --out compute_weights_out/ENSG00000172216.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000172216.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101333.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000172216.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000172216.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000172216.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000172216.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000172216.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000172216.4.bed +\n",
      "temp_files/test_chr20_ENSG00000172216.4.bim +\n",
      "temp_files/test_chr20_ENSG00000172216.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000172216.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000172216.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000172216.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000172216.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000172216.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000172216.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000225127.1 --out compute_weights_out/ENSG00000225127.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000225127.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000172216.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225127.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000225127.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000225127.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000225127.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225127.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000225127.1.bed +\n",
      "temp_files/test_chr20_ENSG00000225127.1.bim +\n",
      "temp_files/test_chr20_ENSG00000225127.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225127.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000225127.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000225127.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225127.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000225127.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000225127.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000100991.5 --out compute_weights_out/ENSG00000100991.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000100991.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000225127.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000100991.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000100991.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000100991.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000100991.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000100991.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000100991.5.bed +\n",
      "temp_files/test_chr20_ENSG00000100991.5.bim +\n",
      "temp_files/test_chr20_ENSG00000100991.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000100991.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000100991.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000100991.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000100991.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000100991.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000100991.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000240848.1 --out compute_weights_out/ENSG00000240848.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000240848.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000100991.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000240848.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000240848.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000240848.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000240848.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000240848.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000240848.1.bed +\n",
      "temp_files/test_chr20_ENSG00000240848.1.bim +\n",
      "temp_files/test_chr20_ENSG00000240848.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000240848.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000240848.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000240848.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000240848.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000240848.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000240848.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000170191.4 --out compute_weights_out/ENSG00000170191.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000170191.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000240848.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000170191.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000170191.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000170191.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000170191.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000170191.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000170191.4.bed +\n",
      "temp_files/test_chr20_ENSG00000170191.4.bim +\n",
      "temp_files/test_chr20_ENSG00000170191.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000170191.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000170191.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000170191.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000170191.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000170191.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000170191.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124256.9 --out compute_weights_out/ENSG00000124256.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124256.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000170191.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124256.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124256.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124256.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000124256.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124256.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124256.9.bed +\n",
      "temp_files/test_chr20_ENSG00000124256.9.bim +\n",
      "temp_files/test_chr20_ENSG00000124256.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124256.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124256.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124256.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124256.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124256.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124256.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101421.3 --out compute_weights_out/ENSG00000101421.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101421.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124256.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101421.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101421.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101421.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000101421.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101421.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101421.3.bed +\n",
      "temp_files/test_chr20_ENSG00000101421.3.bim +\n",
      "temp_files/test_chr20_ENSG00000101421.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101421.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101421.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101421.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101421.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101421.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101421.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000225377.1 --out compute_weights_out/ENSG00000225377.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000225377.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101421.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225377.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000225377.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000225377.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000225377.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225377.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000225377.1.bed +\n",
      "temp_files/test_chr20_ENSG00000225377.1.bim +\n",
      "temp_files/test_chr20_ENSG00000225377.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225377.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000225377.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000225377.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225377.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000225377.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000225377.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000234011.2 --out compute_weights_out/ENSG00000234011.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000234011.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000225377.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000234011.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000234011.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000234011.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000234011.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000234011.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000234011.2.bed +\n",
      "temp_files/test_chr20_ENSG00000234011.2.bim +\n",
      "temp_files/test_chr20_ENSG00000234011.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000234011.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000234011.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000234011.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000234011.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000234011.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000234011.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000232388.1 --out compute_weights_out/ENSG00000232388.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000232388.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000234011.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000232388.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000232388.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000232388.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000232388.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000232388.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000232388.1.bed +\n",
      "temp_files/test_chr20_ENSG00000232388.1.bim +\n",
      "temp_files/test_chr20_ENSG00000232388.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000232388.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000232388.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000232388.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000232388.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000232388.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000232388.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000132824.8 --out compute_weights_out/ENSG00000132824.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000132824.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000232388.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132824.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000132824.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000132824.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000132824.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132824.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000132824.8.bed +\n",
      "temp_files/test_chr20_ENSG00000132824.8.bim +\n",
      "temp_files/test_chr20_ENSG00000132824.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132824.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000132824.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000132824.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132824.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000132824.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000132824.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000089091.10 --out compute_weights_out/ENSG00000089091.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000089091.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000132824.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000089091.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000089091.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000089091.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000089091.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000089091.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000089091.10.bed +\n",
      "temp_files/test_chr20_ENSG00000089091.10.bim +\n",
      "temp_files/test_chr20_ENSG00000089091.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000089091.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000089091.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000089091.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000089091.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000089091.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000089091.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000088812.12 --out compute_weights_out/ENSG00000088812.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000088812.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000089091.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088812.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000088812.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000088812.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000088812.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088812.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000088812.12.bed +\n",
      "temp_files/test_chr20_ENSG00000088812.12.bim +\n",
      "temp_files/test_chr20_ENSG00000088812.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088812.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000088812.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000088812.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088812.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000088812.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000088812.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000149654.5 --out compute_weights_out/ENSG00000149654.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000149654.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000088812.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149654.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000149654.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000149654.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000149654.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149654.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000149654.5.bed +\n",
      "temp_files/test_chr20_ENSG00000149654.5.bim +\n",
      "temp_files/test_chr20_ENSG00000149654.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149654.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000149654.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000149654.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149654.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000149654.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000149654.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000088766.6 --out compute_weights_out/ENSG00000088766.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000088766.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000149654.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088766.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000088766.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000088766.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000088766.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088766.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000088766.6.bed +\n",
      "temp_files/test_chr20_ENSG00000088766.6.bim +\n",
      "temp_files/test_chr20_ENSG00000088766.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088766.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000088766.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000088766.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088766.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000088766.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000088766.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101189.5 --out compute_weights_out/ENSG00000101189.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101189.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000088766.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101189.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101189.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101189.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000101189.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101189.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101189.5.bed +\n",
      "temp_files/test_chr20_ENSG00000101189.5.bim +\n",
      "temp_files/test_chr20_ENSG00000101189.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101189.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101189.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101189.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101189.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101189.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101189.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000225915.2 --out compute_weights_out/ENSG00000225915.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000225915.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101189.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225915.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000225915.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000225915.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000225915.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225915.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000225915.2.bed +\n",
      "temp_files/test_chr20_ENSG00000225915.2.bim +\n",
      "temp_files/test_chr20_ENSG00000225915.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225915.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000225915.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000225915.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225915.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000225915.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000225915.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000243952.1 --out compute_weights_out/ENSG00000243952.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000243952.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000225915.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000243952.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000243952.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000243952.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000243952.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000243952.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000243952.1.bed +\n",
      "temp_files/test_chr20_ENSG00000243952.1.bim +\n",
      "temp_files/test_chr20_ENSG00000243952.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000243952.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000243952.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000243952.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000243952.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000243952.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000243952.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000178726.5 --out compute_weights_out/ENSG00000178726.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000178726.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000243952.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000178726.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000178726.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000178726.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000178726.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000178726.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000178726.5.bed +\n",
      "temp_files/test_chr20_ENSG00000178726.5.bim +\n",
      "temp_files/test_chr20_ENSG00000178726.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000178726.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000178726.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000178726.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000178726.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000178726.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000178726.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101160.9 --out compute_weights_out/ENSG00000101160.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101160.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000178726.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101160.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101160.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101160.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000101160.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101160.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101160.9.bed +\n",
      "temp_files/test_chr20_ENSG00000101160.9.bim +\n",
      "temp_files/test_chr20_ENSG00000101160.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101160.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101160.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101160.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101160.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101160.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101160.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000132670.14 --out compute_weights_out/ENSG00000132670.14_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000132670.14.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101160.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132670.14.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000132670.14.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000132670.14\n",
      "  --pheno ../data/gene_expressions/ENSG00000132670.14.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132670.14.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000132670.14.bed +\n",
      "temp_files/test_chr20_ENSG00000132670.14.bim +\n",
      "temp_files/test_chr20_ENSG00000132670.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132670.14.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000132670.14\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000132670.14\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132670.14.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000132670.14.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000132670.14.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000229853.1 --out compute_weights_out/ENSG00000229853.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000229853.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000132670.14 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000229853.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000229853.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000229853.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000229853.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000229853.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000229853.1.bed +\n",
      "temp_files/test_chr20_ENSG00000229853.1.bim +\n",
      "temp_files/test_chr20_ENSG00000229853.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000229853.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000229853.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000229853.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000229853.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000229853.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000229853.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101247.12 --out compute_weights_out/ENSG00000101247.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101247.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000229853.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101247.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101247.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101247.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000101247.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101247.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101247.12.bed +\n",
      "temp_files/test_chr20_ENSG00000101247.12.bim +\n",
      "temp_files/test_chr20_ENSG00000101247.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101247.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101247.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101247.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101247.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101247.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101247.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000238151.1 --out compute_weights_out/ENSG00000238151.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000238151.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101247.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000238151.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000238151.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000238151.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000238151.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000238151.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000238151.1.bed +\n",
      "temp_files/test_chr20_ENSG00000238151.1.bim +\n",
      "temp_files/test_chr20_ENSG00000238151.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000238151.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000238151.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000238151.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000238151.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000238151.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000238151.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000226465.1 --out compute_weights_out/ENSG00000226465.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000226465.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000238151.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226465.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000226465.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000226465.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000226465.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226465.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000226465.1.bed +\n",
      "temp_files/test_chr20_ENSG00000226465.1.bim +\n",
      "temp_files/test_chr20_ENSG00000226465.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226465.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000226465.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000226465.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226465.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000226465.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000226465.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101306.6 --out compute_weights_out/ENSG00000101306.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101306.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000226465.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101306.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101306.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101306.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000101306.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101306.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101306.6.bed +\n",
      "temp_files/test_chr20_ENSG00000101306.6.bim +\n",
      "temp_files/test_chr20_ENSG00000101306.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101306.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101306.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101306.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101306.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101306.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101306.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125875.8 --out compute_weights_out/ENSG00000125875.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125875.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101306.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125875.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125875.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125875.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000125875.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125875.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125875.8.bed +\n",
      "temp_files/test_chr20_ENSG00000125875.8.bim +\n",
      "temp_files/test_chr20_ENSG00000125875.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125875.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125875.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125875.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125875.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125875.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125875.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101445.4 --out compute_weights_out/ENSG00000101445.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101445.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125875.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101445.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101445.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101445.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000101445.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101445.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101445.4.bed +\n",
      "temp_files/test_chr20_ENSG00000101445.4.bim +\n",
      "temp_files/test_chr20_ENSG00000101445.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101445.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101445.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101445.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101445.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101445.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101445.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000172264.12 --out compute_weights_out/ENSG00000172264.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000172264.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101445.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000172264.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000172264.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000172264.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000172264.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000172264.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000172264.12.bed +\n",
      "temp_files/test_chr20_ENSG00000172264.12.bim +\n",
      "temp_files/test_chr20_ENSG00000172264.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000172264.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000172264.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000172264.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000172264.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000172264.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000172264.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125968.7 --out compute_weights_out/ENSG00000125968.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125968.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000172264.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125968.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125968.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125968.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000125968.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125968.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125968.7.bed +\n",
      "temp_files/test_chr20_ENSG00000125968.7.bim +\n",
      "temp_files/test_chr20_ENSG00000125968.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125968.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125968.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125968.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125968.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125968.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125968.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101457.8 --out compute_weights_out/ENSG00000101457.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101457.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125968.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101457.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101457.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101457.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000101457.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101457.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101457.8.bed +\n",
      "temp_files/test_chr20_ENSG00000101457.8.bim +\n",
      "temp_files/test_chr20_ENSG00000101457.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101457.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101457.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101457.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101457.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101457.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101457.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000231878.1 --out compute_weights_out/ENSG00000231878.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000231878.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101457.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000231878.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000231878.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000231878.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000231878.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000231878.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000231878.1.bed +\n",
      "temp_files/test_chr20_ENSG00000231878.1.bim +\n",
      "temp_files/test_chr20_ENSG00000231878.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000231878.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000231878.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000231878.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000231878.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000231878.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000231878.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101464.6 --out compute_weights_out/ENSG00000101464.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101464.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000231878.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101464.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101464.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101464.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000101464.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101464.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101464.6.bed +\n",
      "temp_files/test_chr20_ENSG00000101464.6.bim +\n",
      "temp_files/test_chr20_ENSG00000101464.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101464.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101464.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101464.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101464.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101464.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101464.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000259729.1 --out compute_weights_out/ENSG00000259729.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000259729.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101464.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000259729.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000259729.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000259729.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000259729.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000259729.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000259729.1.bed +\n",
      "temp_files/test_chr20_ENSG00000259729.1.bim +\n",
      "temp_files/test_chr20_ENSG00000259729.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000259729.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000259729.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000259729.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000259729.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000259729.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000259729.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101443.13 --out compute_weights_out/ENSG00000101443.13_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101443.13.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000259729.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101443.13.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101443.13.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101443.13\n",
      "  --pheno ../data/gene_expressions/ENSG00000101443.13.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101443.13.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101443.13.bed +\n",
      "temp_files/test_chr20_ENSG00000101443.13.bim +\n",
      "temp_files/test_chr20_ENSG00000101443.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101443.13.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101443.13\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101443.13\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101443.13.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101443.13.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101443.13.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000214185.3 --out compute_weights_out/ENSG00000214185.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000214185.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101443.13 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000214185.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000214185.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000214185.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000214185.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000214185.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000214185.3.bed +\n",
      "temp_files/test_chr20_ENSG00000214185.3.bim +\n",
      "temp_files/test_chr20_ENSG00000214185.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000214185.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000214185.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000214185.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000214185.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000214185.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000214185.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000132669.8 --out compute_weights_out/ENSG00000132669.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000132669.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000214185.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132669.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000132669.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000132669.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000132669.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132669.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000132669.8.bed +\n",
      "temp_files/test_chr20_ENSG00000132669.8.bim +\n",
      "temp_files/test_chr20_ENSG00000132669.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132669.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000132669.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000132669.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132669.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000132669.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000132669.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000235313.1 --out compute_weights_out/ENSG00000235313.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000235313.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000132669.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000235313.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000235313.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000235313.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000235313.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000235313.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000235313.1.bed +\n",
      "temp_files/test_chr20_ENSG00000235313.1.bim +\n",
      "temp_files/test_chr20_ENSG00000235313.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000235313.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000235313.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000235313.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000235313.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000235313.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000235313.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000198276.8 --out compute_weights_out/ENSG00000198276.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000198276.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000235313.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000198276.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000198276.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000198276.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000198276.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000198276.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000198276.8.bed +\n",
      "temp_files/test_chr20_ENSG00000198276.8.bim +\n",
      "temp_files/test_chr20_ENSG00000198276.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000198276.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000198276.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000198276.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000198276.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000198276.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000198276.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000225336.2 --out compute_weights_out/ENSG00000225336.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000225336.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000198276.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225336.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000225336.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000225336.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000225336.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225336.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000225336.2.bed +\n",
      "temp_files/test_chr20_ENSG00000225336.2.bim +\n",
      "temp_files/test_chr20_ENSG00000225336.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225336.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000225336.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000225336.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225336.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000225336.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000225336.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000089199.5 --out compute_weights_out/ENSG00000089199.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000089199.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000225336.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000089199.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000089199.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000089199.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000089199.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000089199.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000089199.5.bed +\n",
      "temp_files/test_chr20_ENSG00000089199.5.bim +\n",
      "temp_files/test_chr20_ENSG00000089199.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000089199.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000089199.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000089199.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000089199.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000089199.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000089199.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124134.4 --out compute_weights_out/ENSG00000124134.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124134.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000089199.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124134.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124134.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124134.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000124134.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124134.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124134.4.bed +\n",
      "temp_files/test_chr20_ENSG00000124134.4.bim +\n",
      "temp_files/test_chr20_ENSG00000124134.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124134.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124134.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124134.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124134.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124134.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124134.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000177764.6 --out compute_weights_out/ENSG00000177764.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000177764.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124134.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000177764.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000177764.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000177764.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000177764.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000177764.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000177764.6.bed +\n",
      "temp_files/test_chr20_ENSG00000177764.6.bim +\n",
      "temp_files/test_chr20_ENSG00000177764.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000177764.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000177764.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000177764.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000177764.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000177764.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000177764.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000250090.1 --out compute_weights_out/ENSG00000250090.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000250090.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000177764.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000250090.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000250090.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000250090.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000250090.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000250090.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000250090.1.bed +\n",
      "temp_files/test_chr20_ENSG00000250090.1.bim +\n",
      "temp_files/test_chr20_ENSG00000250090.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000250090.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000250090.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000250090.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000250090.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000250090.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000250090.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000171552.8 --out compute_weights_out/ENSG00000171552.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000171552.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000250090.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000171552.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000171552.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000171552.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000171552.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000171552.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000171552.8.bed +\n",
      "temp_files/test_chr20_ENSG00000171552.8.bim +\n",
      "temp_files/test_chr20_ENSG00000171552.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000171552.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000171552.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000171552.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000171552.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000171552.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000171552.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000126005.10 --out compute_weights_out/ENSG00000126005.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000126005.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000171552.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000126005.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000126005.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000126005.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000126005.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000126005.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000126005.10.bed +\n",
      "temp_files/test_chr20_ENSG00000126005.10.bim +\n",
      "temp_files/test_chr20_ENSG00000126005.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000126005.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000126005.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000126005.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000126005.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000126005.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000126005.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000171940.8 --out compute_weights_out/ENSG00000171940.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000171940.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000126005.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000171940.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000171940.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000171940.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000171940.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000171940.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000171940.8.bed +\n",
      "temp_files/test_chr20_ENSG00000171940.8.bim +\n",
      "temp_files/test_chr20_ENSG00000171940.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000171940.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000171940.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000171940.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000171940.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000171940.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000171940.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124092.7 --out compute_weights_out/ENSG00000124092.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124092.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000171940.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124092.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124092.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124092.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000124092.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124092.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124092.7.bed +\n",
      "temp_files/test_chr20_ENSG00000124092.7.bim +\n",
      "temp_files/test_chr20_ENSG00000124092.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124092.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124092.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124092.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124092.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124092.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124092.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101350.6 --out compute_weights_out/ENSG00000101350.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101350.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124092.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101350.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101350.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101350.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000101350.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101350.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101350.6.bed +\n",
      "temp_files/test_chr20_ENSG00000101350.6.bim +\n",
      "temp_files/test_chr20_ENSG00000101350.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101350.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101350.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101350.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101350.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101350.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101350.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000026559.8 --out compute_weights_out/ENSG00000026559.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000026559.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101350.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000026559.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000026559.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000026559.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000026559.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000026559.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000026559.8.bed +\n",
      "temp_files/test_chr20_ENSG00000026559.8.bim +\n",
      "temp_files/test_chr20_ENSG00000026559.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000026559.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000026559.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000026559.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000026559.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000026559.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000026559.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000227401.1 --out compute_weights_out/ENSG00000227401.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000227401.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000026559.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000227401.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000227401.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000227401.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000227401.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000227401.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000227401.1.bed +\n",
      "temp_files/test_chr20_ENSG00000227401.1.bim +\n",
      "temp_files/test_chr20_ENSG00000227401.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000227401.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000227401.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000227401.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000227401.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000227401.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000227401.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000131044.11 --out compute_weights_out/ENSG00000131044.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000131044.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000227401.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000131044.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000131044.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000131044.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000131044.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000131044.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000131044.11.bed +\n",
      "temp_files/test_chr20_ENSG00000131044.11.bim +\n",
      "temp_files/test_chr20_ENSG00000131044.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000131044.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000131044.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000131044.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000131044.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000131044.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000131044.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000132622.6 --out compute_weights_out/ENSG00000132622.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000132622.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000131044.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132622.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000132622.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000132622.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000132622.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132622.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000132622.6.bed +\n",
      "temp_files/test_chr20_ENSG00000132622.6.bim +\n",
      "temp_files/test_chr20_ENSG00000132622.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132622.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000132622.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000132622.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132622.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000132622.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000132622.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000235590.2 --out compute_weights_out/ENSG00000235590.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000235590.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000132622.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000235590.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000235590.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000235590.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000235590.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000235590.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000235590.2.bed +\n",
      "temp_files/test_chr20_ENSG00000235590.2.bim +\n",
      "temp_files/test_chr20_ENSG00000235590.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000235590.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000235590.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000235590.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000235590.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000235590.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000235590.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000064601.11 --out compute_weights_out/ENSG00000064601.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000064601.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000235590.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000064601.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000064601.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000064601.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000064601.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000064601.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000064601.11.bed +\n",
      "temp_files/test_chr20_ENSG00000064601.11.bim +\n",
      "temp_files/test_chr20_ENSG00000064601.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000064601.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000064601.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000064601.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000064601.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000064601.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000064601.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125827.4 --out compute_weights_out/ENSG00000125827.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125827.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000064601.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125827.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125827.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125827.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000125827.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125827.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125827.4.bed +\n",
      "temp_files/test_chr20_ENSG00000125827.4.bim +\n",
      "temp_files/test_chr20_ENSG00000125827.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125827.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125827.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125827.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125827.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125827.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125827.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000224635.1 --out compute_weights_out/ENSG00000224635.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000224635.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125827.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000224635.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000224635.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000224635.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000224635.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000224635.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000224635.1.bed +\n",
      "temp_files/test_chr20_ENSG00000224635.1.bim +\n",
      "temp_files/test_chr20_ENSG00000224635.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000224635.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000224635.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000224635.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000224635.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000224635.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000224635.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000131069.13 --out compute_weights_out/ENSG00000131069.13_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000131069.13.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000224635.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000131069.13.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000131069.13.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000131069.13\n",
      "  --pheno ../data/gene_expressions/ENSG00000131069.13.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000131069.13.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000131069.13.bed +\n",
      "temp_files/test_chr20_ENSG00000131069.13.bim +\n",
      "temp_files/test_chr20_ENSG00000131069.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000131069.13.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000131069.13\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000131069.13\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000131069.13.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000131069.13.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000131069.13.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000224452.1 --out compute_weights_out/ENSG00000224452.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000224452.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000131069.13 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000224452.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000224452.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000224452.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000224452.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000224452.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000224452.1.bed +\n",
      "temp_files/test_chr20_ENSG00000224452.1.bim +\n",
      "temp_files/test_chr20_ENSG00000224452.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000224452.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000224452.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000224452.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000224452.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000224452.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000224452.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125843.6 --out compute_weights_out/ENSG00000125843.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125843.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000224452.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125843.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125843.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125843.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000125843.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125843.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125843.6.bed +\n",
      "temp_files/test_chr20_ENSG00000125843.6.bim +\n",
      "temp_files/test_chr20_ENSG00000125843.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125843.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125843.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125843.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125843.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125843.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125843.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101182.10 --out compute_weights_out/ENSG00000101182.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101182.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125843.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101182.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101182.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101182.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000101182.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101182.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101182.10.bed +\n",
      "temp_files/test_chr20_ENSG00000101182.10.bim +\n",
      "temp_files/test_chr20_ENSG00000101182.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101182.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101182.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101182.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101182.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101182.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101182.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124207.10 --out compute_weights_out/ENSG00000124207.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124207.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101182.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124207.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124207.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124207.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000124207.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124207.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124207.10.bed +\n",
      "temp_files/test_chr20_ENSG00000124207.10.bim +\n",
      "temp_files/test_chr20_ENSG00000124207.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124207.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124207.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124207.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124207.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124207.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124207.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124257.6 --out compute_weights_out/ENSG00000124257.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124257.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124207.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124257.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124257.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124257.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000124257.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124257.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124257.6.bed +\n",
      "temp_files/test_chr20_ENSG00000124257.6.bim +\n",
      "temp_files/test_chr20_ENSG00000124257.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124257.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124257.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124257.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124257.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124257.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124257.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000088833.11 --out compute_weights_out/ENSG00000088833.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000088833.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124257.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088833.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000088833.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000088833.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000088833.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088833.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000088833.11.bed +\n",
      "temp_files/test_chr20_ENSG00000088833.11.bim +\n",
      "temp_files/test_chr20_ENSG00000088833.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088833.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000088833.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000088833.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088833.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000088833.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000088833.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000132832.4 --out compute_weights_out/ENSG00000132832.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000132832.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000088833.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132832.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000132832.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000132832.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000132832.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132832.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000132832.4.bed +\n",
      "temp_files/test_chr20_ENSG00000132832.4.bim +\n",
      "temp_files/test_chr20_ENSG00000132832.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132832.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000132832.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000132832.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132832.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000132832.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000132832.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000260032.1 --out compute_weights_out/ENSG00000260032.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000260032.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000132832.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000260032.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000260032.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000260032.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000260032.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000260032.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000260032.1.bed +\n",
      "temp_files/test_chr20_ENSG00000260032.1.bim +\n",
      "temp_files/test_chr20_ENSG00000260032.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000260032.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000260032.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000260032.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000260032.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000260032.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000260032.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000088298.7 --out compute_weights_out/ENSG00000088298.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000088298.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000260032.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088298.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000088298.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000088298.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000088298.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088298.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000088298.7.bed +\n",
      "temp_files/test_chr20_ENSG00000088298.7.bim +\n",
      "temp_files/test_chr20_ENSG00000088298.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088298.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000088298.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000088298.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088298.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000088298.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000088298.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101057.11 --out compute_weights_out/ENSG00000101057.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101057.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000088298.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101057.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101057.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101057.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000101057.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101057.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101057.11.bed +\n",
      "temp_files/test_chr20_ENSG00000101057.11.bim +\n",
      "temp_files/test_chr20_ENSG00000101057.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101057.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101057.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101057.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101057.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101057.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101057.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101442.8 --out compute_weights_out/ENSG00000101442.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101442.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101057.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101442.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101442.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101442.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000101442.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101442.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101442.8.bed +\n",
      "temp_files/test_chr20_ENSG00000101442.8.bim +\n",
      "temp_files/test_chr20_ENSG00000101442.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101442.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101442.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101442.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101442.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101442.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101442.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125878.4 --out compute_weights_out/ENSG00000125878.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125878.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101442.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125878.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125878.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125878.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000125878.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125878.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125878.4.bed +\n",
      "temp_files/test_chr20_ENSG00000125878.4.bim +\n",
      "temp_files/test_chr20_ENSG00000125878.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125878.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125878.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125878.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125878.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125878.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125878.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101150.11 --out compute_weights_out/ENSG00000101150.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101150.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125878.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101150.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101150.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101150.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000101150.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101150.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101150.11.bed +\n",
      "temp_files/test_chr20_ENSG00000101150.11.bim +\n",
      "temp_files/test_chr20_ENSG00000101150.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101150.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101150.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101150.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101150.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101150.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101150.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000131067.12 --out compute_weights_out/ENSG00000131067.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000131067.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101150.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000131067.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000131067.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000131067.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000131067.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000131067.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000131067.12.bed +\n",
      "temp_files/test_chr20_ENSG00000131067.12.bim +\n",
      "temp_files/test_chr20_ENSG00000131067.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000131067.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000131067.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000131067.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000131067.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000131067.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000131067.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000132661.3 --out compute_weights_out/ENSG00000132661.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000132661.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000131067.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132661.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000132661.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000132661.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000132661.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132661.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000132661.3.bed +\n",
      "temp_files/test_chr20_ENSG00000132661.3.bim +\n",
      "temp_files/test_chr20_ENSG00000132661.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132661.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000132661.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000132661.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132661.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000132661.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000132661.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124222.15 --out compute_weights_out/ENSG00000124222.15_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124222.15.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000132661.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124222.15.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124222.15.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124222.15\n",
      "  --pheno ../data/gene_expressions/ENSG00000124222.15.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124222.15.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124222.15.bed +\n",
      "temp_files/test_chr20_ENSG00000124222.15.bim +\n",
      "temp_files/test_chr20_ENSG00000124222.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124222.15.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124222.15\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124222.15\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124222.15.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124222.15.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124222.15.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000232907.2 --out compute_weights_out/ENSG00000232907.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000232907.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124222.15 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000232907.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000232907.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000232907.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000232907.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000232907.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000232907.2.bed +\n",
      "temp_files/test_chr20_ENSG00000232907.2.bim +\n",
      "temp_files/test_chr20_ENSG00000232907.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000232907.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000232907.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000232907.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000232907.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000232907.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000232907.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101336.8 --out compute_weights_out/ENSG00000101336.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101336.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000232907.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101336.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101336.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101336.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000101336.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101336.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101336.8.bed +\n",
      "temp_files/test_chr20_ENSG00000101336.8.bim +\n",
      "temp_files/test_chr20_ENSG00000101336.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101336.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101336.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101336.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101336.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101336.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101336.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000077984.4 --out compute_weights_out/ENSG00000077984.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000077984.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101336.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000077984.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000077984.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000077984.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000077984.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000077984.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000077984.4.bed +\n",
      "temp_files/test_chr20_ENSG00000077984.4.bim +\n",
      "temp_files/test_chr20_ENSG00000077984.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000077984.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000077984.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000077984.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000077984.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000077984.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000077984.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000197670.5 --out compute_weights_out/ENSG00000197670.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000197670.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000077984.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000197670.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000197670.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000197670.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000197670.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000197670.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000197670.5.bed +\n",
      "temp_files/test_chr20_ENSG00000197670.5.bim +\n",
      "temp_files/test_chr20_ENSG00000197670.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000197670.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000197670.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000197670.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000197670.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000197670.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000197670.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000227688.2 --out compute_weights_out/ENSG00000227688.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000227688.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000197670.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000227688.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000227688.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000227688.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000227688.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000227688.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000227688.2.bed +\n",
      "temp_files/test_chr20_ENSG00000227688.2.bim +\n",
      "temp_files/test_chr20_ENSG00000227688.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000227688.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000227688.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000227688.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000227688.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000227688.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000227688.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000229539.1 --out compute_weights_out/ENSG00000229539.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000229539.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000227688.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000229539.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000229539.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000229539.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000229539.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000229539.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000229539.1.bed +\n",
      "temp_files/test_chr20_ENSG00000229539.1.bim +\n",
      "temp_files/test_chr20_ENSG00000229539.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000229539.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000229539.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000229539.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000229539.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000229539.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000229539.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000225419.1 --out compute_weights_out/ENSG00000225419.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000225419.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000229539.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225419.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000225419.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000225419.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000225419.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225419.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000225419.1.bed +\n",
      "temp_files/test_chr20_ENSG00000225419.1.bim +\n",
      "temp_files/test_chr20_ENSG00000225419.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225419.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000225419.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000225419.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225419.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000225419.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000225419.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000196074.7 --out compute_weights_out/ENSG00000196074.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000196074.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000225419.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000196074.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000196074.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000196074.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000196074.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000196074.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000196074.7.bed +\n",
      "temp_files/test_chr20_ENSG00000196074.7.bim +\n",
      "temp_files/test_chr20_ENSG00000196074.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000196074.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000196074.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000196074.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000196074.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000196074.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000196074.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000149657.13 --out compute_weights_out/ENSG00000149657.13_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000149657.13.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000196074.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149657.13.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000149657.13.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000149657.13\n",
      "  --pheno ../data/gene_expressions/ENSG00000149657.13.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149657.13.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000149657.13.bed +\n",
      "temp_files/test_chr20_ENSG00000149657.13.bim +\n",
      "temp_files/test_chr20_ENSG00000149657.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149657.13.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000149657.13\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000149657.13\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149657.13.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000149657.13.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000149657.13.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000232838.2 --out compute_weights_out/ENSG00000232838.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000232838.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000149657.13 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000232838.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000232838.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000232838.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000232838.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000232838.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000232838.2.bed +\n",
      "temp_files/test_chr20_ENSG00000232838.2.bim +\n",
      "temp_files/test_chr20_ENSG00000232838.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000232838.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000232838.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000232838.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000232838.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000232838.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000232838.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000196700.3 --out compute_weights_out/ENSG00000196700.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000196700.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000232838.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000196700.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000196700.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000196700.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000196700.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000196700.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000196700.3.bed +\n",
      "temp_files/test_chr20_ENSG00000196700.3.bim +\n",
      "temp_files/test_chr20_ENSG00000196700.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000196700.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000196700.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000196700.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000196700.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000196700.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000196700.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000229728.1 --out compute_weights_out/ENSG00000229728.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000229728.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000196700.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000229728.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000229728.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000229728.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000229728.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000229728.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000229728.1.bed +\n",
      "temp_files/test_chr20_ENSG00000229728.1.bim +\n",
      "temp_files/test_chr20_ENSG00000229728.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000229728.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000229728.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000229728.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000229728.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000229728.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000229728.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124143.6 --out compute_weights_out/ENSG00000124143.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124143.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000229728.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124143.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124143.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124143.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000124143.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124143.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124143.6.bed +\n",
      "temp_files/test_chr20_ENSG00000124143.6.bim +\n",
      "temp_files/test_chr20_ENSG00000124143.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124143.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124143.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124143.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124143.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124143.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124143.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000227477.1 --out compute_weights_out/ENSG00000227477.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000227477.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124143.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000227477.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000227477.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000227477.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000227477.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000227477.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000227477.1.bed +\n",
      "temp_files/test_chr20_ENSG00000227477.1.bim +\n",
      "temp_files/test_chr20_ENSG00000227477.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000227477.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000227477.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000227477.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000227477.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000227477.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000227477.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124104.13 --out compute_weights_out/ENSG00000124104.13_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124104.13.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000227477.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124104.13.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124104.13.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124104.13\n",
      "  --pheno ../data/gene_expressions/ENSG00000124104.13.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124104.13.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124104.13.bed +\n",
      "temp_files/test_chr20_ENSG00000124104.13.bim +\n",
      "temp_files/test_chr20_ENSG00000124104.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124104.13.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124104.13\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124104.13\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124104.13.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124104.13.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124104.13.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000185019.12 --out compute_weights_out/ENSG00000185019.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000185019.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124104.13 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000185019.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000185019.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000185019.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000185019.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000185019.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000185019.12.bed +\n",
      "temp_files/test_chr20_ENSG00000185019.12.bim +\n",
      "temp_files/test_chr20_ENSG00000185019.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000185019.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000185019.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000185019.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000185019.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000185019.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000185019.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000223651.1 --out compute_weights_out/ENSG00000223651.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000223651.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000185019.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000223651.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000223651.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000223651.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000223651.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000223651.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000223651.1.bed +\n",
      "temp_files/test_chr20_ENSG00000223651.1.bim +\n",
      "temp_files/test_chr20_ENSG00000223651.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000223651.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000223651.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000223651.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000223651.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000223651.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000223651.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000224565.1 --out compute_weights_out/ENSG00000224565.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000224565.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000223651.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000224565.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000224565.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000224565.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000224565.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000224565.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000224565.1.bed +\n",
      "temp_files/test_chr20_ENSG00000224565.1.bim +\n",
      "temp_files/test_chr20_ENSG00000224565.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000224565.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000224565.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000224565.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000224565.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000224565.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000224565.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000228340.1 --out compute_weights_out/ENSG00000228340.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000228340.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000224565.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000228340.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000228340.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000228340.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000228340.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000228340.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000228340.1.bed +\n",
      "temp_files/test_chr20_ENSG00000228340.1.bim +\n",
      "temp_files/test_chr20_ENSG00000228340.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000228340.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000228340.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000228340.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000228340.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000228340.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000228340.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000234933.1 --out compute_weights_out/ENSG00000234933.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000234933.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000228340.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000234933.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000234933.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000234933.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000234933.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000234933.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000234933.1.bed +\n",
      "temp_files/test_chr20_ENSG00000234933.1.bim +\n",
      "temp_files/test_chr20_ENSG00000234933.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000234933.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000234933.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000234933.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000234933.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000234933.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000234933.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000130590.9 --out compute_weights_out/ENSG00000130590.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000130590.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000234933.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000130590.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000130590.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000130590.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000130590.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000130590.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000130590.9.bed +\n",
      "temp_files/test_chr20_ENSG00000130590.9.bim +\n",
      "temp_files/test_chr20_ENSG00000130590.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000130590.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000130590.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000130590.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000130590.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000130590.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000130590.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000130702.9 --out compute_weights_out/ENSG00000130702.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000130702.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000130590.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000130702.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000130702.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000130702.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000130702.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000130702.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000130702.9.bed +\n",
      "temp_files/test_chr20_ENSG00000130702.9.bim +\n",
      "temp_files/test_chr20_ENSG00000130702.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000130702.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000130702.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000130702.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000130702.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000130702.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000130702.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000226243.1 --out compute_weights_out/ENSG00000226243.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000226243.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000130702.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226243.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000226243.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000226243.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000226243.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226243.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000226243.1.bed +\n",
      "temp_files/test_chr20_ENSG00000226243.1.bim +\n",
      "temp_files/test_chr20_ENSG00000226243.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226243.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000226243.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000226243.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226243.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000226243.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000226243.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000196476.7 --out compute_weights_out/ENSG00000196476.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000196476.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000226243.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000196476.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000196476.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000196476.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000196476.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000196476.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000196476.7.bed +\n",
      "temp_files/test_chr20_ENSG00000196476.7.bim +\n",
      "temp_files/test_chr20_ENSG00000196476.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000196476.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000196476.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000196476.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000196476.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000196476.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000196476.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101019.16 --out compute_weights_out/ENSG00000101019.16_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101019.16.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000196476.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101019.16.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101019.16.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101019.16\n",
      "  --pheno ../data/gene_expressions/ENSG00000101019.16.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101019.16.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101019.16.bed +\n",
      "temp_files/test_chr20_ENSG00000101019.16.bim +\n",
      "temp_files/test_chr20_ENSG00000101019.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101019.16.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101019.16\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101019.16\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101019.16.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101019.16.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101019.16.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124214.14 --out compute_weights_out/ENSG00000124214.14_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124214.14.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101019.16 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124214.14.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124214.14.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124214.14\n",
      "  --pheno ../data/gene_expressions/ENSG00000124214.14.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124214.14.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124214.14.bed +\n",
      "temp_files/test_chr20_ENSG00000124214.14.bim +\n",
      "temp_files/test_chr20_ENSG00000124214.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124214.14.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124214.14\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124214.14\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124214.14.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124214.14.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124214.14.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000232645.1 --out compute_weights_out/ENSG00000232645.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000232645.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124214.14 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000232645.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000232645.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000232645.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000232645.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000232645.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000232645.1.bed +\n",
      "temp_files/test_chr20_ENSG00000232645.1.bim +\n",
      "temp_files/test_chr20_ENSG00000232645.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000232645.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000232645.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000232645.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000232645.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000232645.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000232645.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101213.5 --out compute_weights_out/ENSG00000101213.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101213.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000232645.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101213.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101213.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101213.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000101213.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101213.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101213.5.bed +\n",
      "temp_files/test_chr20_ENSG00000101213.5.bim +\n",
      "temp_files/test_chr20_ENSG00000101213.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101213.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101213.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101213.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101213.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101213.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101213.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000214078.6 --out compute_weights_out/ENSG00000214078.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000214078.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101213.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000214078.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000214078.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000214078.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000214078.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000214078.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000214078.6.bed +\n",
      "temp_files/test_chr20_ENSG00000214078.6.bim +\n",
      "temp_files/test_chr20_ENSG00000214078.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000214078.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000214078.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000214078.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000214078.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000214078.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000214078.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000229222.1 --out compute_weights_out/ENSG00000229222.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000229222.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000214078.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000229222.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000229222.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000229222.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000229222.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000229222.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000229222.1.bed +\n",
      "temp_files/test_chr20_ENSG00000229222.1.bim +\n",
      "temp_files/test_chr20_ENSG00000229222.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000229222.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000229222.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000229222.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000229222.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000229222.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000229222.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000215440.7 --out compute_weights_out/ENSG00000215440.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000215440.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000229222.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000215440.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000215440.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000215440.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000215440.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000215440.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000215440.7.bed +\n",
      "temp_files/test_chr20_ENSG00000215440.7.bim +\n",
      "temp_files/test_chr20_ENSG00000215440.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000215440.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000215440.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000215440.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000215440.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000215440.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000215440.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101004.9 --out compute_weights_out/ENSG00000101004.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101004.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000215440.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101004.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101004.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101004.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000101004.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101004.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101004.9.bed +\n",
      "temp_files/test_chr20_ENSG00000101004.9.bim +\n",
      "temp_files/test_chr20_ENSG00000101004.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101004.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101004.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101004.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101004.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101004.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101004.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000240742.1 --out compute_weights_out/ENSG00000240742.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000240742.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101004.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000240742.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000240742.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000240742.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000240742.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000240742.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000240742.1.bed +\n",
      "temp_files/test_chr20_ENSG00000240742.1.bim +\n",
      "temp_files/test_chr20_ENSG00000240742.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000240742.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000240742.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000240742.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000240742.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000240742.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000240742.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000089063.10 --out compute_weights_out/ENSG00000089063.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000089063.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000240742.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000089063.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000089063.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000089063.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000089063.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000089063.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000089063.10.bed +\n",
      "temp_files/test_chr20_ENSG00000089063.10.bim +\n",
      "temp_files/test_chr20_ENSG00000089063.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000089063.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000089063.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000089063.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000089063.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000089063.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000089063.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000149646.7 --out compute_weights_out/ENSG00000149646.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000149646.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000089063.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149646.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000149646.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000149646.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000149646.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149646.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000149646.7.bed +\n",
      "temp_files/test_chr20_ENSG00000149646.7.bim +\n",
      "temp_files/test_chr20_ENSG00000149646.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149646.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000149646.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000149646.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149646.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000149646.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000149646.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101310.10 --out compute_weights_out/ENSG00000101310.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101310.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000149646.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101310.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101310.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101310.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000101310.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101310.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101310.10.bed +\n",
      "temp_files/test_chr20_ENSG00000101310.10.bim +\n",
      "temp_files/test_chr20_ENSG00000101310.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101310.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101310.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101310.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101310.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101310.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101310.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000250917.1 --out compute_weights_out/ENSG00000250917.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000250917.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101310.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000250917.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000250917.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000250917.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000250917.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000250917.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000250917.1.bed +\n",
      "temp_files/test_chr20_ENSG00000250917.1.bim +\n",
      "temp_files/test_chr20_ENSG00000250917.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000250917.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000250917.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000250917.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000250917.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000250917.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000250917.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125798.8 --out compute_weights_out/ENSG00000125798.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125798.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000250917.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125798.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125798.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125798.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000125798.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125798.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125798.8.bed +\n",
      "temp_files/test_chr20_ENSG00000125798.8.bim +\n",
      "temp_files/test_chr20_ENSG00000125798.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125798.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125798.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125798.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125798.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125798.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125798.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000233578.1 --out compute_weights_out/ENSG00000233578.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000233578.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125798.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000233578.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000233578.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000233578.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000233578.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000233578.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000233578.1.bed +\n",
      "temp_files/test_chr20_ENSG00000233578.1.bim +\n",
      "temp_files/test_chr20_ENSG00000233578.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000233578.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000233578.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000233578.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000233578.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000233578.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000233578.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124181.7 --out compute_weights_out/ENSG00000124181.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124181.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000233578.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124181.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124181.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124181.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000124181.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124181.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124181.7.bed +\n",
      "temp_files/test_chr20_ENSG00000124181.7.bim +\n",
      "temp_files/test_chr20_ENSG00000124181.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124181.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124181.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124181.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124181.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124181.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124181.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000197818.7 --out compute_weights_out/ENSG00000197818.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000197818.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124181.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000197818.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000197818.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000197818.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000197818.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000197818.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000197818.7.bed +\n",
      "temp_files/test_chr20_ENSG00000197818.7.bim +\n",
      "temp_files/test_chr20_ENSG00000197818.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000197818.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000197818.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000197818.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000197818.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000197818.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000197818.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000231081.1 --out compute_weights_out/ENSG00000231081.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000231081.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000197818.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000231081.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000231081.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000231081.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000231081.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000231081.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000231081.1.bed +\n",
      "temp_files/test_chr20_ENSG00000231081.1.bim +\n",
      "temp_files/test_chr20_ENSG00000231081.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000231081.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000231081.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000231081.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000231081.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000231081.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000231081.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000168734.9 --out compute_weights_out/ENSG00000168734.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000168734.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000231081.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000168734.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000168734.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000168734.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000168734.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000168734.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000168734.9.bed +\n",
      "temp_files/test_chr20_ENSG00000168734.9.bim +\n",
      "temp_files/test_chr20_ENSG00000168734.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000168734.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000168734.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000168734.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000168734.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000168734.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000168734.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000198547.3 --out compute_weights_out/ENSG00000198547.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000198547.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000168734.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000198547.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000198547.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000198547.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000198547.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000198547.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000198547.3.bed +\n",
      "temp_files/test_chr20_ENSG00000198547.3.bim +\n",
      "temp_files/test_chr20_ENSG00000198547.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000198547.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000198547.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000198547.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000198547.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000198547.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000198547.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000235508.2 --out compute_weights_out/ENSG00000235508.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000235508.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000198547.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000235508.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000235508.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000235508.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000235508.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000235508.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000235508.2.bed +\n",
      "temp_files/test_chr20_ENSG00000235508.2.bim +\n",
      "temp_files/test_chr20_ENSG00000235508.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000235508.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000235508.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000235508.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000235508.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000235508.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000235508.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125970.7 --out compute_weights_out/ENSG00000125970.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125970.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000235508.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125970.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125970.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125970.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000125970.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125970.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125970.7.bed +\n",
      "temp_files/test_chr20_ENSG00000125970.7.bim +\n",
      "temp_files/test_chr20_ENSG00000125970.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125970.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125970.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125970.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125970.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125970.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125970.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101210.6 --out compute_weights_out/ENSG00000101210.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101210.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125970.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101210.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101210.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101210.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000101210.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101210.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101210.6.bed +\n",
      "temp_files/test_chr20_ENSG00000101210.6.bim +\n",
      "temp_files/test_chr20_ENSG00000101210.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101210.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101210.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101210.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101210.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101210.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101210.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000022277.8 --out compute_weights_out/ENSG00000022277.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000022277.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101210.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000022277.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000022277.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000022277.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000022277.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000022277.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000022277.8.bed +\n",
      "temp_files/test_chr20_ENSG00000022277.8.bim +\n",
      "temp_files/test_chr20_ENSG00000022277.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000022277.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000022277.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000022277.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000022277.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000022277.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000022277.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125845.5 --out compute_weights_out/ENSG00000125845.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125845.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000022277.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125845.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125845.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125845.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000125845.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125845.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125845.5.bed +\n",
      "temp_files/test_chr20_ENSG00000125845.5.bim +\n",
      "temp_files/test_chr20_ENSG00000125845.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125845.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125845.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125845.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125845.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125845.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125845.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101109.7 --out compute_weights_out/ENSG00000101109.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101109.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125845.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101109.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101109.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101109.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000101109.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101109.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101109.7.bed +\n",
      "temp_files/test_chr20_ENSG00000101109.7.bim +\n",
      "temp_files/test_chr20_ENSG00000101109.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101109.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101109.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101109.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101109.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101109.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101109.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000088899.10 --out compute_weights_out/ENSG00000088899.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000088899.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101109.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088899.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000088899.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000088899.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000088899.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088899.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000088899.10.bed +\n",
      "temp_files/test_chr20_ENSG00000088899.10.bim +\n",
      "temp_files/test_chr20_ENSG00000088899.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088899.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000088899.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000088899.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088899.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000088899.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000088899.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000170471.9 --out compute_weights_out/ENSG00000170471.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000170471.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000088899.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000170471.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000170471.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000170471.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000170471.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000170471.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000170471.9.bed +\n",
      "temp_files/test_chr20_ENSG00000170471.9.bim +\n",
      "temp_files/test_chr20_ENSG00000170471.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000170471.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000170471.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000170471.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000170471.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000170471.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000170471.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000261582.1 --out compute_weights_out/ENSG00000261582.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000261582.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000170471.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000261582.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000261582.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000261582.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000261582.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000261582.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000261582.1.bed +\n",
      "temp_files/test_chr20_ENSG00000261582.1.bim +\n",
      "temp_files/test_chr20_ENSG00000261582.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000261582.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000261582.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000261582.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000261582.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000261582.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000261582.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000251390.1 --out compute_weights_out/ENSG00000251390.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000251390.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000261582.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000251390.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000251390.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000251390.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000251390.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000251390.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000251390.1.bed +\n",
      "temp_files/test_chr20_ENSG00000251390.1.bim +\n",
      "temp_files/test_chr20_ENSG00000251390.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000251390.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000251390.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000251390.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000251390.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000251390.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000251390.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000225849.1 --out compute_weights_out/ENSG00000225849.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000225849.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000251390.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225849.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000225849.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000225849.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000225849.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225849.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000225849.1.bed +\n",
      "temp_files/test_chr20_ENSG00000225849.1.bim +\n",
      "temp_files/test_chr20_ENSG00000225849.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225849.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000225849.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000225849.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225849.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000225849.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000225849.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101096.13 --out compute_weights_out/ENSG00000101096.13_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101096.13.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000225849.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101096.13.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101096.13.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101096.13\n",
      "  --pheno ../data/gene_expressions/ENSG00000101096.13.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101096.13.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101096.13.bed +\n",
      "temp_files/test_chr20_ENSG00000101096.13.bim +\n",
      "temp_files/test_chr20_ENSG00000101096.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101096.13.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101096.13\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101096.13\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101096.13.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101096.13.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101096.13.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000166619.7 --out compute_weights_out/ENSG00000166619.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000166619.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101096.13 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000166619.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000166619.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000166619.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000166619.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000166619.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000166619.7.bed +\n",
      "temp_files/test_chr20_ENSG00000166619.7.bim +\n",
      "temp_files/test_chr20_ENSG00000166619.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000166619.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000166619.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000166619.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000166619.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000166619.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000166619.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000088836.7 --out compute_weights_out/ENSG00000088836.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000088836.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000166619.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088836.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000088836.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000088836.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000088836.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088836.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000088836.7.bed +\n",
      "temp_files/test_chr20_ENSG00000088836.7.bim +\n",
      "temp_files/test_chr20_ENSG00000088836.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088836.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000088836.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000088836.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088836.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000088836.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000088836.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125520.8 --out compute_weights_out/ENSG00000125520.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125520.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000088836.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125520.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125520.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125520.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000125520.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125520.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125520.8.bed +\n",
      "temp_files/test_chr20_ENSG00000125520.8.bim +\n",
      "temp_files/test_chr20_ENSG00000125520.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125520.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125520.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125520.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125520.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125520.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125520.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000261062.1 --out compute_weights_out/ENSG00000261062.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000261062.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125520.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000261062.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000261062.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000261062.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000261062.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000261062.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000261062.1.bed +\n",
      "temp_files/test_chr20_ENSG00000261062.1.bim +\n",
      "temp_files/test_chr20_ENSG00000261062.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000261062.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000261062.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000261062.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000261062.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000261062.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000261062.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000171867.11 --out compute_weights_out/ENSG00000171867.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000171867.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000261062.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000171867.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000171867.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000171867.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000171867.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000171867.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000171867.11.bed +\n",
      "temp_files/test_chr20_ENSG00000171867.11.bim +\n",
      "temp_files/test_chr20_ENSG00000171867.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000171867.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000171867.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000171867.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000171867.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000171867.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000171867.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000256222.1 --out compute_weights_out/ENSG00000256222.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000256222.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000171867.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000256222.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000256222.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000256222.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000256222.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000256222.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000256222.1.bed +\n",
      "temp_files/test_chr20_ENSG00000256222.1.bim +\n",
      "temp_files/test_chr20_ENSG00000256222.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000256222.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000256222.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000256222.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000256222.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000256222.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000256222.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000149596.6 --out compute_weights_out/ENSG00000149596.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000149596.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000256222.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149596.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000149596.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000149596.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000149596.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149596.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000149596.6.bed +\n",
      "temp_files/test_chr20_ENSG00000149596.6.bim +\n",
      "temp_files/test_chr20_ENSG00000149596.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149596.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000149596.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000149596.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149596.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000149596.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000149596.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000198171.7 --out compute_weights_out/ENSG00000198171.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000198171.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000149596.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000198171.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000198171.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000198171.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000198171.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000198171.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000198171.7.bed +\n",
      "temp_files/test_chr20_ENSG00000198171.7.bim +\n",
      "temp_files/test_chr20_ENSG00000198171.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000198171.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000198171.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000198171.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000198171.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000198171.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000198171.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000118705.11 --out compute_weights_out/ENSG00000118705.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000118705.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000198171.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000118705.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000118705.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000118705.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000118705.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000118705.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000118705.11.bed +\n",
      "temp_files/test_chr20_ENSG00000118705.11.bim +\n",
      "temp_files/test_chr20_ENSG00000118705.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000118705.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000118705.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000118705.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000118705.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000118705.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000118705.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000224628.2 --out compute_weights_out/ENSG00000224628.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000224628.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000118705.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000224628.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000224628.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000224628.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000224628.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000224628.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000224628.2.bed +\n",
      "temp_files/test_chr20_ENSG00000224628.2.bim +\n",
      "temp_files/test_chr20_ENSG00000224628.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000224628.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000224628.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000224628.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000224628.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000224628.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000224628.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000149600.6 --out compute_weights_out/ENSG00000149600.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000149600.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000224628.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149600.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000149600.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000149600.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000149600.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149600.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000149600.6.bed +\n",
      "temp_files/test_chr20_ENSG00000149600.6.bim +\n",
      "temp_files/test_chr20_ENSG00000149600.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149600.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000149600.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000149600.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149600.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000149600.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000149600.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000235217.5 --out compute_weights_out/ENSG00000235217.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000235217.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000149600.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000235217.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000235217.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000235217.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000235217.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000235217.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000235217.5.bed +\n",
      "temp_files/test_chr20_ENSG00000235217.5.bim +\n",
      "temp_files/test_chr20_ENSG00000235217.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000235217.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000235217.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000235217.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000235217.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000235217.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000235217.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124172.5 --out compute_weights_out/ENSG00000124172.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124172.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000235217.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124172.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124172.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124172.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000124172.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124172.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124172.5.bed +\n",
      "temp_files/test_chr20_ENSG00000124172.5.bim +\n",
      "temp_files/test_chr20_ENSG00000124172.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124172.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124172.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124172.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124172.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124172.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124172.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000130584.5 --out compute_weights_out/ENSG00000130584.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000130584.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124172.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000130584.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000130584.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000130584.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000130584.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000130584.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000130584.5.bed +\n",
      "temp_files/test_chr20_ENSG00000130584.5.bim +\n",
      "temp_files/test_chr20_ENSG00000130584.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000130584.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000130584.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000130584.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000130584.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000130584.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000130584.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000175063.12 --out compute_weights_out/ENSG00000175063.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000175063.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000130584.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000175063.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000175063.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000175063.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000175063.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000175063.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000175063.12.bed +\n",
      "temp_files/test_chr20_ENSG00000175063.12.bim +\n",
      "temp_files/test_chr20_ENSG00000175063.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000175063.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000175063.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000175063.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000175063.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000175063.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000175063.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124193.9 --out compute_weights_out/ENSG00000124193.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124193.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000175063.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124193.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124193.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124193.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000124193.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124193.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124193.9.bed +\n",
      "temp_files/test_chr20_ENSG00000124193.9.bim +\n",
      "temp_files/test_chr20_ENSG00000124193.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124193.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124193.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124193.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124193.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124193.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124193.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000256566.1 --out compute_weights_out/ENSG00000256566.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000256566.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124193.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000256566.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000256566.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000256566.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000256566.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000256566.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000256566.1.bed +\n",
      "temp_files/test_chr20_ENSG00000256566.1.bim +\n",
      "temp_files/test_chr20_ENSG00000256566.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000256566.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000256566.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000256566.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000256566.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000256566.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000256566.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101347.7 --out compute_weights_out/ENSG00000101347.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101347.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000256566.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101347.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101347.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101347.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000101347.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101347.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101347.7.bed +\n",
      "temp_files/test_chr20_ENSG00000101347.7.bim +\n",
      "temp_files/test_chr20_ENSG00000101347.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101347.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101347.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101347.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101347.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101347.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101347.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000218902.3 --out compute_weights_out/ENSG00000218902.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000218902.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101347.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000218902.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000218902.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000218902.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000218902.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000218902.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000218902.3.bed +\n",
      "temp_files/test_chr20_ENSG00000218902.3.bim +\n",
      "temp_files/test_chr20_ENSG00000218902.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000218902.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000218902.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000218902.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000218902.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000218902.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000218902.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000088826.12 --out compute_weights_out/ENSG00000088826.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000088826.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000218902.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088826.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000088826.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000088826.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000088826.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088826.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000088826.12.bed +\n",
      "temp_files/test_chr20_ENSG00000088826.12.bim +\n",
      "temp_files/test_chr20_ENSG00000088826.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088826.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000088826.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000088826.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088826.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000088826.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000088826.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000236526.1 --out compute_weights_out/ENSG00000236526.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000236526.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000088826.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000236526.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000236526.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000236526.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000236526.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000236526.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000236526.1.bed +\n",
      "temp_files/test_chr20_ENSG00000236526.1.bim +\n",
      "temp_files/test_chr20_ENSG00000236526.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000236526.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000236526.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000236526.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000236526.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000236526.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000236526.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000225978.1 --out compute_weights_out/ENSG00000225978.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000225978.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000236526.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225978.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000225978.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000225978.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000225978.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225978.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000225978.1.bed +\n",
      "temp_files/test_chr20_ENSG00000225978.1.bim +\n",
      "temp_files/test_chr20_ENSG00000225978.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225978.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000225978.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000225978.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225978.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000225978.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000225978.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000130684.8 --out compute_weights_out/ENSG00000130684.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000130684.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000225978.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000130684.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000130684.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000130684.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000130684.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000130684.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000130684.8.bed +\n",
      "temp_files/test_chr20_ENSG00000130684.8.bim +\n",
      "temp_files/test_chr20_ENSG00000130684.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000130684.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000130684.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000130684.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000130684.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000130684.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000130684.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101000.3 --out compute_weights_out/ENSG00000101000.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101000.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000130684.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101000.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101000.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101000.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000101000.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101000.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101000.3.bed +\n",
      "temp_files/test_chr20_ENSG00000101000.3.bim +\n",
      "temp_files/test_chr20_ENSG00000101000.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101000.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101000.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101000.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101000.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101000.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101000.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000205181.2 --out compute_weights_out/ENSG00000205181.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000205181.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101000.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000205181.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000205181.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000205181.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000205181.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000205181.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000205181.2.bed +\n",
      "temp_files/test_chr20_ENSG00000205181.2.bim +\n",
      "temp_files/test_chr20_ENSG00000205181.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000205181.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000205181.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000205181.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000205181.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000205181.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000205181.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101187.10 --out compute_weights_out/ENSG00000101187.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101187.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000205181.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101187.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101187.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101187.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000101187.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101187.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101187.10.bed +\n",
      "temp_files/test_chr20_ENSG00000101187.10.bim +\n",
      "temp_files/test_chr20_ENSG00000101187.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101187.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101187.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101187.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101187.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101187.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101187.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000182584.4 --out compute_weights_out/ENSG00000182584.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000182584.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101187.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000182584.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000182584.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000182584.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000182584.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000182584.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000182584.4.bed +\n",
      "temp_files/test_chr20_ENSG00000182584.4.bim +\n",
      "temp_files/test_chr20_ENSG00000182584.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000182584.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000182584.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000182584.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000182584.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000182584.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000182584.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101199.8 --out compute_weights_out/ENSG00000101199.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101199.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000182584.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101199.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101199.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101199.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000101199.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101199.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101199.8.bed +\n",
      "temp_files/test_chr20_ENSG00000101199.8.bim +\n",
      "temp_files/test_chr20_ENSG00000101199.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101199.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101199.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101199.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101199.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101199.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101199.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101190.8 --out compute_weights_out/ENSG00000101190.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101190.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101199.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101190.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101190.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101190.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000101190.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101190.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101190.8.bed +\n",
      "temp_files/test_chr20_ENSG00000101190.8.bim +\n",
      "temp_files/test_chr20_ENSG00000101190.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101190.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101190.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101190.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101190.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101190.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101190.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124155.11 --out compute_weights_out/ENSG00000124155.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124155.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101190.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124155.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124155.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124155.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000124155.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124155.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124155.11.bed +\n",
      "temp_files/test_chr20_ENSG00000124155.11.bim +\n",
      "temp_files/test_chr20_ENSG00000124155.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124155.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124155.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124155.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124155.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124155.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124155.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000149435.8 --out compute_weights_out/ENSG00000149435.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000149435.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124155.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149435.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000149435.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000149435.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000149435.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149435.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000149435.8.bed +\n",
      "temp_files/test_chr20_ENSG00000149435.8.bim +\n",
      "temp_files/test_chr20_ENSG00000149435.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149435.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000149435.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000149435.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149435.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000149435.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000149435.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000149635.2 --out compute_weights_out/ENSG00000149635.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000149635.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000149435.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149635.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000149635.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000149635.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000149635.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149635.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000149635.2.bed +\n",
      "temp_files/test_chr20_ENSG00000149635.2.bim +\n",
      "temp_files/test_chr20_ENSG00000149635.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149635.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000149635.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000149635.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149635.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000149635.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000149635.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125841.7 --out compute_weights_out/ENSG00000125841.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125841.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000149635.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125841.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125841.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125841.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000125841.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125841.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125841.7.bed +\n",
      "temp_files/test_chr20_ENSG00000125841.7.bim +\n",
      "temp_files/test_chr20_ENSG00000125841.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125841.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125841.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125841.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125841.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125841.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125841.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000230837.1 --out compute_weights_out/ENSG00000230837.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000230837.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125841.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000230837.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000230837.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000230837.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000230837.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000230837.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000230837.1.bed +\n",
      "temp_files/test_chr20_ENSG00000230837.1.bim +\n",
      "temp_files/test_chr20_ENSG00000230837.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000230837.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000230837.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000230837.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000230837.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000230837.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000230837.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000230563.2 --out compute_weights_out/ENSG00000230563.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000230563.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000230837.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000230563.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000230563.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000230563.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000230563.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000230563.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000230563.2.bed +\n",
      "temp_files/test_chr20_ENSG00000230563.2.bim +\n",
      "temp_files/test_chr20_ENSG00000230563.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000230563.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000230563.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000230563.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000230563.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000230563.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000230563.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000242372.2 --out compute_weights_out/ENSG00000242372.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000242372.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000230563.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000242372.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000242372.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000242372.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000242372.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000242372.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000242372.2.bed +\n",
      "temp_files/test_chr20_ENSG00000242372.2.bim +\n",
      "temp_files/test_chr20_ENSG00000242372.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000242372.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000242372.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000242372.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000242372.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000242372.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000242372.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000184402.9 --out compute_weights_out/ENSG00000184402.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000184402.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000242372.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000184402.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000184402.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000184402.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000184402.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000184402.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000184402.9.bed +\n",
      "temp_files/test_chr20_ENSG00000184402.9.bim +\n",
      "temp_files/test_chr20_ENSG00000184402.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000184402.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000184402.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000184402.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000184402.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000184402.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000184402.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101417.7 --out compute_weights_out/ENSG00000101417.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101417.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000184402.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101417.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101417.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101417.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000101417.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101417.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101417.7.bed +\n",
      "temp_files/test_chr20_ENSG00000101417.7.bim +\n",
      "temp_files/test_chr20_ENSG00000101417.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101417.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101417.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101417.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101417.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101417.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101417.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000173418.7 --out compute_weights_out/ENSG00000173418.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000173418.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101417.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000173418.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000173418.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000173418.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000173418.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000173418.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000173418.7.bed +\n",
      "temp_files/test_chr20_ENSG00000173418.7.bim +\n",
      "temp_files/test_chr20_ENSG00000173418.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000173418.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000173418.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000173418.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000173418.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000173418.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000173418.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000233402.1 --out compute_weights_out/ENSG00000233402.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000233402.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000173418.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000233402.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000233402.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000233402.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000233402.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000233402.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000233402.1.bed +\n",
      "temp_files/test_chr20_ENSG00000233402.1.bim +\n",
      "temp_files/test_chr20_ENSG00000233402.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000233402.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000233402.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000233402.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000233402.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000233402.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000233402.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000230750.1 --out compute_weights_out/ENSG00000230750.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000230750.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000233402.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000230750.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000230750.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000230750.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000230750.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000230750.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000230750.1.bed +\n",
      "temp_files/test_chr20_ENSG00000230750.1.bim +\n",
      "temp_files/test_chr20_ENSG00000230750.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000230750.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000230750.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000230750.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000230750.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000230750.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000230750.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000258366.2 --out compute_weights_out/ENSG00000258366.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000258366.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000230750.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000258366.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000258366.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000258366.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000258366.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000258366.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000258366.2.bed +\n",
      "temp_files/test_chr20_ENSG00000258366.2.bim +\n",
      "temp_files/test_chr20_ENSG00000258366.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000258366.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000258366.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000258366.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000258366.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000258366.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000258366.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101017.8 --out compute_weights_out/ENSG00000101017.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101017.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000258366.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101017.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101017.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101017.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000101017.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101017.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101017.8.bed +\n",
      "temp_files/test_chr20_ENSG00000101017.8.bim +\n",
      "temp_files/test_chr20_ENSG00000101017.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101017.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101017.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101017.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101017.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101017.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101017.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125810.8 --out compute_weights_out/ENSG00000125810.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125810.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101017.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125810.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125810.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125810.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000125810.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125810.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125810.8.bed +\n",
      "temp_files/test_chr20_ENSG00000125810.8.bim +\n",
      "temp_files/test_chr20_ENSG00000125810.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125810.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125810.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125810.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125810.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125810.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125810.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000204044.5 --out compute_weights_out/ENSG00000204044.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000204044.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125810.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000204044.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000204044.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000204044.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000204044.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000204044.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000204044.5.bed +\n",
      "temp_files/test_chr20_ENSG00000204044.5.bim +\n",
      "temp_files/test_chr20_ENSG00000204044.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000204044.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000204044.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000204044.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000204044.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000204044.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000204044.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000196756.5 --out compute_weights_out/ENSG00000196756.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000196756.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000204044.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000196756.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000196756.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000196756.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000196756.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000196756.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000196756.5.bed +\n",
      "temp_files/test_chr20_ENSG00000196756.5.bim +\n",
      "temp_files/test_chr20_ENSG00000196756.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000196756.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000196756.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000196756.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000196756.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000196756.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000196756.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124196.5 --out compute_weights_out/ENSG00000124196.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124196.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000196756.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124196.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124196.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124196.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000124196.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124196.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124196.5.bed +\n",
      "temp_files/test_chr20_ENSG00000124196.5.bim +\n",
      "temp_files/test_chr20_ENSG00000124196.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124196.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124196.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124196.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124196.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124196.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124196.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125775.10 --out compute_weights_out/ENSG00000125775.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125775.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124196.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125775.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125775.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125775.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000125775.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125775.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125775.10.bed +\n",
      "temp_files/test_chr20_ENSG00000125775.10.bim +\n",
      "temp_files/test_chr20_ENSG00000125775.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125775.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125775.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125775.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125775.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125775.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125775.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000132646.6 --out compute_weights_out/ENSG00000132646.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000132646.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125775.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132646.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000132646.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000132646.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000132646.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132646.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000132646.6.bed +\n",
      "temp_files/test_chr20_ENSG00000132646.6.bim +\n",
      "temp_files/test_chr20_ENSG00000132646.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132646.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000132646.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000132646.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132646.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000132646.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000132646.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000203883.5 --out compute_weights_out/ENSG00000203883.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000203883.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000132646.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000203883.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000203883.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000203883.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000203883.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000203883.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000203883.5.bed +\n",
      "temp_files/test_chr20_ENSG00000203883.5.bim +\n",
      "temp_files/test_chr20_ENSG00000203883.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000203883.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000203883.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000203883.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000203883.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000203883.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000203883.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000213742.2 --out compute_weights_out/ENSG00000213742.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000213742.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000203883.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000213742.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000213742.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000213742.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000213742.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000213742.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000213742.2.bed +\n",
      "temp_files/test_chr20_ENSG00000213742.2.bim +\n",
      "temp_files/test_chr20_ENSG00000213742.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000213742.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000213742.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000213742.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000213742.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000213742.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000213742.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000215251.3 --out compute_weights_out/ENSG00000215251.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000215251.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000213742.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000215251.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000215251.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000215251.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000215251.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000215251.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000215251.3.bed +\n",
      "temp_files/test_chr20_ENSG00000215251.3.bim +\n",
      "temp_files/test_chr20_ENSG00000215251.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000215251.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000215251.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000215251.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000215251.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000215251.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000215251.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000185834.9 --out compute_weights_out/ENSG00000185834.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000185834.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000215251.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000185834.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000185834.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000185834.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000185834.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000185834.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000185834.9.bed +\n",
      "temp_files/test_chr20_ENSG00000185834.9.bim +\n",
      "temp_files/test_chr20_ENSG00000185834.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000185834.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000185834.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000185834.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000185834.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000185834.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000185834.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000053438.7 --out compute_weights_out/ENSG00000053438.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000053438.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000185834.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000053438.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000053438.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000053438.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000053438.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000053438.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000053438.7.bed +\n",
      "temp_files/test_chr20_ENSG00000053438.7.bim +\n",
      "temp_files/test_chr20_ENSG00000053438.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000053438.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000053438.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000053438.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000053438.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000053438.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000053438.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000088325.11 --out compute_weights_out/ENSG00000088325.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000088325.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000053438.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088325.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000088325.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000088325.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000088325.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088325.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000088325.11.bed +\n",
      "temp_files/test_chr20_ENSG00000088325.11.bim +\n",
      "temp_files/test_chr20_ENSG00000088325.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088325.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000088325.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000088325.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088325.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000088325.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000088325.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000223864.1 --out compute_weights_out/ENSG00000223864.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000223864.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000088325.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000223864.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000223864.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000223864.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000223864.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000223864.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000223864.1.bed +\n",
      "temp_files/test_chr20_ENSG00000223864.1.bim +\n",
      "temp_files/test_chr20_ENSG00000223864.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000223864.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000223864.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000223864.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000223864.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000223864.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000223864.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000078747.7 --out compute_weights_out/ENSG00000078747.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000078747.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000223864.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000078747.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000078747.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000078747.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000078747.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000078747.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000078747.7.bed +\n",
      "temp_files/test_chr20_ENSG00000078747.7.bim +\n",
      "temp_files/test_chr20_ENSG00000078747.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000078747.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000078747.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000078747.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000078747.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000078747.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000078747.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124217.3 --out compute_weights_out/ENSG00000124217.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124217.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000078747.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124217.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124217.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124217.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000124217.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124217.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124217.3.bed +\n",
      "temp_files/test_chr20_ENSG00000124217.3.bim +\n",
      "temp_files/test_chr20_ENSG00000124217.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124217.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124217.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124217.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124217.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124217.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124217.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101197.8 --out compute_weights_out/ENSG00000101197.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101197.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124217.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101197.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101197.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101197.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000101197.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101197.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101197.8.bed +\n",
      "temp_files/test_chr20_ENSG00000101197.8.bim +\n",
      "temp_files/test_chr20_ENSG00000101197.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101197.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101197.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101197.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101197.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101197.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101197.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101470.5 --out compute_weights_out/ENSG00000101470.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101470.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101197.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101470.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101470.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101470.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000101470.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101470.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101470.5.bed +\n",
      "temp_files/test_chr20_ENSG00000101470.5.bim +\n",
      "temp_files/test_chr20_ENSG00000101470.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101470.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101470.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101470.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101470.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101470.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101470.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101082.9 --out compute_weights_out/ENSG00000101082.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101082.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101470.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101082.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101082.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101082.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000101082.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101082.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101082.9.bed +\n",
      "temp_files/test_chr20_ENSG00000101082.9.bim +\n",
      "temp_files/test_chr20_ENSG00000101082.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101082.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101082.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101082.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101082.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101082.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101082.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125835.12 --out compute_weights_out/ENSG00000125835.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125835.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101082.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125835.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125835.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125835.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000125835.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125835.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125835.12.bed +\n",
      "temp_files/test_chr20_ENSG00000125835.12.bim +\n",
      "temp_files/test_chr20_ENSG00000125835.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125835.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125835.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125835.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125835.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125835.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125835.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000236542.1 --out compute_weights_out/ENSG00000236542.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000236542.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125835.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000236542.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000236542.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000236542.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000236542.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000236542.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000236542.1.bed +\n",
      "temp_files/test_chr20_ENSG00000236542.1.bim +\n",
      "temp_files/test_chr20_ENSG00000236542.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000236542.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000236542.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000236542.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000236542.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000236542.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000236542.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124091.5 --out compute_weights_out/ENSG00000124091.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124091.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000236542.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124091.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124091.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124091.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000124091.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124091.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124091.5.bed +\n",
      "temp_files/test_chr20_ENSG00000124091.5.bim +\n",
      "temp_files/test_chr20_ENSG00000124091.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124091.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124091.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124091.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124091.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124091.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124091.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000166913.8 --out compute_weights_out/ENSG00000166913.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000166913.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124091.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000166913.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000166913.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000166913.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000166913.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000166913.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000166913.8.bed +\n",
      "temp_files/test_chr20_ENSG00000166913.8.bim +\n",
      "temp_files/test_chr20_ENSG00000166913.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000166913.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000166913.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000166913.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000166913.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000166913.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000166913.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000198026.5 --out compute_weights_out/ENSG00000198026.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000198026.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000166913.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000198026.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000198026.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000198026.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000198026.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000198026.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000198026.5.bed +\n",
      "temp_files/test_chr20_ENSG00000198026.5.bim +\n",
      "temp_files/test_chr20_ENSG00000198026.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000198026.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000198026.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000198026.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000198026.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000198026.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000198026.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101246.12 --out compute_weights_out/ENSG00000101246.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101246.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000198026.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101246.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101246.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101246.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000101246.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101246.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101246.12.bed +\n",
      "temp_files/test_chr20_ENSG00000101246.12.bim +\n",
      "temp_files/test_chr20_ENSG00000101246.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101246.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101246.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101246.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101246.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101246.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101246.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000235698.1 --out compute_weights_out/ENSG00000235698.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000235698.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101246.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000235698.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000235698.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000235698.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000235698.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000235698.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000235698.1.bed +\n",
      "temp_files/test_chr20_ENSG00000235698.1.bim +\n",
      "temp_files/test_chr20_ENSG00000235698.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000235698.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000235698.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000235698.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000235698.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000235698.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000235698.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000198959.6 --out compute_weights_out/ENSG00000198959.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000198959.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000235698.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000198959.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000198959.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000198959.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000198959.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000198959.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000198959.6.bed +\n",
      "temp_files/test_chr20_ENSG00000198959.6.bim +\n",
      "temp_files/test_chr20_ENSG00000198959.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000198959.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000198959.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000198959.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000198959.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000198959.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000198959.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000235036.3 --out compute_weights_out/ENSG00000235036.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000235036.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000198959.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000235036.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000235036.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000235036.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000235036.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000235036.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000235036.3.bed +\n",
      "temp_files/test_chr20_ENSG00000235036.3.bim +\n",
      "temp_files/test_chr20_ENSG00000235036.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000235036.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000235036.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000235036.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000235036.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000235036.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000235036.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125821.7 --out compute_weights_out/ENSG00000125821.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125821.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000235036.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125821.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125821.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125821.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000125821.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125821.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125821.7.bed +\n",
      "temp_files/test_chr20_ENSG00000125821.7.bim +\n",
      "temp_files/test_chr20_ENSG00000125821.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125821.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125821.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125821.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125821.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125821.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125821.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125834.8 --out compute_weights_out/ENSG00000125834.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125834.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125821.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125834.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125834.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125834.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000125834.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125834.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125834.8.bed +\n",
      "temp_files/test_chr20_ENSG00000125834.8.bim +\n",
      "temp_files/test_chr20_ENSG00000125834.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125834.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125834.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125834.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125834.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125834.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125834.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000126001.11 --out compute_weights_out/ENSG00000126001.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000126001.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125834.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000126001.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000126001.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000126001.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000126001.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000126001.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000126001.11.bed +\n",
      "temp_files/test_chr20_ENSG00000126001.11.bim +\n",
      "temp_files/test_chr20_ENSG00000126001.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000126001.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000126001.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000126001.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000126001.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000126001.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000126001.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125868.10 --out compute_weights_out/ENSG00000125868.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125868.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000126001.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125868.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125868.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125868.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000125868.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125868.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125868.10.bed +\n",
      "temp_files/test_chr20_ENSG00000125868.10.bim +\n",
      "temp_files/test_chr20_ENSG00000125868.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125868.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125868.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125868.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125868.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125868.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125868.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000080839.7 --out compute_weights_out/ENSG00000080839.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000080839.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125868.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000080839.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000080839.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000080839.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000080839.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000080839.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000080839.7.bed +\n",
      "temp_files/test_chr20_ENSG00000080839.7.bim +\n",
      "temp_files/test_chr20_ENSG00000080839.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000080839.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000080839.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000080839.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000080839.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000080839.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000080839.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000225246.1 --out compute_weights_out/ENSG00000225246.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000225246.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000080839.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225246.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000225246.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000225246.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000225246.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225246.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000225246.1.bed +\n",
      "temp_files/test_chr20_ENSG00000225246.1.bim +\n",
      "temp_files/test_chr20_ENSG00000225246.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225246.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000225246.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000225246.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225246.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000225246.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000225246.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125870.6 --out compute_weights_out/ENSG00000125870.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125870.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000225246.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125870.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125870.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125870.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000125870.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125870.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125870.6.bed +\n",
      "temp_files/test_chr20_ENSG00000125870.6.bim +\n",
      "temp_files/test_chr20_ENSG00000125870.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125870.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125870.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125870.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125870.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125870.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125870.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000087586.13 --out compute_weights_out/ENSG00000087586.13_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000087586.13.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125870.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000087586.13.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000087586.13.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000087586.13\n",
      "  --pheno ../data/gene_expressions/ENSG00000087586.13.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000087586.13.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000087586.13.bed +\n",
      "temp_files/test_chr20_ENSG00000087586.13.bim +\n",
      "temp_files/test_chr20_ENSG00000087586.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000087586.13.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000087586.13\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000087586.13\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000087586.13.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000087586.13.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000087586.13.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000126003.6 --out compute_weights_out/ENSG00000126003.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000126003.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000087586.13 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000126003.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000126003.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000126003.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000126003.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000126003.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000126003.6.bed +\n",
      "temp_files/test_chr20_ENSG00000126003.6.bim +\n",
      "temp_files/test_chr20_ENSG00000126003.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000126003.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000126003.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000126003.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000126003.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000126003.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000126003.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000087589.12 --out compute_weights_out/ENSG00000087589.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000087589.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000126003.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000087589.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000087589.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000087589.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000087589.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000087589.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000087589.12.bed +\n",
      "temp_files/test_chr20_ENSG00000087589.12.bim +\n",
      "temp_files/test_chr20_ENSG00000087589.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000087589.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000087589.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000087589.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000087589.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000087589.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000087589.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101412.9 --out compute_weights_out/ENSG00000101412.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101412.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000087589.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101412.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101412.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101412.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000101412.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101412.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101412.9.bed +\n",
      "temp_files/test_chr20_ENSG00000101412.9.bim +\n",
      "temp_files/test_chr20_ENSG00000101412.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101412.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101412.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101412.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101412.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101412.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101412.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000215595.1 --out compute_weights_out/ENSG00000215595.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000215595.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101412.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000215595.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000215595.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000215595.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000215595.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000215595.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000215595.1.bed +\n",
      "temp_files/test_chr20_ENSG00000215595.1.bim +\n",
      "temp_files/test_chr20_ENSG00000215595.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000215595.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000215595.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000215595.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000215595.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000215595.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000215595.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000214541.3 --out compute_weights_out/ENSG00000214541.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000214541.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000215595.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000214541.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000214541.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000214541.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000214541.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000214541.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000214541.3.bed +\n",
      "temp_files/test_chr20_ENSG00000214541.3.bim +\n",
      "temp_files/test_chr20_ENSG00000214541.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000214541.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000214541.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000214541.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000214541.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000214541.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000214541.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000234516.1 --out compute_weights_out/ENSG00000234516.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000234516.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000214541.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000234516.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000234516.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000234516.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000234516.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000234516.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000234516.1.bed +\n",
      "temp_files/test_chr20_ENSG00000234516.1.bim +\n",
      "temp_files/test_chr20_ENSG00000234516.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000234516.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000234516.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000234516.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000234516.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000234516.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000234516.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000132640.10 --out compute_weights_out/ENSG00000132640.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000132640.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000234516.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132640.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000132640.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000132640.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000132640.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132640.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000132640.10.bed +\n",
      "temp_files/test_chr20_ENSG00000132640.10.bim +\n",
      "temp_files/test_chr20_ENSG00000132640.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132640.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000132640.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000132640.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132640.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000132640.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000132640.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000171858.11 --out compute_weights_out/ENSG00000171858.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000171858.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000132640.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000171858.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000171858.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000171858.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000171858.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000171858.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000171858.11.bed +\n",
      "temp_files/test_chr20_ENSG00000171858.11.bim +\n",
      "temp_files/test_chr20_ENSG00000171858.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000171858.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000171858.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000171858.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000171858.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000171858.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000171858.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000226632.1 --out compute_weights_out/ENSG00000226632.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000226632.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000171858.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226632.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000226632.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000226632.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000226632.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226632.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000226632.1.bed +\n",
      "temp_files/test_chr20_ENSG00000226632.1.bim +\n",
      "temp_files/test_chr20_ENSG00000226632.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226632.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000226632.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000226632.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226632.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000226632.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000226632.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101438.3 --out compute_weights_out/ENSG00000101438.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101438.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000226632.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101438.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101438.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101438.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000101438.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101438.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101438.3.bed +\n",
      "temp_files/test_chr20_ENSG00000101438.3.bim +\n",
      "temp_files/test_chr20_ENSG00000101438.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101438.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101438.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101438.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101438.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101438.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101438.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125848.8 --out compute_weights_out/ENSG00000125848.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125848.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101438.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125848.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125848.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125848.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000125848.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125848.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125848.8.bed +\n",
      "temp_files/test_chr20_ENSG00000125848.8.bim +\n",
      "temp_files/test_chr20_ENSG00000125848.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125848.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125848.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125848.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125848.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125848.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125848.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000197496.4 --out compute_weights_out/ENSG00000197496.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000197496.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125848.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000197496.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000197496.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000197496.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000197496.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000197496.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000197496.4.bed +\n",
      "temp_files/test_chr20_ENSG00000197496.4.bim +\n",
      "temp_files/test_chr20_ENSG00000197496.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000197496.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000197496.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000197496.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000197496.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000197496.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000197496.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000196132.7 --out compute_weights_out/ENSG00000196132.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000196132.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000197496.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000196132.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000196132.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000196132.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000196132.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000196132.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000196132.7.bed +\n",
      "temp_files/test_chr20_ENSG00000196132.7.bim +\n",
      "temp_files/test_chr20_ENSG00000196132.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000196132.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000196132.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000196132.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000196132.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000196132.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000196132.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000089006.11 --out compute_weights_out/ENSG00000089006.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000089006.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000196132.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000089006.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000089006.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000089006.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000089006.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000089006.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000089006.11.bed +\n",
      "temp_files/test_chr20_ENSG00000089006.11.bim +\n",
      "temp_files/test_chr20_ENSG00000089006.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000089006.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000089006.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000089006.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000089006.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000089006.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000089006.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000130706.7 --out compute_weights_out/ENSG00000130706.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000130706.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000089006.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000130706.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000130706.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000130706.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000130706.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000130706.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000130706.7.bed +\n",
      "temp_files/test_chr20_ENSG00000130706.7.bim +\n",
      "temp_files/test_chr20_ENSG00000130706.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000130706.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000130706.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000130706.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000130706.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000130706.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000130706.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000025772.7 --out compute_weights_out/ENSG00000025772.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000025772.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000130706.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000025772.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000025772.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000025772.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000025772.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000025772.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000025772.7.bed +\n",
      "temp_files/test_chr20_ENSG00000025772.7.bim +\n",
      "temp_files/test_chr20_ENSG00000025772.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000025772.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000025772.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000025772.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000025772.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000025772.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000025772.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000236811.1 --out compute_weights_out/ENSG00000236811.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000236811.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000025772.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000236811.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000236811.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000236811.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000236811.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000236811.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000236811.1.bed +\n",
      "temp_files/test_chr20_ENSG00000236811.1.bim +\n",
      "temp_files/test_chr20_ENSG00000236811.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000236811.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000236811.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000236811.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000236811.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000236811.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000236811.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000100997.12 --out compute_weights_out/ENSG00000100997.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000100997.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000236811.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000100997.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000100997.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000100997.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000100997.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000100997.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000100997.12.bed +\n",
      "temp_files/test_chr20_ENSG00000100997.12.bim +\n",
      "temp_files/test_chr20_ENSG00000100997.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000100997.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000100997.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000100997.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000100997.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000100997.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000100997.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000149346.10 --out compute_weights_out/ENSG00000149346.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000149346.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000100997.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149346.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000149346.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000149346.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000149346.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149346.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000149346.10.bed +\n",
      "temp_files/test_chr20_ENSG00000149346.10.bim +\n",
      "temp_files/test_chr20_ENSG00000149346.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149346.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000149346.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000149346.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149346.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000149346.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000149346.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000225170.2 --out compute_weights_out/ENSG00000225170.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000225170.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000149346.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225170.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000225170.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000225170.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000225170.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225170.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000225170.2.bed +\n",
      "temp_files/test_chr20_ENSG00000225170.2.bim +\n",
      "temp_files/test_chr20_ENSG00000225170.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225170.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000225170.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000225170.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225170.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000225170.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000225170.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000089012.9 --out compute_weights_out/ENSG00000089012.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000089012.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000225170.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000089012.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000089012.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000089012.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000089012.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000089012.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000089012.9.bed +\n",
      "temp_files/test_chr20_ENSG00000089012.9.bim +\n",
      "temp_files/test_chr20_ENSG00000089012.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000089012.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000089012.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000089012.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000089012.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000089012.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000089012.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101146.8 --out compute_weights_out/ENSG00000101146.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101146.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000089012.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101146.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101146.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101146.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000101146.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101146.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101146.8.bed +\n",
      "temp_files/test_chr20_ENSG00000101146.8.bim +\n",
      "temp_files/test_chr20_ENSG00000101146.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101146.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101146.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101146.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101146.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101146.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101146.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000227379.2 --out compute_weights_out/ENSG00000227379.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000227379.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101146.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000227379.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000227379.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000227379.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000227379.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000227379.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000227379.2.bed +\n",
      "temp_files/test_chr20_ENSG00000227379.2.bim +\n",
      "temp_files/test_chr20_ENSG00000227379.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000227379.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000227379.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000227379.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000227379.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000227379.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000227379.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124243.11 --out compute_weights_out/ENSG00000124243.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124243.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000227379.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124243.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124243.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124243.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000124243.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124243.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124243.11.bed +\n",
      "temp_files/test_chr20_ENSG00000124243.11.bim +\n",
      "temp_files/test_chr20_ENSG00000124243.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124243.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124243.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124243.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124243.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124243.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124243.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101230.4 --out compute_weights_out/ENSG00000101230.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101230.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124243.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101230.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101230.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101230.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000101230.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101230.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101230.4.bed +\n",
      "temp_files/test_chr20_ENSG00000101230.4.bim +\n",
      "temp_files/test_chr20_ENSG00000101230.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101230.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101230.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101230.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101230.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101230.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101230.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101460.8 --out compute_weights_out/ENSG00000101460.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101460.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101230.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101460.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101460.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101460.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000101460.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101460.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101460.8.bed +\n",
      "temp_files/test_chr20_ENSG00000101460.8.bim +\n",
      "temp_files/test_chr20_ENSG00000101460.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101460.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101460.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101460.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101460.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101460.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101460.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000132821.7 --out compute_weights_out/ENSG00000132821.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000132821.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101460.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132821.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000132821.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000132821.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000132821.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132821.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000132821.7.bed +\n",
      "temp_files/test_chr20_ENSG00000132821.7.bim +\n",
      "temp_files/test_chr20_ENSG00000132821.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132821.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000132821.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000132821.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132821.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000132821.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000132821.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000149636.9 --out compute_weights_out/ENSG00000149636.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000149636.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000132821.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149636.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000149636.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000149636.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000149636.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149636.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000149636.9.bed +\n",
      "temp_files/test_chr20_ENSG00000149636.9.bim +\n",
      "temp_files/test_chr20_ENSG00000149636.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149636.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000149636.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000149636.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149636.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000149636.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000149636.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000213714.1 --out compute_weights_out/ENSG00000213714.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000213714.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000149636.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000213714.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000213714.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000213714.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000213714.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000213714.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000213714.1.bed +\n",
      "temp_files/test_chr20_ENSG00000213714.1.bim +\n",
      "temp_files/test_chr20_ENSG00000213714.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000213714.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000213714.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000213714.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000213714.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000213714.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000213714.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101407.8 --out compute_weights_out/ENSG00000101407.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101407.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000213714.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101407.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101407.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101407.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000101407.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101407.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101407.8.bed +\n",
      "temp_files/test_chr20_ENSG00000101407.8.bim +\n",
      "temp_files/test_chr20_ENSG00000101407.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101407.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101407.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101407.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101407.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101407.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101407.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000196839.8 --out compute_weights_out/ENSG00000196839.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000196839.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101407.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000196839.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000196839.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000196839.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000196839.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000196839.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000196839.8.bed +\n",
      "temp_files/test_chr20_ENSG00000196839.8.bim +\n",
      "temp_files/test_chr20_ENSG00000196839.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000196839.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000196839.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000196839.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000196839.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000196839.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000196839.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000196396.5 --out compute_weights_out/ENSG00000196396.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000196396.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000196839.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000196396.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000196396.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000196396.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000196396.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000196396.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000196396.5.bed +\n",
      "temp_files/test_chr20_ENSG00000196396.5.bim +\n",
      "temp_files/test_chr20_ENSG00000196396.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000196396.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000196396.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000196396.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000196396.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000196396.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000196396.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000060491.10 --out compute_weights_out/ENSG00000060491.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000060491.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000196396.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000060491.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000060491.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000060491.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000060491.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000060491.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000060491.10.bed +\n",
      "temp_files/test_chr20_ENSG00000060491.10.bim +\n",
      "temp_files/test_chr20_ENSG00000060491.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000060491.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000060491.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000060491.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000060491.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000060491.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000060491.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101084.12 --out compute_weights_out/ENSG00000101084.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101084.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000060491.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101084.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101084.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101084.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000101084.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101084.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101084.12.bed +\n",
      "temp_files/test_chr20_ENSG00000101084.12.bim +\n",
      "temp_files/test_chr20_ENSG00000101084.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101084.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101084.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101084.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101084.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101084.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101084.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101342.4 --out compute_weights_out/ENSG00000101342.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101342.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101084.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101342.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101342.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101342.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000101342.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101342.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101342.4.bed +\n",
      "temp_files/test_chr20_ENSG00000101342.4.bim +\n",
      "temp_files/test_chr20_ENSG00000101342.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101342.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101342.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101342.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101342.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101342.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101342.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000254806.1 --out compute_weights_out/ENSG00000254806.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000254806.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101342.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000254806.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000254806.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000254806.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000254806.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000254806.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000254806.1.bed +\n",
      "temp_files/test_chr20_ENSG00000254806.1.bim +\n",
      "temp_files/test_chr20_ENSG00000254806.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000254806.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000254806.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000254806.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000254806.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000254806.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000254806.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000064787.7 --out compute_weights_out/ENSG00000064787.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000064787.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000254806.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000064787.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000064787.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000064787.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000064787.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000064787.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000064787.7.bed +\n",
      "temp_files/test_chr20_ENSG00000064787.7.bim +\n",
      "temp_files/test_chr20_ENSG00000064787.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000064787.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000064787.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000064787.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000064787.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000064787.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000064787.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000088876.7 --out compute_weights_out/ENSG00000088876.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000088876.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000064787.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088876.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000088876.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000088876.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000088876.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088876.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000088876.7.bed +\n",
      "temp_files/test_chr20_ENSG00000088876.7.bim +\n",
      "temp_files/test_chr20_ENSG00000088876.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088876.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000088876.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000088876.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088876.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000088876.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000088876.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000226608.2 --out compute_weights_out/ENSG00000226608.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000226608.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000088876.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226608.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000226608.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000226608.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000226608.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226608.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000226608.2.bed +\n",
      "temp_files/test_chr20_ENSG00000226608.2.bim +\n",
      "temp_files/test_chr20_ENSG00000226608.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226608.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000226608.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000226608.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226608.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000226608.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000226608.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000061656.5 --out compute_weights_out/ENSG00000061656.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000061656.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000226608.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000061656.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000061656.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000061656.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000061656.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000061656.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000061656.5.bed +\n",
      "temp_files/test_chr20_ENSG00000061656.5.bim +\n",
      "temp_files/test_chr20_ENSG00000061656.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000061656.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000061656.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000061656.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000061656.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000061656.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000061656.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000196562.8 --out compute_weights_out/ENSG00000196562.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000196562.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000061656.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000196562.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000196562.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000196562.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000196562.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000196562.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000196562.8.bed +\n",
      "temp_files/test_chr20_ENSG00000196562.8.bim +\n",
      "temp_files/test_chr20_ENSG00000196562.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000196562.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000196562.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000196562.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000196562.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000196562.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000196562.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101193.5 --out compute_weights_out/ENSG00000101193.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101193.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000196562.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101193.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101193.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101193.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000101193.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101193.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101193.5.bed +\n",
      "temp_files/test_chr20_ENSG00000101193.5.bim +\n",
      "temp_files/test_chr20_ENSG00000101193.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101193.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101193.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101193.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101193.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101193.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101193.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000132664.7 --out compute_weights_out/ENSG00000132664.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000132664.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101193.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132664.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000132664.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000132664.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000132664.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132664.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000132664.7.bed +\n",
      "temp_files/test_chr20_ENSG00000132664.7.bim +\n",
      "temp_files/test_chr20_ENSG00000132664.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132664.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000132664.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000132664.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132664.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000132664.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000132664.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000236352.1 --out compute_weights_out/ENSG00000236352.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000236352.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000132664.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000236352.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000236352.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000236352.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000236352.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000236352.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000236352.1.bed +\n",
      "temp_files/test_chr20_ENSG00000236352.1.bim +\n",
      "temp_files/test_chr20_ENSG00000236352.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000236352.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000236352.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000236352.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000236352.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000236352.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000236352.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000226332.1 --out compute_weights_out/ENSG00000226332.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000226332.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000236352.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226332.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000226332.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000226332.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000226332.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226332.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000226332.1.bed +\n",
      "temp_files/test_chr20_ENSG00000226332.1.bim +\n",
      "temp_files/test_chr20_ENSG00000226332.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226332.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000226332.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000226332.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226332.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000226332.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000226332.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000198185.6 --out compute_weights_out/ENSG00000198185.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000198185.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000226332.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000198185.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000198185.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000198185.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000198185.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000198185.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000198185.6.bed +\n",
      "temp_files/test_chr20_ENSG00000198185.6.bim +\n",
      "temp_files/test_chr20_ENSG00000198185.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000198185.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000198185.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000198185.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000198185.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000198185.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000198185.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101216.6 --out compute_weights_out/ENSG00000101216.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101216.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000198185.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101216.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101216.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101216.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000101216.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101216.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101216.6.bed +\n",
      "temp_files/test_chr20_ENSG00000101216.6.bim +\n",
      "temp_files/test_chr20_ENSG00000101216.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101216.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101216.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101216.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101216.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101216.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101216.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000231302.1 --out compute_weights_out/ENSG00000231302.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000231302.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101216.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000231302.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000231302.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000231302.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000231302.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000231302.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000231302.1.bed +\n",
      "temp_files/test_chr20_ENSG00000231302.1.bim +\n",
      "temp_files/test_chr20_ENSG00000231302.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000231302.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000231302.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000231302.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000231302.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000231302.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000231302.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124120.6 --out compute_weights_out/ENSG00000124120.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124120.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000231302.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124120.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124120.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124120.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000124120.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124120.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124120.6.bed +\n",
      "temp_files/test_chr20_ENSG00000124120.6.bim +\n",
      "temp_files/test_chr20_ENSG00000124120.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124120.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124120.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124120.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124120.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124120.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124120.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000080189.9 --out compute_weights_out/ENSG00000080189.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000080189.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124120.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000080189.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000080189.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000080189.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000080189.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000080189.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000080189.9.bed +\n",
      "temp_files/test_chr20_ENSG00000080189.9.bim +\n",
      "temp_files/test_chr20_ENSG00000080189.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000080189.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000080189.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000080189.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000080189.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000080189.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000080189.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101224.13 --out compute_weights_out/ENSG00000101224.13_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101224.13.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000080189.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101224.13.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101224.13.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101224.13\n",
      "  --pheno ../data/gene_expressions/ENSG00000101224.13.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101224.13.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101224.13.bed +\n",
      "temp_files/test_chr20_ENSG00000101224.13.bim +\n",
      "temp_files/test_chr20_ENSG00000101224.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101224.13.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101224.13\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101224.13\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101224.13.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101224.13.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101224.13.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125514.5 --out compute_weights_out/ENSG00000125514.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125514.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101224.13 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125514.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125514.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125514.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000125514.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125514.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125514.5.bed +\n",
      "temp_files/test_chr20_ENSG00000125514.5.bim +\n",
      "temp_files/test_chr20_ENSG00000125514.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125514.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125514.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125514.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125514.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125514.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125514.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101363.8 --out compute_weights_out/ENSG00000101363.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101363.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125514.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101363.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101363.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101363.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000101363.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101363.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101363.8.bed +\n",
      "temp_files/test_chr20_ENSG00000101363.8.bim +\n",
      "temp_files/test_chr20_ENSG00000101363.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101363.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101363.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101363.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101363.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101363.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101363.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000132623.10 --out compute_weights_out/ENSG00000132623.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000132623.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101363.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132623.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000132623.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000132623.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000132623.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132623.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000132623.10.bed +\n",
      "temp_files/test_chr20_ENSG00000132623.10.bim +\n",
      "temp_files/test_chr20_ENSG00000132623.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132623.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000132623.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000132623.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132623.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000132623.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000132623.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000244687.5 --out compute_weights_out/ENSG00000244687.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000244687.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000132623.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000244687.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000244687.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000244687.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000244687.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000244687.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000244687.5.bed +\n",
      "temp_files/test_chr20_ENSG00000244687.5.bim +\n",
      "temp_files/test_chr20_ENSG00000244687.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000244687.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000244687.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000244687.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000244687.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000244687.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000244687.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000172296.8 --out compute_weights_out/ENSG00000172296.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000172296.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000244687.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000172296.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000172296.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000172296.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000172296.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000172296.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000172296.8.bed +\n",
      "temp_files/test_chr20_ENSG00000172296.8.bim +\n",
      "temp_files/test_chr20_ENSG00000172296.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000172296.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000172296.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000172296.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000172296.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000172296.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000172296.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000089123.10 --out compute_weights_out/ENSG00000089123.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000089123.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000172296.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000089123.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000089123.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000089123.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000089123.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000089123.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000089123.10.bed +\n",
      "temp_files/test_chr20_ENSG00000089123.10.bim +\n",
      "temp_files/test_chr20_ENSG00000089123.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000089123.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000089123.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000089123.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000089123.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000089123.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000089123.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000232442.1 --out compute_weights_out/ENSG00000232442.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000232442.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000089123.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000232442.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000232442.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000232442.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000232442.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000232442.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000232442.1.bed +\n",
      "temp_files/test_chr20_ENSG00000232442.1.bim +\n",
      "temp_files/test_chr20_ENSG00000232442.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000232442.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000232442.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000232442.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000232442.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000232442.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000232442.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101132.5 --out compute_weights_out/ENSG00000101132.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101132.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000232442.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101132.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101132.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101132.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000101132.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101132.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101132.5.bed +\n",
      "temp_files/test_chr20_ENSG00000101132.5.bim +\n",
      "temp_files/test_chr20_ENSG00000101132.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101132.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101132.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101132.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101132.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101132.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101132.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000215388.3 --out compute_weights_out/ENSG00000215388.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000215388.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101132.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000215388.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000215388.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000215388.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000215388.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000215388.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000215388.3.bed +\n",
      "temp_files/test_chr20_ENSG00000215388.3.bim +\n",
      "temp_files/test_chr20_ENSG00000215388.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000215388.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000215388.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000215388.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000215388.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000215388.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000215388.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000235621.3 --out compute_weights_out/ENSG00000235621.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000235621.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000215388.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000235621.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000235621.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000235621.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000235621.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000235621.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000235621.3.bed +\n",
      "temp_files/test_chr20_ENSG00000235621.3.bim +\n",
      "temp_files/test_chr20_ENSG00000235621.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000235621.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000235621.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000235621.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000235621.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000235621.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000235621.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000197114.6 --out compute_weights_out/ENSG00000197114.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000197114.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000235621.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000197114.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000197114.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000197114.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000197114.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000197114.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000197114.6.bed +\n",
      "temp_files/test_chr20_ENSG00000197114.6.bim +\n",
      "temp_files/test_chr20_ENSG00000197114.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000197114.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000197114.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000197114.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000197114.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000197114.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000197114.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124126.9 --out compute_weights_out/ENSG00000124126.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124126.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000197114.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124126.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124126.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124126.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000124126.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124126.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124126.9.bed +\n",
      "temp_files/test_chr20_ENSG00000124126.9.bim +\n",
      "temp_files/test_chr20_ENSG00000124126.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124126.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124126.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124126.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124126.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124126.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124126.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000215452.3 --out compute_weights_out/ENSG00000215452.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000215452.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124126.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000215452.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000215452.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000215452.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000215452.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000215452.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000215452.3.bed +\n",
      "temp_files/test_chr20_ENSG00000215452.3.bim +\n",
      "temp_files/test_chr20_ENSG00000215452.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000215452.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000215452.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000215452.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000215452.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000215452.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000215452.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124097.7 --out compute_weights_out/ENSG00000124097.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124097.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000215452.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124097.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124097.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124097.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000124097.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124097.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124097.7.bed +\n",
      "temp_files/test_chr20_ENSG00000124097.7.bim +\n",
      "temp_files/test_chr20_ENSG00000124097.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124097.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124097.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124097.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124097.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124097.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124097.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101452.8 --out compute_weights_out/ENSG00000101452.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101452.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124097.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101452.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101452.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101452.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000101452.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101452.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101452.8.bed +\n",
      "temp_files/test_chr20_ENSG00000101452.8.bim +\n",
      "temp_files/test_chr20_ENSG00000101452.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101452.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101452.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101452.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101452.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101452.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101452.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000149656.4 --out compute_weights_out/ENSG00000149656.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000149656.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101452.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149656.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000149656.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000149656.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000149656.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149656.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000149656.4.bed +\n",
      "temp_files/test_chr20_ENSG00000149656.4.bim +\n",
      "temp_files/test_chr20_ENSG00000149656.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149656.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000149656.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000149656.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149656.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000149656.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000149656.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000088367.14 --out compute_weights_out/ENSG00000088367.14_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000088367.14.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000149656.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088367.14.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000088367.14.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000088367.14\n",
      "  --pheno ../data/gene_expressions/ENSG00000088367.14.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088367.14.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000088367.14.bed +\n",
      "temp_files/test_chr20_ENSG00000088367.14.bim +\n",
      "temp_files/test_chr20_ENSG00000088367.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088367.14.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000088367.14\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000088367.14\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088367.14.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000088367.14.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000088367.14.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000234684.1 --out compute_weights_out/ENSG00000234684.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000234684.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000088367.14 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000234684.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000234684.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000234684.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000234684.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000234684.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000234684.1.bed +\n",
      "temp_files/test_chr20_ENSG00000234684.1.bim +\n",
      "temp_files/test_chr20_ENSG00000234684.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000234684.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000234684.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000234684.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000234684.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000234684.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000234684.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000225806.1 --out compute_weights_out/ENSG00000225806.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000225806.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000234684.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225806.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000225806.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000225806.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000225806.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225806.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000225806.1.bed +\n",
      "temp_files/test_chr20_ENSG00000225806.1.bim +\n",
      "temp_files/test_chr20_ENSG00000225806.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225806.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000225806.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000225806.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225806.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000225806.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000225806.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000215467.2 --out compute_weights_out/ENSG00000215467.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000215467.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000225806.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000215467.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000215467.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000215467.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000215467.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000215467.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000215467.2.bed +\n",
      "temp_files/test_chr20_ENSG00000215467.2.bim +\n",
      "temp_files/test_chr20_ENSG00000215467.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000215467.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000215467.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000215467.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000215467.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000215467.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000215467.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000088832.10 --out compute_weights_out/ENSG00000088832.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000088832.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000215467.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088832.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000088832.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000088832.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000088832.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088832.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000088832.10.bed +\n",
      "temp_files/test_chr20_ENSG00000088832.10.bim +\n",
      "temp_files/test_chr20_ENSG00000088832.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088832.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000088832.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000088832.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088832.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000088832.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000088832.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124194.10 --out compute_weights_out/ENSG00000124194.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124194.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000088832.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124194.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124194.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124194.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000124194.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124194.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124194.10.bed +\n",
      "temp_files/test_chr20_ENSG00000124194.10.bim +\n",
      "temp_files/test_chr20_ENSG00000124194.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124194.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124194.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124194.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124194.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124194.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124194.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000158470.5 --out compute_weights_out/ENSG00000158470.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000158470.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124194.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000158470.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000158470.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000158470.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000158470.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000158470.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000158470.5.bed +\n",
      "temp_files/test_chr20_ENSG00000158470.5.bim +\n",
      "temp_files/test_chr20_ENSG00000158470.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000158470.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000158470.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000158470.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000158470.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000158470.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000158470.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125814.12 --out compute_weights_out/ENSG00000125814.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125814.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000158470.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125814.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125814.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125814.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000125814.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125814.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125814.12.bed +\n",
      "temp_files/test_chr20_ENSG00000125814.12.bim +\n",
      "temp_files/test_chr20_ENSG00000125814.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125814.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125814.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125814.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125814.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125814.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125814.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000075043.12 --out compute_weights_out/ENSG00000075043.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000075043.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125814.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000075043.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000075043.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000075043.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000075043.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000075043.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000075043.12.bed +\n",
      "temp_files/test_chr20_ENSG00000075043.12.bim +\n",
      "temp_files/test_chr20_ENSG00000075043.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000075043.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000075043.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000075043.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000075043.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000075043.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000075043.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000234747.1 --out compute_weights_out/ENSG00000234747.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000234747.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000075043.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000234747.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000234747.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000234747.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000234747.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000234747.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000234747.1.bed +\n",
      "temp_files/test_chr20_ENSG00000234747.1.bim +\n",
      "temp_files/test_chr20_ENSG00000234747.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000234747.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000234747.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000234747.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000234747.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000234747.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000234747.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000226948.1 --out compute_weights_out/ENSG00000226948.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000226948.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000234747.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226948.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000226948.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000226948.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000226948.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226948.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000226948.1.bed +\n",
      "temp_files/test_chr20_ENSG00000226948.1.bim +\n",
      "temp_files/test_chr20_ENSG00000226948.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226948.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000226948.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000226948.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226948.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000226948.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000226948.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000000419.7 --out compute_weights_out/ENSG00000000419.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000000419.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000226948.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000000419.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000000419.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000000419.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000000419.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000000419.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000000419.7.bed +\n",
      "temp_files/test_chr20_ENSG00000000419.7.bim +\n",
      "temp_files/test_chr20_ENSG00000000419.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000000419.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000000419.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000000419.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000000419.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000000419.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000000419.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101335.5 --out compute_weights_out/ENSG00000101335.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101335.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000000419.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101335.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101335.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101335.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000101335.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101335.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101335.5.bed +\n",
      "temp_files/test_chr20_ENSG00000101335.5.bim +\n",
      "temp_files/test_chr20_ENSG00000101335.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101335.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101335.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101335.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101335.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101335.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101335.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124191.11 --out compute_weights_out/ENSG00000124191.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124191.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101335.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124191.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124191.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124191.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000124191.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124191.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124191.11.bed +\n",
      "temp_files/test_chr20_ENSG00000124191.11.bim +\n",
      "temp_files/test_chr20_ENSG00000124191.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124191.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124191.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124191.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124191.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124191.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124191.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101236.11 --out compute_weights_out/ENSG00000101236.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101236.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124191.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101236.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101236.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101236.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000101236.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101236.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101236.11.bed +\n",
      "temp_files/test_chr20_ENSG00000101236.11.bim +\n",
      "temp_files/test_chr20_ENSG00000101236.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101236.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101236.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101236.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101236.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101236.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101236.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000213820.3 --out compute_weights_out/ENSG00000213820.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000213820.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101236.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000213820.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000213820.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000213820.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000213820.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000213820.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000213820.3.bed +\n",
      "temp_files/test_chr20_ENSG00000213820.3.bim +\n",
      "temp_files/test_chr20_ENSG00000213820.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000213820.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000213820.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000213820.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000213820.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000213820.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000213820.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000089177.12 --out compute_weights_out/ENSG00000089177.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000089177.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000213820.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000089177.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000089177.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000089177.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000089177.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000089177.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000089177.12.bed +\n",
      "temp_files/test_chr20_ENSG00000089177.12.bim +\n",
      "temp_files/test_chr20_ENSG00000089177.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000089177.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000089177.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000089177.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000089177.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000089177.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000089177.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124201.9 --out compute_weights_out/ENSG00000124201.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124201.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000089177.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124201.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124201.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124201.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000124201.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124201.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124201.9.bed +\n",
      "temp_files/test_chr20_ENSG00000124201.9.bim +\n",
      "temp_files/test_chr20_ENSG00000124201.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124201.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124201.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124201.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124201.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124201.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124201.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000182621.11 --out compute_weights_out/ENSG00000182621.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000182621.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124201.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000182621.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000182621.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000182621.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000182621.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000182621.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000182621.11.bed +\n",
      "temp_files/test_chr20_ENSG00000182621.11.bim +\n",
      "temp_files/test_chr20_ENSG00000182621.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000182621.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000182621.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000182621.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000182621.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000182621.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000182621.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101337.10 --out compute_weights_out/ENSG00000101337.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101337.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000182621.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101337.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101337.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101337.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000101337.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101337.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101337.10.bed +\n",
      "temp_files/test_chr20_ENSG00000101337.10.bim +\n",
      "temp_files/test_chr20_ENSG00000101337.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101337.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101337.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101337.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101337.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101337.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101337.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000231290.1 --out compute_weights_out/ENSG00000231290.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000231290.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101337.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000231290.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000231290.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000231290.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000231290.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000231290.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000231290.1.bed +\n",
      "temp_files/test_chr20_ENSG00000231290.1.bim +\n",
      "temp_files/test_chr20_ENSG00000231290.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000231290.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000231290.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000231290.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000231290.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000231290.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000231290.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101049.10 --out compute_weights_out/ENSG00000101049.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101049.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000231290.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101049.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101049.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101049.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000101049.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101049.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101049.10.bed +\n",
      "temp_files/test_chr20_ENSG00000101049.10.bim +\n",
      "temp_files/test_chr20_ENSG00000101049.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101049.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101049.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101049.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101049.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101049.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101049.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000226644.1 --out compute_weights_out/ENSG00000226644.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000226644.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101049.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226644.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000226644.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000226644.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000226644.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226644.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000226644.1.bed +\n",
      "temp_files/test_chr20_ENSG00000226644.1.bim +\n",
      "temp_files/test_chr20_ENSG00000226644.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226644.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000226644.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000226644.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226644.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000226644.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000226644.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000088356.4 --out compute_weights_out/ENSG00000088356.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000088356.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000226644.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088356.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000088356.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000088356.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000088356.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088356.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000088356.4.bed +\n",
      "temp_files/test_chr20_ENSG00000088356.4.bim +\n",
      "temp_files/test_chr20_ENSG00000088356.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088356.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000088356.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000088356.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088356.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000088356.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000088356.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000233838.4 --out compute_weights_out/ENSG00000233838.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000233838.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000088356.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000233838.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000233838.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000233838.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000233838.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000233838.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000233838.4.bed +\n",
      "temp_files/test_chr20_ENSG00000233838.4.bim +\n",
      "temp_files/test_chr20_ENSG00000233838.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000233838.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000233838.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000233838.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000233838.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000233838.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000233838.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124145.5 --out compute_weights_out/ENSG00000124145.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124145.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000233838.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124145.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124145.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124145.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000124145.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124145.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124145.5.bed +\n",
      "temp_files/test_chr20_ENSG00000124145.5.bim +\n",
      "temp_files/test_chr20_ENSG00000124145.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124145.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124145.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124145.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124145.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124145.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124145.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000236992.1 --out compute_weights_out/ENSG00000236992.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000236992.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124145.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000236992.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000236992.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000236992.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000236992.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000236992.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000236992.1.bed +\n",
      "temp_files/test_chr20_ENSG00000236992.1.bim +\n",
      "temp_files/test_chr20_ENSG00000236992.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000236992.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000236992.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000236992.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000236992.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000236992.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000236992.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000197183.7 --out compute_weights_out/ENSG00000197183.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000197183.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000236992.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000197183.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000197183.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000197183.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000197183.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000197183.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000197183.7.bed +\n",
      "temp_files/test_chr20_ENSG00000197183.7.bim +\n",
      "temp_files/test_chr20_ENSG00000197183.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000197183.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000197183.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000197183.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000197183.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000197183.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000197183.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101255.6 --out compute_weights_out/ENSG00000101255.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101255.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000197183.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101255.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101255.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101255.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000101255.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101255.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101255.6.bed +\n",
      "temp_files/test_chr20_ENSG00000101255.6.bim +\n",
      "temp_files/test_chr20_ENSG00000101255.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101255.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101255.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101255.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101255.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101255.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101255.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000089050.9 --out compute_weights_out/ENSG00000089050.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000089050.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101255.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000089050.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000089050.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000089050.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000089050.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000089050.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000089050.9.bed +\n",
      "temp_files/test_chr20_ENSG00000089050.9.bim +\n",
      "temp_files/test_chr20_ENSG00000089050.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000089050.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000089050.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000089050.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000089050.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000089050.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000089050.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000179935.5 --out compute_weights_out/ENSG00000179935.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000179935.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000089050.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000179935.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000179935.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000179935.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000179935.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000179935.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000179935.5.bed +\n",
      "temp_files/test_chr20_ENSG00000179935.5.bim +\n",
      "temp_files/test_chr20_ENSG00000179935.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000179935.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000179935.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000179935.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000179935.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000179935.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000179935.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000088882.6 --out compute_weights_out/ENSG00000088882.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000088882.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000179935.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088882.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000088882.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000088882.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000088882.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088882.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000088882.6.bed +\n",
      "temp_files/test_chr20_ENSG00000088882.6.bim +\n",
      "temp_files/test_chr20_ENSG00000088882.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088882.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000088882.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000088882.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088882.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000088882.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000088882.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101353.10 --out compute_weights_out/ENSG00000101353.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101353.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000088882.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101353.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101353.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101353.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000101353.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101353.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101353.10.bed +\n",
      "temp_files/test_chr20_ENSG00000101353.10.bim +\n",
      "temp_files/test_chr20_ENSG00000101353.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101353.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101353.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101353.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101353.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101353.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101353.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124160.7 --out compute_weights_out/ENSG00000124160.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124160.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101353.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124160.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124160.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124160.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000124160.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124160.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124160.7.bed +\n",
      "temp_files/test_chr20_ENSG00000124160.7.bim +\n",
      "temp_files/test_chr20_ENSG00000124160.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124160.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124160.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124160.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124160.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124160.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124160.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000231119.2 --out compute_weights_out/ENSG00000231119.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000231119.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124160.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000231119.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000231119.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000231119.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000231119.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000231119.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000231119.2.bed +\n",
      "temp_files/test_chr20_ENSG00000231119.2.bim +\n",
      "temp_files/test_chr20_ENSG00000231119.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000231119.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000231119.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000231119.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000231119.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000231119.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000231119.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000130699.10 --out compute_weights_out/ENSG00000130699.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000130699.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000231119.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000130699.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000130699.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000130699.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000130699.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000130699.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000130699.10.bed +\n",
      "temp_files/test_chr20_ENSG00000130699.10.bim +\n",
      "temp_files/test_chr20_ENSG00000130699.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000130699.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000130699.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000130699.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000130699.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000130699.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000130699.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000214535.3 --out compute_weights_out/ENSG00000214535.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000214535.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000130699.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000214535.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000214535.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000214535.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000214535.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000214535.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000214535.3.bed +\n",
      "temp_files/test_chr20_ENSG00000214535.3.bim +\n",
      "temp_files/test_chr20_ENSG00000214535.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000214535.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000214535.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000214535.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000214535.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000214535.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000214535.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000198053.7 --out compute_weights_out/ENSG00000198053.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000198053.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000214535.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000198053.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000198053.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000198053.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000198053.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000198053.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000198053.7.bed +\n",
      "temp_files/test_chr20_ENSG00000198053.7.bim +\n",
      "temp_files/test_chr20_ENSG00000198053.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000198053.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000198053.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000198053.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000198053.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000198053.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000198053.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000185513.9 --out compute_weights_out/ENSG00000185513.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000185513.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000198053.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000185513.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000185513.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000185513.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000185513.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000185513.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000185513.9.bed +\n",
      "temp_files/test_chr20_ENSG00000185513.9.bim +\n",
      "temp_files/test_chr20_ENSG00000185513.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000185513.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000185513.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000185513.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000185513.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000185513.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000185513.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125818.10 --out compute_weights_out/ENSG00000125818.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125818.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000185513.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125818.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125818.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125818.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000125818.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125818.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125818.10.bed +\n",
      "temp_files/test_chr20_ENSG00000125818.10.bim +\n",
      "temp_files/test_chr20_ENSG00000125818.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125818.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125818.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125818.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125818.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125818.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125818.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124203.5 --out compute_weights_out/ENSG00000124203.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124203.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125818.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124203.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124203.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124203.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000124203.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124203.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124203.5.bed +\n",
      "temp_files/test_chr20_ENSG00000124203.5.bim +\n",
      "temp_files/test_chr20_ENSG00000124203.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124203.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124203.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124203.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124203.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124203.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124203.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000196227.6 --out compute_weights_out/ENSG00000196227.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000196227.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124203.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000196227.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000196227.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000196227.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000196227.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000196227.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000196227.6.bed +\n",
      "temp_files/test_chr20_ENSG00000196227.6.bim +\n",
      "temp_files/test_chr20_ENSG00000196227.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000196227.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000196227.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000196227.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000196227.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000196227.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000196227.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125991.12 --out compute_weights_out/ENSG00000125991.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125991.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000196227.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125991.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125991.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125991.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000125991.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125991.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125991.12.bed +\n",
      "temp_files/test_chr20_ENSG00000125991.12.bim +\n",
      "temp_files/test_chr20_ENSG00000125991.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125991.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125991.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125991.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125991.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125991.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125991.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000188559.7 --out compute_weights_out/ENSG00000188559.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000188559.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125991.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000188559.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000188559.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000188559.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000188559.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000188559.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000188559.7.bed +\n",
      "temp_files/test_chr20_ENSG00000188559.7.bim +\n",
      "temp_files/test_chr20_ENSG00000188559.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000188559.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000188559.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000188559.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000188559.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000188559.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000188559.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000204556.4 --out compute_weights_out/ENSG00000204556.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000204556.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000188559.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000204556.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000204556.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000204556.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000204556.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000204556.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000204556.4.bed +\n",
      "temp_files/test_chr20_ENSG00000204556.4.bim +\n",
      "temp_files/test_chr20_ENSG00000204556.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000204556.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000204556.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000204556.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000204556.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000204556.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000204556.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000062598.12 --out compute_weights_out/ENSG00000062598.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000062598.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000204556.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000062598.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000062598.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000062598.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000062598.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000062598.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000062598.12.bed +\n",
      "temp_files/test_chr20_ENSG00000062598.12.bim +\n",
      "temp_files/test_chr20_ENSG00000062598.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000062598.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000062598.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000062598.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000062598.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000062598.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000062598.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125977.6 --out compute_weights_out/ENSG00000125977.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125977.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000062598.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125977.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125977.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125977.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000125977.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125977.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125977.6.bed +\n",
      "temp_files/test_chr20_ENSG00000125977.6.bim +\n",
      "temp_files/test_chr20_ENSG00000125977.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125977.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125977.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125977.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125977.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125977.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125977.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125510.11 --out compute_weights_out/ENSG00000125510.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125510.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125977.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125510.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125510.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125510.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000125510.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125510.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125510.11.bed +\n",
      "temp_files/test_chr20_ENSG00000125510.11.bim +\n",
      "temp_files/test_chr20_ENSG00000125510.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125510.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125510.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125510.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125510.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125510.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125510.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101052.8 --out compute_weights_out/ENSG00000101052.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101052.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125510.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101052.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101052.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101052.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000101052.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101052.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101052.8.bed +\n",
      "temp_files/test_chr20_ENSG00000101052.8.bim +\n",
      "temp_files/test_chr20_ENSG00000101052.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101052.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101052.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101052.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101052.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101052.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101052.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000225490.1 --out compute_weights_out/ENSG00000225490.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000225490.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101052.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225490.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000225490.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000225490.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000225490.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225490.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000225490.1.bed +\n",
      "temp_files/test_chr20_ENSG00000225490.1.bim +\n",
      "temp_files/test_chr20_ENSG00000225490.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225490.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000225490.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000225490.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225490.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000225490.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000225490.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000240849.6 --out compute_weights_out/ENSG00000240849.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000240849.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000225490.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000240849.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000240849.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000240849.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000240849.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000240849.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000240849.6.bed +\n",
      "temp_files/test_chr20_ENSG00000240849.6.bim +\n",
      "temp_files/test_chr20_ENSG00000240849.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000240849.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000240849.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000240849.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000240849.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000240849.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000240849.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101204.10 --out compute_weights_out/ENSG00000101204.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101204.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000240849.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101204.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101204.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101204.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000101204.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101204.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101204.10.bed +\n",
      "temp_files/test_chr20_ENSG00000101204.10.bim +\n",
      "temp_files/test_chr20_ENSG00000101204.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101204.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101204.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101204.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101204.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101204.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101204.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125895.5 --out compute_weights_out/ENSG00000125895.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125895.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101204.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125895.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125895.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125895.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000125895.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125895.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125895.5.bed +\n",
      "temp_files/test_chr20_ENSG00000125895.5.bim +\n",
      "temp_files/test_chr20_ENSG00000125895.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125895.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125895.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125895.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125895.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125895.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125895.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000237119.1 --out compute_weights_out/ENSG00000237119.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000237119.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125895.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000237119.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000237119.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000237119.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000237119.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000237119.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000237119.1.bed +\n",
      "temp_files/test_chr20_ENSG00000237119.1.bim +\n",
      "temp_files/test_chr20_ENSG00000237119.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000237119.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000237119.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000237119.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000237119.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000237119.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000237119.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000232493.2 --out compute_weights_out/ENSG00000232493.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000232493.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000237119.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000232493.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000232493.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000232493.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000232493.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000232493.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000232493.2.bed +\n",
      "temp_files/test_chr20_ENSG00000232493.2.bim +\n",
      "temp_files/test_chr20_ENSG00000232493.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000232493.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000232493.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000232493.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000232493.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000232493.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000232493.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124253.8 --out compute_weights_out/ENSG00000124253.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124253.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000232493.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124253.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124253.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124253.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000124253.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124253.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124253.8.bed +\n",
      "temp_files/test_chr20_ENSG00000124253.8.bim +\n",
      "temp_files/test_chr20_ENSG00000124253.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124253.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124253.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124253.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124253.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124253.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124253.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000171456.11 --out compute_weights_out/ENSG00000171456.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000171456.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124253.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000171456.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000171456.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000171456.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000171456.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000171456.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000171456.11.bed +\n",
      "temp_files/test_chr20_ENSG00000171456.11.bim +\n",
      "temp_files/test_chr20_ENSG00000171456.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000171456.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000171456.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000171456.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000171456.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000171456.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000171456.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000226648.1 --out compute_weights_out/ENSG00000226648.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000226648.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000171456.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226648.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000226648.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000226648.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000226648.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226648.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000226648.1.bed +\n",
      "temp_files/test_chr20_ENSG00000226648.1.bim +\n",
      "temp_files/test_chr20_ENSG00000226648.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226648.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000226648.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000226648.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226648.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000226648.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000226648.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101220.11 --out compute_weights_out/ENSG00000101220.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101220.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000226648.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101220.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101220.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101220.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000101220.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101220.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101220.11.bed +\n",
      "temp_files/test_chr20_ENSG00000101220.11.bim +\n",
      "temp_files/test_chr20_ENSG00000101220.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101220.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101220.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101220.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101220.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101220.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101220.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101126.9 --out compute_weights_out/ENSG00000101126.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101126.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101220.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101126.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101126.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101126.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000101126.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101126.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101126.9.bed +\n",
      "temp_files/test_chr20_ENSG00000101126.9.bim +\n",
      "temp_files/test_chr20_ENSG00000101126.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101126.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101126.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101126.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101126.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101126.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101126.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000087495.12 --out compute_weights_out/ENSG00000087495.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000087495.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101126.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000087495.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000087495.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000087495.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000087495.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000087495.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000087495.12.bed +\n",
      "temp_files/test_chr20_ENSG00000087495.12.bim +\n",
      "temp_files/test_chr20_ENSG00000087495.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000087495.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000087495.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000087495.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000087495.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000087495.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000087495.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000132792.14 --out compute_weights_out/ENSG00000132792.14_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000132792.14.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000087495.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132792.14.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000132792.14.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000132792.14\n",
      "  --pheno ../data/gene_expressions/ENSG00000132792.14.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132792.14.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000132792.14.bed +\n",
      "temp_files/test_chr20_ENSG00000132792.14.bim +\n",
      "temp_files/test_chr20_ENSG00000132792.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132792.14.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000132792.14\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000132792.14\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132792.14.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000132792.14.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000132792.14.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000230613.1 --out compute_weights_out/ENSG00000230613.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000230613.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000132792.14 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000230613.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000230613.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000230613.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000230613.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000230613.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000230613.1.bed +\n",
      "temp_files/test_chr20_ENSG00000230613.1.bim +\n",
      "temp_files/test_chr20_ENSG00000230613.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000230613.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000230613.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000230613.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000230613.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000230613.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000230613.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000215586.2 --out compute_weights_out/ENSG00000215586.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000215586.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000230613.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000215586.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000215586.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000215586.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000215586.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000215586.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000215586.2.bed +\n",
      "temp_files/test_chr20_ENSG00000215586.2.bim +\n",
      "temp_files/test_chr20_ENSG00000215586.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000215586.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000215586.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000215586.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000215586.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000215586.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000215586.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000244462.2 --out compute_weights_out/ENSG00000244462.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000244462.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000215586.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000244462.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000244462.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000244462.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000244462.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000244462.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000244462.2.bed +\n",
      "temp_files/test_chr20_ENSG00000244462.2.bim +\n",
      "temp_files/test_chr20_ENSG00000244462.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000244462.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000244462.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000244462.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000244462.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000244462.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000244462.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101079.15 --out compute_weights_out/ENSG00000101079.15_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101079.15.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000244462.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101079.15.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101079.15.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101079.15\n",
      "  --pheno ../data/gene_expressions/ENSG00000101079.15.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101079.15.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101079.15.bed +\n",
      "temp_files/test_chr20_ENSG00000101079.15.bim +\n",
      "temp_files/test_chr20_ENSG00000101079.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101079.15.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101079.15\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101079.15\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101079.15.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101079.15.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101079.15.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000088930.6 --out compute_weights_out/ENSG00000088930.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000088930.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101079.15 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088930.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000088930.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000088930.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000088930.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088930.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000088930.6.bed +\n",
      "temp_files/test_chr20_ENSG00000088930.6.bim +\n",
      "temp_files/test_chr20_ENSG00000088930.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088930.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000088930.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000088930.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088930.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000088930.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000088930.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101898.5 --out compute_weights_out/ENSG00000101898.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101898.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000088930.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101898.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101898.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101898.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000101898.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101898.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101898.5.bed +\n",
      "temp_files/test_chr20_ENSG00000101898.5.bim +\n",
      "temp_files/test_chr20_ENSG00000101898.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101898.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101898.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101898.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101898.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101898.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101898.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124151.11 --out compute_weights_out/ENSG00000124151.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124151.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101898.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124151.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124151.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124151.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000124151.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124151.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124151.11.bed +\n",
      "temp_files/test_chr20_ENSG00000124151.11.bim +\n",
      "temp_files/test_chr20_ENSG00000124151.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124151.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124151.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124151.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124151.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124151.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124151.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000131051.15 --out compute_weights_out/ENSG00000131051.15_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000131051.15.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124151.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000131051.15.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000131051.15.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000131051.15\n",
      "  --pheno ../data/gene_expressions/ENSG00000131051.15.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000131051.15.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000131051.15.bed +\n",
      "temp_files/test_chr20_ENSG00000131051.15.bim +\n",
      "temp_files/test_chr20_ENSG00000131051.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000131051.15.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000131051.15\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000131051.15\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000131051.15.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000131051.15.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000131051.15.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101298.9 --out compute_weights_out/ENSG00000101298.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101298.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000131051.15 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101298.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101298.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101298.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000101298.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101298.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101298.9.bed +\n",
      "temp_files/test_chr20_ENSG00000101298.9.bim +\n",
      "temp_files/test_chr20_ENSG00000101298.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101298.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101298.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101298.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101298.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101298.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101298.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000225357.2 --out compute_weights_out/ENSG00000225357.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000225357.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101298.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225357.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000225357.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000225357.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000225357.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225357.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000225357.2.bed +\n",
      "temp_files/test_chr20_ENSG00000225357.2.bim +\n",
      "temp_files/test_chr20_ENSG00000225357.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225357.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000225357.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000225357.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225357.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000225357.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000225357.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000203951.2 --out compute_weights_out/ENSG00000203951.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000203951.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000225357.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000203951.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000203951.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000203951.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000203951.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000203951.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000203951.2.bed +\n",
      "temp_files/test_chr20_ENSG00000203951.2.bim +\n",
      "temp_files/test_chr20_ENSG00000203951.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000203951.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000203951.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000203951.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000203951.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000203951.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000203951.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000218018.2 --out compute_weights_out/ENSG00000218018.2_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000218018.2.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000203951.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000218018.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000218018.2.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000218018.2\n",
      "  --pheno ../data/gene_expressions/ENSG00000218018.2.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000218018.2.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000218018.2.bed +\n",
      "temp_files/test_chr20_ENSG00000218018.2.bim +\n",
      "temp_files/test_chr20_ENSG00000218018.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000218018.2.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000218018.2\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000218018.2\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000218018.2.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000218018.2.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000218018.2.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124216.3 --out compute_weights_out/ENSG00000124216.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124216.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000218018.2 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124216.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124216.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124216.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000124216.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124216.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124216.3.bed +\n",
      "temp_files/test_chr20_ENSG00000124216.3.bim +\n",
      "temp_files/test_chr20_ENSG00000124216.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124216.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124216.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124216.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124216.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124216.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124216.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125826.14 --out compute_weights_out/ENSG00000125826.14_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125826.14.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124216.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125826.14.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125826.14.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125826.14\n",
      "  --pheno ../data/gene_expressions/ENSG00000125826.14.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125826.14.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125826.14.bed +\n",
      "temp_files/test_chr20_ENSG00000125826.14.bim +\n",
      "temp_files/test_chr20_ENSG00000125826.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125826.14.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125826.14\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125826.14\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125826.14.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125826.14.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125826.14.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000183798.4 --out compute_weights_out/ENSG00000183798.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000183798.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125826.14 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000183798.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000183798.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000183798.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000183798.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000183798.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000183798.4.bed +\n",
      "temp_files/test_chr20_ENSG00000183798.4.bim +\n",
      "temp_files/test_chr20_ENSG00000183798.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000183798.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000183798.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000183798.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000183798.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000183798.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000183798.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000232852.1 --out compute_weights_out/ENSG00000232852.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000232852.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000183798.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000232852.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000232852.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000232852.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000232852.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000232852.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000232852.1.bed +\n",
      "temp_files/test_chr20_ENSG00000232852.1.bim +\n",
      "temp_files/test_chr20_ENSG00000232852.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000232852.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000232852.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000232852.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000232852.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000232852.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000232852.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000184617.9 --out compute_weights_out/ENSG00000184617.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000184617.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000232852.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000184617.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000184617.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000184617.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000184617.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000184617.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000184617.9.bed +\n",
      "temp_files/test_chr20_ENSG00000184617.9.bim +\n",
      "temp_files/test_chr20_ENSG00000184617.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000184617.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000184617.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000184617.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000184617.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000184617.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000184617.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000149488.11 --out compute_weights_out/ENSG00000149488.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000149488.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000184617.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149488.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000149488.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000149488.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000149488.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149488.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000149488.11.bed +\n",
      "temp_files/test_chr20_ENSG00000149488.11.bim +\n",
      "temp_files/test_chr20_ENSG00000149488.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149488.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000149488.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000149488.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149488.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000149488.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000149488.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101384.6 --out compute_weights_out/ENSG00000101384.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101384.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000149488.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101384.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101384.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101384.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000101384.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101384.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101384.6.bed +\n",
      "temp_files/test_chr20_ENSG00000101384.6.bim +\n",
      "temp_files/test_chr20_ENSG00000101384.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101384.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101384.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101384.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101384.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101384.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101384.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000132801.5 --out compute_weights_out/ENSG00000132801.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000132801.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101384.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132801.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000132801.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000132801.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000132801.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132801.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000132801.5.bed +\n",
      "temp_files/test_chr20_ENSG00000132801.5.bim +\n",
      "temp_files/test_chr20_ENSG00000132801.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000132801.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000132801.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000132801.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000132801.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000132801.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000132801.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101003.8 --out compute_weights_out/ENSG00000101003.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101003.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000132801.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101003.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101003.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101003.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000101003.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101003.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101003.8.bed +\n",
      "temp_files/test_chr20_ENSG00000101003.8.bim +\n",
      "temp_files/test_chr20_ENSG00000101003.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101003.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101003.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101003.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101003.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101003.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101003.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124226.6 --out compute_weights_out/ENSG00000124226.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124226.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101003.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124226.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124226.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124226.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000124226.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124226.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124226.6.bed +\n",
      "temp_files/test_chr20_ENSG00000124226.6.bim +\n",
      "temp_files/test_chr20_ENSG00000124226.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124226.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124226.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124226.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124226.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124226.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124226.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000198900.5 --out compute_weights_out/ENSG00000198900.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000198900.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124226.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000198900.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000198900.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000198900.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000198900.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000198900.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000198900.5.bed +\n",
      "temp_files/test_chr20_ENSG00000198900.5.bim +\n",
      "temp_files/test_chr20_ENSG00000198900.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000198900.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000198900.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000198900.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000198900.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000198900.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000198900.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000154930.9 --out compute_weights_out/ENSG00000154930.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000154930.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000198900.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000154930.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000154930.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000154930.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000154930.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000154930.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000154930.9.bed +\n",
      "temp_files/test_chr20_ENSG00000154930.9.bim +\n",
      "temp_files/test_chr20_ENSG00000154930.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000154930.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000154930.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000154930.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000154930.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000154930.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000154930.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124224.11 --out compute_weights_out/ENSG00000124224.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124224.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000154930.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124224.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124224.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124224.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000124224.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124224.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124224.11.bed +\n",
      "temp_files/test_chr20_ENSG00000124224.11.bim +\n",
      "temp_files/test_chr20_ENSG00000124224.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124224.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124224.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124224.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124224.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124224.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124224.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101473.11 --out compute_weights_out/ENSG00000101473.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101473.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124224.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101473.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101473.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101473.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000101473.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101473.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101473.11.bed +\n",
      "temp_files/test_chr20_ENSG00000101473.11.bim +\n",
      "temp_files/test_chr20_ENSG00000101473.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101473.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101473.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101473.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101473.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101473.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101473.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101361.9 --out compute_weights_out/ENSG00000101361.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101361.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101473.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101361.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101361.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101361.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000101361.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101361.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101361.9.bed +\n",
      "temp_files/test_chr20_ENSG00000101361.9.bim +\n",
      "temp_files/test_chr20_ENSG00000101361.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101361.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101361.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101361.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101361.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101361.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101361.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000177732.6 --out compute_weights_out/ENSG00000177732.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000177732.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101361.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000177732.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000177732.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000177732.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000177732.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000177732.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000177732.6.bed +\n",
      "temp_files/test_chr20_ENSG00000177732.6.bim +\n",
      "temp_files/test_chr20_ENSG00000177732.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000177732.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000177732.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000177732.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000177732.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000177732.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000177732.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101367.8 --out compute_weights_out/ENSG00000101367.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101367.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000177732.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101367.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101367.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101367.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000101367.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101367.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101367.8.bed +\n",
      "temp_files/test_chr20_ENSG00000101367.8.bim +\n",
      "temp_files/test_chr20_ENSG00000101367.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101367.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101367.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101367.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101367.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101367.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101367.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000167046.4 --out compute_weights_out/ENSG00000167046.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000167046.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101367.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000167046.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000167046.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000167046.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000167046.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000167046.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000167046.4.bed +\n",
      "temp_files/test_chr20_ENSG00000167046.4.bim +\n",
      "temp_files/test_chr20_ENSG00000167046.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000167046.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000167046.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000167046.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000167046.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000167046.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000167046.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000181741.7 --out compute_weights_out/ENSG00000181741.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000181741.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000167046.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000181741.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000181741.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000181741.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000181741.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000181741.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000181741.7.bed +\n",
      "temp_files/test_chr20_ENSG00000181741.7.bim +\n",
      "temp_files/test_chr20_ENSG00000181741.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000181741.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000181741.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000181741.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000181741.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000181741.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000181741.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000175121.7 --out compute_weights_out/ENSG00000175121.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000175121.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000181741.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000175121.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000175121.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000175121.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000175121.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000175121.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000175121.7.bed +\n",
      "temp_files/test_chr20_ENSG00000175121.7.bim +\n",
      "temp_files/test_chr20_ENSG00000175121.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000175121.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000175121.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000175121.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000175121.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000175121.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000175121.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000230155.1 --out compute_weights_out/ENSG00000230155.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000230155.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000175121.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000230155.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000230155.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000230155.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000230155.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000230155.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000230155.1.bed +\n",
      "temp_files/test_chr20_ENSG00000230155.1.bim +\n",
      "temp_files/test_chr20_ENSG00000230155.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000230155.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000230155.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000230155.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000230155.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000230155.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000230155.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000171222.6 --out compute_weights_out/ENSG00000171222.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000171222.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000230155.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000171222.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000171222.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000171222.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000171222.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000171222.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000171222.6.bed +\n",
      "temp_files/test_chr20_ENSG00000171222.6.bim +\n",
      "temp_files/test_chr20_ENSG00000171222.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000171222.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000171222.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000171222.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000171222.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000171222.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000171222.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000232333.1 --out compute_weights_out/ENSG00000232333.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000232333.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000171222.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000232333.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000232333.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000232333.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000232333.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000232333.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000232333.1.bed +\n",
      "temp_files/test_chr20_ENSG00000232333.1.bim +\n",
      "temp_files/test_chr20_ENSG00000232333.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000232333.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000232333.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000232333.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000232333.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000232333.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000232333.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000197457.5 --out compute_weights_out/ENSG00000197457.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000197457.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000232333.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000197457.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000197457.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000197457.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000197457.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000197457.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000197457.5.bed +\n",
      "temp_files/test_chr20_ENSG00000197457.5.bim +\n",
      "temp_files/test_chr20_ENSG00000197457.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000197457.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000197457.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000197457.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000197457.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000197457.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000197457.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101158.7 --out compute_weights_out/ENSG00000101158.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101158.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000197457.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101158.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101158.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101158.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000101158.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101158.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101158.7.bed +\n",
      "temp_files/test_chr20_ENSG00000101158.7.bim +\n",
      "temp_files/test_chr20_ENSG00000101158.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101158.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101158.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101158.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101158.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101158.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101158.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000020256.14 --out compute_weights_out/ENSG00000020256.14_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000020256.14.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101158.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000020256.14.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000020256.14.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000020256.14\n",
      "  --pheno ../data/gene_expressions/ENSG00000020256.14.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000020256.14.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000020256.14.bed +\n",
      "temp_files/test_chr20_ENSG00000020256.14.bim +\n",
      "temp_files/test_chr20_ENSG00000020256.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000020256.14.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000020256.14\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000020256.14\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000020256.14.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000020256.14.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000020256.14.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101311.9 --out compute_weights_out/ENSG00000101311.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101311.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000020256.14 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101311.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101311.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101311.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000101311.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101311.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101311.9.bed +\n",
      "temp_files/test_chr20_ENSG00000101311.9.bim +\n",
      "temp_files/test_chr20_ENSG00000101311.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101311.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101311.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101311.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101311.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101311.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101311.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101365.13 --out compute_weights_out/ENSG00000101365.13_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101365.13.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101311.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101365.13.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101365.13.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101365.13\n",
      "  --pheno ../data/gene_expressions/ENSG00000101365.13.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101365.13.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101365.13.bed +\n",
      "temp_files/test_chr20_ENSG00000101365.13.bim +\n",
      "temp_files/test_chr20_ENSG00000101365.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101365.13.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101365.13\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101365.13\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101365.13.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101365.13.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101365.13.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000228265.1 --out compute_weights_out/ENSG00000228265.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000228265.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101365.13 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000228265.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000228265.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000228265.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000228265.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000228265.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000228265.1.bed +\n",
      "temp_files/test_chr20_ENSG00000228265.1.bim +\n",
      "temp_files/test_chr20_ENSG00000228265.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000228265.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000228265.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000228265.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000228265.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000228265.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000228265.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101278.6 --out compute_weights_out/ENSG00000101278.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101278.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000228265.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101278.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101278.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101278.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000101278.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101278.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101278.6.bed +\n",
      "temp_files/test_chr20_ENSG00000101278.6.bim +\n",
      "temp_files/test_chr20_ENSG00000101278.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101278.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101278.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101278.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101278.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101278.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101278.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000226038.4 --out compute_weights_out/ENSG00000226038.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000226038.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101278.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226038.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000226038.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000226038.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000226038.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226038.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000226038.4.bed +\n",
      "temp_files/test_chr20_ENSG00000226038.4.bim +\n",
      "temp_files/test_chr20_ENSG00000226038.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000226038.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000226038.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000226038.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000226038.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000226038.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000226038.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000229873.1 --out compute_weights_out/ENSG00000229873.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000229873.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000226038.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000229873.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000229873.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000229873.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000229873.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000229873.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000229873.1.bed +\n",
      "temp_files/test_chr20_ENSG00000229873.1.bim +\n",
      "temp_files/test_chr20_ENSG00000229873.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000229873.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000229873.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000229873.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000229873.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000229873.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000229873.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000259716.1 --out compute_weights_out/ENSG00000259716.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000259716.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000229873.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000259716.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000259716.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000259716.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000259716.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000259716.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000259716.1.bed +\n",
      "temp_files/test_chr20_ENSG00000259716.1.bim +\n",
      "temp_files/test_chr20_ENSG00000259716.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000259716.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000259716.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000259716.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000259716.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000259716.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000259716.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125531.5 --out compute_weights_out/ENSG00000125531.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125531.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000259716.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125531.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125531.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125531.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000125531.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125531.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125531.5.bed +\n",
      "temp_files/test_chr20_ENSG00000125531.5.bim +\n",
      "temp_files/test_chr20_ENSG00000125531.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125531.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125531.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125531.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125531.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125531.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125531.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124215.12 --out compute_weights_out/ENSG00000124215.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124215.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125531.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124215.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124215.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124215.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000124215.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124215.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124215.12.bed +\n",
      "temp_files/test_chr20_ENSG00000124215.12.bim +\n",
      "temp_files/test_chr20_ENSG00000124215.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124215.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124215.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124215.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124215.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124215.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124215.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000235065.1 --out compute_weights_out/ENSG00000235065.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000235065.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124215.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000235065.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000235065.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000235065.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000235065.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000235065.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000235065.1.bed +\n",
      "temp_files/test_chr20_ENSG00000235065.1.bim +\n",
      "temp_files/test_chr20_ENSG00000235065.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000235065.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000235065.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000235065.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000235065.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000235065.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000235065.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000228663.1 --out compute_weights_out/ENSG00000228663.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000228663.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000235065.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000228663.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000228663.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000228663.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000228663.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000228663.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000228663.1.bed +\n",
      "temp_files/test_chr20_ENSG00000228663.1.bim +\n",
      "temp_files/test_chr20_ENSG00000228663.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000228663.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000228663.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000228663.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000228663.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000228663.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000228663.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000225022.1 --out compute_weights_out/ENSG00000225022.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000225022.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000228663.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225022.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000225022.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000225022.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000225022.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225022.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000225022.1.bed +\n",
      "temp_files/test_chr20_ENSG00000225022.1.bim +\n",
      "temp_files/test_chr20_ENSG00000225022.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000225022.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000225022.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000225022.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000225022.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000225022.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000225022.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000213979.3 --out compute_weights_out/ENSG00000213979.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000213979.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000225022.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000213979.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000213979.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000213979.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000213979.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000213979.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000213979.3.bed +\n",
      "temp_files/test_chr20_ENSG00000213979.3.bim +\n",
      "temp_files/test_chr20_ENSG00000213979.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000213979.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000213979.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000213979.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000213979.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000213979.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000213979.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124209.3 --out compute_weights_out/ENSG00000124209.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124209.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000213979.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124209.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124209.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124209.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000124209.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124209.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124209.3.bed +\n",
      "temp_files/test_chr20_ENSG00000124209.3.bim +\n",
      "temp_files/test_chr20_ENSG00000124209.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124209.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124209.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124209.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124209.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124209.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124209.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000171873.6 --out compute_weights_out/ENSG00000171873.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000171873.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124209.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000171873.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000171873.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000171873.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000171873.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000171873.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000171873.6.bed +\n",
      "temp_files/test_chr20_ENSG00000171873.6.bim +\n",
      "temp_files/test_chr20_ENSG00000171873.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000171873.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000171873.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000171873.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000171873.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000171873.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000171873.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101343.10 --out compute_weights_out/ENSG00000101343.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101343.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000171873.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101343.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101343.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101343.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000101343.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101343.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101343.10.bed +\n",
      "temp_files/test_chr20_ENSG00000101343.10.bim +\n",
      "temp_files/test_chr20_ENSG00000101343.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101343.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101343.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101343.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101343.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101343.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101343.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101266.10 --out compute_weights_out/ENSG00000101266.10_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101266.10.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101343.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101266.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101266.10.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101266.10\n",
      "  --pheno ../data/gene_expressions/ENSG00000101266.10.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101266.10.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101266.10.bed +\n",
      "temp_files/test_chr20_ENSG00000101266.10.bim +\n",
      "temp_files/test_chr20_ENSG00000101266.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101266.10.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101266.10\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101266.10\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101266.10.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101266.10.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101266.10.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000233347.1 --out compute_weights_out/ENSG00000233347.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000233347.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101266.10 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000233347.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000233347.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000233347.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000233347.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000233347.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000233347.1.bed +\n",
      "temp_files/test_chr20_ENSG00000233347.1.bim +\n",
      "temp_files/test_chr20_ENSG00000233347.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000233347.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000233347.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000233347.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000233347.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000233347.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000233347.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124198.7 --out compute_weights_out/ENSG00000124198.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124198.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000233347.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124198.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124198.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124198.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000124198.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124198.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124198.7.bed +\n",
      "temp_files/test_chr20_ENSG00000124198.7.bim +\n",
      "temp_files/test_chr20_ENSG00000124198.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124198.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124198.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124198.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124198.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124198.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124198.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000168612.4 --out compute_weights_out/ENSG00000168612.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000168612.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124198.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000168612.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000168612.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000168612.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000168612.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000168612.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000168612.4.bed +\n",
      "temp_files/test_chr20_ENSG00000168612.4.bim +\n",
      "temp_files/test_chr20_ENSG00000168612.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000168612.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000168612.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000168612.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000168612.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000168612.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000168612.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124225.11 --out compute_weights_out/ENSG00000124225.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124225.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000168612.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124225.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124225.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124225.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000124225.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124225.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124225.11.bed +\n",
      "temp_files/test_chr20_ENSG00000124225.11.bim +\n",
      "temp_files/test_chr20_ENSG00000124225.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124225.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124225.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124225.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124225.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124225.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124225.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124164.11 --out compute_weights_out/ENSG00000124164.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124164.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124225.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124164.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124164.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124164.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000124164.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124164.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124164.11.bed +\n",
      "temp_files/test_chr20_ENSG00000124164.11.bim +\n",
      "temp_files/test_chr20_ENSG00000124164.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124164.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124164.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124164.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124164.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124164.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124164.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101307.11 --out compute_weights_out/ENSG00000101307.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101307.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124164.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101307.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101307.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101307.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000101307.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101307.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101307.11.bed +\n",
      "temp_files/test_chr20_ENSG00000101307.11.bim +\n",
      "temp_files/test_chr20_ENSG00000101307.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101307.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101307.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101307.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101307.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101307.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101307.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101138.6 --out compute_weights_out/ENSG00000101138.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101138.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101307.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101138.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101138.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101138.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000101138.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101138.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101138.6.bed +\n",
      "temp_files/test_chr20_ENSG00000101138.6.bim +\n",
      "temp_files/test_chr20_ENSG00000101138.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101138.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101138.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101138.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101138.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101138.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101138.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000131061.9 --out compute_weights_out/ENSG00000131061.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000131061.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101138.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000131061.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000131061.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000131061.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000131061.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000131061.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000131061.9.bed +\n",
      "temp_files/test_chr20_ENSG00000131061.9.bim +\n",
      "temp_files/test_chr20_ENSG00000131061.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000131061.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000131061.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000131061.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000131061.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000131061.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000131061.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101290.9 --out compute_weights_out/ENSG00000101290.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101290.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000131061.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101290.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101290.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101290.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000101290.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101290.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101290.9.bed +\n",
      "temp_files/test_chr20_ENSG00000101290.9.bim +\n",
      "temp_files/test_chr20_ENSG00000101290.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101290.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101290.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101290.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101290.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101290.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101290.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000244558.1 --out compute_weights_out/ENSG00000244558.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000244558.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101290.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000244558.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000244558.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000244558.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000244558.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000244558.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000244558.1.bed +\n",
      "temp_files/test_chr20_ENSG00000244558.1.bim +\n",
      "temp_files/test_chr20_ENSG00000244558.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000244558.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000244558.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000244558.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000244558.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000244558.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000244558.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000064655.13 --out compute_weights_out/ENSG00000064655.13_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000064655.13.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000244558.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000064655.13.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000064655.13.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000064655.13\n",
      "  --pheno ../data/gene_expressions/ENSG00000064655.13.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000064655.13.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000064655.13.bed +\n",
      "temp_files/test_chr20_ENSG00000064655.13.bim +\n",
      "temp_files/test_chr20_ENSG00000064655.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000064655.13.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000064655.13\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000064655.13\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000064655.13.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000064655.13.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000064655.13.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101400.5 --out compute_weights_out/ENSG00000101400.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101400.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000064655.13 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101400.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101400.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101400.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000101400.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101400.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101400.5.bed +\n",
      "temp_files/test_chr20_ENSG00000101400.5.bim +\n",
      "temp_files/test_chr20_ENSG00000101400.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101400.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101400.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101400.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101400.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101400.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101400.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000227063.4 --out compute_weights_out/ENSG00000227063.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000227063.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101400.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000227063.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000227063.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000227063.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000227063.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000227063.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000227063.4.bed +\n",
      "temp_files/test_chr20_ENSG00000227063.4.bim +\n",
      "temp_files/test_chr20_ENSG00000227063.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000227063.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000227063.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000227063.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000227063.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000227063.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000227063.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000130703.9 --out compute_weights_out/ENSG00000130703.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000130703.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000227063.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000130703.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000130703.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000130703.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000130703.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000130703.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000130703.9.bed +\n",
      "temp_files/test_chr20_ENSG00000130703.9.bim +\n",
      "temp_files/test_chr20_ENSG00000130703.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000130703.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000130703.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000130703.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000130703.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000130703.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000130703.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000078699.15 --out compute_weights_out/ENSG00000078699.15_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000078699.15.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000130703.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000078699.15.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000078699.15.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000078699.15\n",
      "  --pheno ../data/gene_expressions/ENSG00000078699.15.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000078699.15.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000078699.15.bed +\n",
      "temp_files/test_chr20_ENSG00000078699.15.bim +\n",
      "temp_files/test_chr20_ENSG00000078699.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000078699.15.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000078699.15\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000078699.15\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000078699.15.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000078699.15.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000078699.15.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124140.7 --out compute_weights_out/ENSG00000124140.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124140.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000078699.15 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124140.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124140.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124140.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000124140.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124140.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124140.7.bed +\n",
      "temp_files/test_chr20_ENSG00000124140.7.bim +\n",
      "temp_files/test_chr20_ENSG00000124140.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124140.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124140.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124140.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124140.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124140.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124140.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101294.11 --out compute_weights_out/ENSG00000101294.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101294.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124140.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101294.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101294.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101294.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000101294.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101294.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101294.11.bed +\n",
      "temp_files/test_chr20_ENSG00000101294.11.bim +\n",
      "temp_files/test_chr20_ENSG00000101294.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101294.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101294.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101294.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101294.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101294.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101294.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000088888.12 --out compute_weights_out/ENSG00000088888.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000088888.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101294.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088888.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000088888.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000088888.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000088888.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088888.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000088888.12.bed +\n",
      "temp_files/test_chr20_ENSG00000088888.12.bim +\n",
      "temp_files/test_chr20_ENSG00000088888.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000088888.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000088888.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000088888.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000088888.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000088888.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000088888.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000078814.11 --out compute_weights_out/ENSG00000078814.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000078814.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000088888.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000078814.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000078814.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000078814.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000078814.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000078814.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000078814.11.bed +\n",
      "temp_files/test_chr20_ENSG00000078814.11.bim +\n",
      "temp_files/test_chr20_ENSG00000078814.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000078814.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000078814.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000078814.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000078814.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000078814.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000078814.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000172315.5 --out compute_weights_out/ENSG00000172315.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000172315.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000078814.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000172315.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000172315.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000172315.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000172315.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000172315.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000172315.5.bed +\n",
      "temp_files/test_chr20_ENSG00000172315.5.bim +\n",
      "temp_files/test_chr20_ENSG00000172315.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000172315.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000172315.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000172315.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000172315.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000172315.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000172315.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000244005.5 --out compute_weights_out/ENSG00000244005.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000244005.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000172315.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000244005.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000244005.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000244005.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000244005.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000244005.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000244005.5.bed +\n",
      "temp_files/test_chr20_ENSG00000244005.5.bim +\n",
      "temp_files/test_chr20_ENSG00000244005.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000244005.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000244005.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000244005.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000244005.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000244005.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000244005.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000174306.14 --out compute_weights_out/ENSG00000174306.14_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000174306.14.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000244005.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000174306.14.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000174306.14.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000174306.14\n",
      "  --pheno ../data/gene_expressions/ENSG00000174306.14.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000174306.14.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000174306.14.bed +\n",
      "temp_files/test_chr20_ENSG00000174306.14.bim +\n",
      "temp_files/test_chr20_ENSG00000174306.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000174306.14.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000174306.14\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000174306.14\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000174306.14.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000174306.14.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000174306.14.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125508.3 --out compute_weights_out/ENSG00000125508.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125508.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000174306.14 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125508.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125508.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125508.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000125508.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125508.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125508.3.bed +\n",
      "temp_files/test_chr20_ENSG00000125508.3.bim +\n",
      "temp_files/test_chr20_ENSG00000125508.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125508.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125508.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125508.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125508.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125508.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125508.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000234241.1 --out compute_weights_out/ENSG00000234241.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000234241.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125508.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000234241.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000234241.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000234241.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000234241.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000234241.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000234241.1.bed +\n",
      "temp_files/test_chr20_ENSG00000234241.1.bim +\n",
      "temp_files/test_chr20_ENSG00000234241.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000234241.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000234241.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000234241.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000234241.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000234241.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000234241.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101391.14 --out compute_weights_out/ENSG00000101391.14_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101391.14.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000234241.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101391.14.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101391.14.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101391.14\n",
      "  --pheno ../data/gene_expressions/ENSG00000101391.14.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101391.14.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101391.14.bed +\n",
      "temp_files/test_chr20_ENSG00000101391.14.bim +\n",
      "temp_files/test_chr20_ENSG00000101391.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101391.14.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101391.14\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101391.14\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101391.14.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101391.14.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101391.14.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000100994.7 --out compute_weights_out/ENSG00000100994.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000100994.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101391.14 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000100994.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000100994.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000100994.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000100994.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000100994.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000100994.7.bed +\n",
      "temp_files/test_chr20_ENSG00000100994.7.bim +\n",
      "temp_files/test_chr20_ENSG00000100994.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000100994.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000100994.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000100994.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000100994.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000100994.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000100994.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000149658.12 --out compute_weights_out/ENSG00000149658.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000149658.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000100994.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149658.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000149658.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000149658.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000149658.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149658.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000149658.12.bed +\n",
      "temp_files/test_chr20_ENSG00000149658.12.bim +\n",
      "temp_files/test_chr20_ENSG00000149658.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149658.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000149658.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000149658.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149658.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000149658.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000149658.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125901.5 --out compute_weights_out/ENSG00000125901.5_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125901.5.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000149658.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125901.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125901.5.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125901.5\n",
      "  --pheno ../data/gene_expressions/ENSG00000125901.5.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125901.5.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125901.5.bed +\n",
      "temp_files/test_chr20_ENSG00000125901.5.bim +\n",
      "temp_files/test_chr20_ENSG00000125901.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125901.5.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125901.5\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125901.5\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125901.5.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125901.5.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125901.5.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000196421.3 --out compute_weights_out/ENSG00000196421.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000196421.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125901.5 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000196421.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000196421.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000196421.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000196421.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000196421.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000196421.3.bed +\n",
      "temp_files/test_chr20_ENSG00000196421.3.bim +\n",
      "temp_files/test_chr20_ENSG00000196421.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000196421.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000196421.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000196421.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000196421.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000196421.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000196421.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000230043.1 --out compute_weights_out/ENSG00000230043.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000230043.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000196421.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000230043.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000230043.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000230043.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000230043.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000230043.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000230043.1.bed +\n",
      "temp_files/test_chr20_ENSG00000230043.1.bim +\n",
      "temp_files/test_chr20_ENSG00000230043.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000230043.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000230043.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000230043.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000230043.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000230043.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000230043.1.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000100985.6 --out compute_weights_out/ENSG00000100985.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000100985.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000230043.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000100985.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000100985.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000100985.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000100985.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000100985.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000100985.6.bed +\n",
      "temp_files/test_chr20_ENSG00000100985.6.bim +\n",
      "temp_files/test_chr20_ENSG00000100985.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000100985.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000100985.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000100985.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000100985.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000100985.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000100985.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000149639.8 --out compute_weights_out/ENSG00000149639.8_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000149639.8.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000100985.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149639.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000149639.8.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000149639.8\n",
      "  --pheno ../data/gene_expressions/ENSG00000149639.8.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149639.8.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000149639.8.bed +\n",
      "temp_files/test_chr20_ENSG00000149639.8.bim +\n",
      "temp_files/test_chr20_ENSG00000149639.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000149639.8.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000149639.8\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000149639.8\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000149639.8.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000149639.8.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000149639.8.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000080845.12 --out compute_weights_out/ENSG00000080845.12_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000080845.12.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000149639.8 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000080845.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000080845.12.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000080845.12\n",
      "  --pheno ../data/gene_expressions/ENSG00000080845.12.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000080845.12.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000080845.12.bed +\n",
      "temp_files/test_chr20_ENSG00000080845.12.bim +\n",
      "temp_files/test_chr20_ENSG00000080845.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000080845.12.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000080845.12\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000080845.12\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000080845.12.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000080845.12.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000080845.12.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101166.11 --out compute_weights_out/ENSG00000101166.11_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101166.11.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000080845.12 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101166.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101166.11.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101166.11\n",
      "  --pheno ../data/gene_expressions/ENSG00000101166.11.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101166.11.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101166.11.bed +\n",
      "temp_files/test_chr20_ENSG00000101166.11.bim +\n",
      "temp_files/test_chr20_ENSG00000101166.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101166.11.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101166.11\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101166.11\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101166.11.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101166.11.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101166.11.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000101152.6 --out compute_weights_out/ENSG00000101152.6_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000101152.6.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101166.11 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101152.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000101152.6.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000101152.6\n",
      "  --pheno ../data/gene_expressions/ENSG00000101152.6.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101152.6.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000101152.6.bed +\n",
      "temp_files/test_chr20_ENSG00000101152.6.bim +\n",
      "temp_files/test_chr20_ENSG00000101152.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000101152.6.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000101152.6\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000101152.6\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000101152.6.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000101152.6.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000101152.6.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000215063.3 --out compute_weights_out/ENSG00000215063.3_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000215063.3.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000101152.6 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000215063.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000215063.3.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000215063.3\n",
      "  --pheno ../data/gene_expressions/ENSG00000215063.3.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000215063.3.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000215063.3.bed +\n",
      "temp_files/test_chr20_ENSG00000215063.3.bim +\n",
      "temp_files/test_chr20_ENSG00000215063.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000215063.3.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000215063.3\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000215063.3\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000215063.3.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000215063.3.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000215063.3.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000124228.9 --out compute_weights_out/ENSG00000124228.9_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000124228.9.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000215063.3 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124228.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000124228.9.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000124228.9\n",
      "  --pheno ../data/gene_expressions/ENSG00000124228.9.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124228.9.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000124228.9.bed +\n",
      "temp_files/test_chr20_ENSG00000124228.9.bim +\n",
      "temp_files/test_chr20_ENSG00000124228.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000124228.9.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000124228.9\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000124228.9\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000124228.9.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000124228.9.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000124228.9.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000125966.7 --out compute_weights_out/ENSG00000125966.7_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000125966.7.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000124228.9 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125966.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000125966.7.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000125966.7\n",
      "  --pheno ../data/gene_expressions/ENSG00000125966.7.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125966.7.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000125966.7.bed +\n",
      "temp_files/test_chr20_ENSG00000125966.7.bim +\n",
      "temp_files/test_chr20_ENSG00000125966.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000125966.7.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000125966.7\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000125966.7\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000125966.7.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000125966.7.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000125966.7.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000254995.4 --out compute_weights_out/ENSG00000254995.4_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000254995.4.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000125966.7 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000254995.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000254995.4.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000254995.4\n",
      "  --pheno ../data/gene_expressions/ENSG00000254995.4.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000254995.4.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000254995.4.bed +\n",
      "temp_files/test_chr20_ENSG00000254995.4.bim +\n",
      "temp_files/test_chr20_ENSG00000254995.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000254995.4.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000254995.4\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000254995.4\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000254995.4.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000254995.4.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000254995.4.grm.id .\n",
      "Running: Rscript fusion_twas-master/FUSION.compute_weights.R --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered --tmp temp_files/test_chr20_ENSG00000232241.1 --out compute_weights_out/ENSG00000232241.1_chr20 --models enet --PATH_plink ./plink --PATH_gcta fusion_twas-master/gcta --verbose 2 --pheno ../data/gene_expressions/ENSG00000232241.1.txt\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000254995.4 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000232241.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile ../data/LDREF_filtered/1000G.EUR.20_filtered\n",
      "  --keep ../data/gene_expressions/ENSG00000232241.1.txt\n",
      "  --make-bed\n",
      "  --out temp_files/test_chr20_ENSG00000232241.1\n",
      "  --pheno ../data/gene_expressions/ENSG00000232241.1.txt\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000232241.1.nosex .\n",
      "343 phenotype values present after --pheno.\n",
      "--keep: 343 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "--make-bed to temp_files/test_chr20_ENSG00000232241.1.bed +\n",
      "temp_files/test_chr20_ENSG00000232241.1.bim +\n",
      "temp_files/test_chr20_ENSG00000232241.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "### Estimating heritability\n",
      "PLINK v1.9.0-b.7.7 64-bit (22 Oct 2024)            cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2024 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to temp_files/test_chr20_ENSG00000232241.1.log.\n",
      "Options in effect:\n",
      "  --allow-no-sex\n",
      "  --bfile temp_files/test_chr20_ENSG00000232241.1\n",
      "  --make-grm-bin\n",
      "  --out temp_files/test_chr20_ENSG00000232241.1\n",
      "\n",
      "24576 MB RAM detected; reserving 12288 MB for main workspace.\n",
      "31101 variants loaded from .bim file.\n",
      "343 people (0 males, 0 females, 343 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to temp_files/test_chr20_ENSG00000232241.1.nosex .\n",
      "343 phenotype values loaded from .fam.\n",
      "Using up to 9 threads (change this with --threads).\n",
      "Before main variant filters, 343 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "31101 variants and 343 people pass filters and QC.\n",
      "Phenotype data is quantitative.\n",
      "Relationship matrix calculation complete.\n",
      "Relationship matrix written to temp_files/test_chr20_ENSG00000232241.1.grm.bin\n",
      ", and IDs written to temp_files/test_chr20_ENSG00000232241.1.grm.id .\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "sh: fusion_twas-master/gcta: No such file or directory\n",
      "temp_files/test_chr20_ENSG00000232241.1 does not exist, likely GCTA could not converge, skipping gene\n",
      "Warning message:\n",
      "In system(arg, ignore.stdout = SYS_PRINT, ignore.stderr = SYS_PRINT) :\n",
      "  error in running command\n"
     ]
    }
   ],
   "source": [
    "# Define constants for paths and parameters\n",
    "RSCRIPT_PATH = \"Rscript\"\n",
    "FUSION_SCRIPT = \"fusion_twas-master/FUSION.compute_weights.R\"\n",
    "BFILE_TEMPLATE = \"../data/LDREF_filtered/1000G.EUR.{chr}_filtered\"  # Placeholder for chromosome\n",
    "TMP_DIR = \"temp_files\"\n",
    "OUTPUT_DIR = \"compute_weights_out\"\n",
    "MODELS = \"enet\"\n",
    "PLINK_PATH = \"./plink\"\n",
    "GCTA_PATH = \"fusion_twas-master/gcta\"\n",
    "PHENO_DIR = \"../data/gene_expressions\"  # Directory with phenotype files\n",
    "VERBOSE = 2\n",
    "\n",
    "def compute_weights(gene, chromosome):\n",
    "    \"\"\"Runs the FUSION TWAS pipeline for a specific gene and chromosome.\"\"\"\n",
    "    # Construct file paths\n",
    "    bfile = BFILE_TEMPLATE.format(chr=chromosome)\n",
    "    tmp_file_prefix = os.path.join(TMP_DIR, f\"test_chr{chromosome}_{gene}\")\n",
    "    output_path = os.path.join(OUTPUT_DIR, f\"{gene}_chr{chromosome}\")\n",
    "    pheno_file = os.path.join(PHENO_DIR, f\"{gene}.txt\")\n",
    "    \n",
    "    # Construct the command\n",
    "    command = [\n",
    "        RSCRIPT_PATH, FUSION_SCRIPT,\n",
    "        \"--bfile\", bfile,\n",
    "        \"--tmp\", tmp_file_prefix,\n",
    "        \"--out\", output_path,\n",
    "        \"--models\", MODELS,\n",
    "        \"--PATH_plink\", PLINK_PATH,\n",
    "        \"--PATH_gcta\", GCTA_PATH,\n",
    "        \"--verbose\", str(VERBOSE),\n",
    "        \"--pheno\", pheno_file\n",
    "    ]\n",
    "    clear_command = ['rm', '-rf', 'temp_files/*']\n",
    "\n",
    "    # Print command for debugging\n",
    "    print(f\"Running: {' '.join(command)}\")\n",
    "    \n",
    "    # Run the command\n",
    "    try:\n",
    "        subprocess.run(command, check=True)\n",
    "        subprocess.run(clear_command, check=True)\n",
    "        \n",
    "    except subprocess.CalledProcessError as e:\n",
    "        print(f\"Error running command for gene {gene}, chromosome {chromosome}: {e}\")\n",
    "\n",
    "\n",
    "\n",
    "for i in to_calc.index:\n",
    "    gene_data = to_calc.loc[i]\n",
    "    compute_weights(gene_data['TargetID'], gene_data['Chr'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "myenv",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.11"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
